{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# cPEPmatch"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Recent updates: \n",
    "\n",
    "- Supports all motif size selections (4-6 work best). \n",
    "- Allows a selection of non-consecutive amino acids (set \"conscutive = False\").\n",
    "- The database program creator is now attached within the same program (if you want to run it, set \"CREATE_CYCLOLIB=1\")\n",
    "\n",
    "\n",
    "Created by Brianda L. Santini - \n",
    "Physics Department T38, Technical University of Munich, Garching, Germany."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys,os,re, glob\n",
    "import operator #set of efficient functions\n",
    "import pickle #store data on disc\n",
    "import time\n",
    "import warnings\n",
    "import numpy as np\n",
    "import shutil #import copyfile\n",
    "import csv\n",
    "\n",
    "from scipy import spatial\n",
    "from glob import glob\n",
    "from collections import defaultdict\n",
    "from itertools import product, combinations, permutations\n",
    "from sys import exit\n",
    "\n",
    "from modeller import *\n",
    "from modeller.optimizers import MolecularDynamics, ConjugateGradients\n",
    "from modeller.automodel import autosched\n",
    "\n",
    "import Bio\n",
    "from Bio.PDB import PDBParser\n",
    "from Bio.PDB.parse_pdb_header import parse_pdb_header\n",
    "\n",
    "import vmd\n",
    "from vmd import molecule, atomsel\n",
    "\n",
    "import MDAnalysis as mda\n",
    "import nglview as nv\n",
    "import pytraj as pt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "CREATE_INTERFACES=1\n",
    "PROCESS_INTERFACE=1\n",
    "FIND_MATCH=1\n",
    "CREATE_CYCLOLIB=1\n",
    "MUTATE=1\n",
    "\n",
    "interface_cutoff = 7\n",
    "frmsd_threshold = 0.7\n",
    "motif_size = 4 #can support all motif sizes (4-7 work best) \n",
    "consecutive = True #True or False\n",
    "\n",
    "\n",
    "protein='1 to 418' #Resid selection of protein to mimic.\n",
    "target = '419 to 424' #Resid selection of protein to target.\n",
    "pdb_name = '1e03'\n",
    "\n",
    "\n",
    "my_location = '/home/brianda/Dev/cPEPmatch/' #EXAMPLE: '/home/margrethe/Documents/cPEPmatch/'\n",
    "database_location = \"/cyclo_database/\"\n",
    "os.chdir('{}{}/' .format(my_location, pdb_name))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Cyclo Database"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Characterization the cyclic peptide motifs."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "def int_list(lyst, sep=' ', quote=None):\n",
    "    \"\"\"Return the list of numbers in 'lyst' as a string separated by 'sep'.\"\"\"\n",
    "    f = str if quote is None else lambda i: quote+str(i)+quote\n",
    "    return sep.join(map(f, lyst))\n",
    "\n",
    "def cyclo_distance_matrix(nres, R, fname):\n",
    "    M = dict()\n",
    "    combination = combinations(R, nres)\n",
    "    #residue_combination = list(combination)\n",
    "    #print(len(combination))\n",
    "    count = 1\n",
    "    for i in combination:\n",
    "        resids = i\n",
    "        #if not np.all(np.diff(resids)==1): continue\n",
    "        rlist = int_list(resids, quote='\"')\n",
    "        #if not rnbrs: continue\n",
    "        QA = '''(chain M and name CA and (altloc \"A\" or altloc \"\") and resid %(rlist)s)'''\n",
    "        QA = ' '.join(str(QA).split())\n",
    "        \n",
    "        resid_query = atomsel(QA % dict(rlist=rlist))\n",
    "        r = np.array(resid_query.centerperresidue())\n",
    "        # Ignore sequences that are either near the ends of a chain, or are near ends of a broken chain\n",
    "        if len(r) != nres: continue\n",
    "        #assert len(r) == nres, (len(r), r, nres, res, rnbrs, resid_query, fname)\n",
    "        dr = spatial.distance_matrix(r,r)\n",
    "        m = dr[np.triu_indices(nres,k=1)]\n",
    "        M[str(count)] = dict(m=m, resid=resids)\n",
    "        count = count + 1\n",
    "    return M\n",
    "\n",
    "def cyclo_distance_matrix_consecutive(nres, R, fname):\n",
    "    M = dict()\n",
    "    \n",
    "    for i,res in enumerate(R[:-nres+1]):\n",
    "        #print(res, res+nres, R[i:i+nres])\n",
    "        resids = R[i:i+nres]\n",
    "        if not np.all(np.diff(resids)==1): continue\n",
    "        rlist = int_list(resids, quote='\"')\n",
    "        #if not rnbrs: continue\n",
    "        QA = '''(chain M and name CA and (altloc \"A\" or altloc \"\") and resid %(rlist)s)'''\n",
    "        QA = ' '.join(str(QA).split())\n",
    "        \n",
    "        resid_query = atomsel(QA % dict(rlist=rlist))\n",
    "        r = np.array(resid_query.centerperresidue())\n",
    "        \n",
    "        #\n",
    "        # Ignore sequences that are either near the ends of a chain, or are near ends of a broken chain\n",
    "        #\n",
    "        if len(r) != nres: continue\n",
    "        assert len(r) == nres, (len(r), r, nres, res, rnbrs, resid_query, fname)\n",
    "        \n",
    "        dr = spatial.distance_matrix(r,r)\n",
    "        m = dr[np.triu_indices(nres,k=2)]\n",
    "        M[resids[0]] = dict(\n",
    "            m=m,\n",
    "#            r=r,\n",
    "            resid=resids,\n",
    "        )\n",
    "    return M\n",
    "\n",
    "\n",
    "def cyclo_motifs(molid):\n",
    "    MC = dict()\n",
    "    C = atomsel('chain M and protein and backbone and (altloc \"A\" or altloc \"\") ')\n",
    "    C_res = list(sorted(set(C.resid)))\n",
    "    if consecutive:\n",
    "        MC = cyclo_distance_matrix_consecutive(motif_size, C_res, cyclo)\n",
    "    else:\n",
    "        MC = cyclo_distance_matrix(motif_size, C_res, cyclo)\n",
    "    return (MC)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "if CREATE_CYCLOLIB:\n",
    "    os.chdir(my_location + database_location)\n",
    "    #os.chdir('/home/brianda/Documents/cPEPmatch/cyclo_database')\n",
    "\n",
    "    parser = PDBParser()\n",
    "    cyclo_mtfs = []\n",
    "    \n",
    "    if MUTATE:\n",
    "        lib = ['1cnl', '1ebp', '1foz', '1jbl', '1l5g', '1npo', '1qx9', '1sld', '1sle', '1urc', '1vwb','2ajw', \n",
    "               '2ak0', '2c9t', '2jrw', '2lws', '2lwt', '2lwu', '2lwv', '2m2g', '2mgo', '2n07', '2ox2', '3av9', \n",
    "               '3ava', '3avb', '3avi', '3p8f', '3wbn', '3wne', '3wnf', '3zwz', '4gly', '4ib5', \n",
    "               '4k8y', '4kts', '4ktu', '4mq9', '5bxo', '5eoc', '5glh', '5lff', '5lso', \n",
    "               '5vav', '5xco', '5xn3', '6awk', '6awm', '6axi', '6pin', '6pio', '6pip' \n",
    "              ]\n",
    "    else:\n",
    "        lib = ['2c9t', '1cnl', '1tk2', '5eoc', '3zwz', '4z0f', '4zjx', '4ib5', '3oe0', '6c1r', '1urc', '4w4z', \n",
    "              '1ebp', '3pqz', '3avb', '4n7y', '5xco', '4k8y', '1jbl', '3p8f', '4i80', '5afg', '3wbn', '1npo', \n",
    "              '2mgo', '3wmg', '5kgn', '5b4w', '5lso', '5xn3', '1sld', '5bxo', '4zqw', '4ktu', '4kts', '4twt', \n",
    "              '3qn7', '3av9', '3ava', '3avi', '3wnf', '1sle', '1vwb', '5n99', '4os1', '4mnw', '4jk5', '4gly', \n",
    "              '4x1n', '5glh', '2m2g', '2lwv', '6pip', '2lwt', '1foz', '2ajw', '5lff', '2ox2', '6axi', '2lwu', \n",
    "              '3wne', '2ak0', '2lws', '1qx9', '6pin', '2n07', '1l5g', '6pio', '2jrw', '6awm', '5vav', '6awk' ]\n",
    "    \n",
    "\n",
    "    \n",
    "    for c in lib:\n",
    "        motifs = dict()\n",
    "        cyclo = ('%s-cp.pdb' % c)\n",
    "        molid = molecule.load(\"pdb\", cyclo )\n",
    "        c_mtfs = cyclo_motifs(molid)\n",
    "        if c_mtfs is not None:\n",
    "            #save_interacting_sidechains(os.path.splitext(f)[0], isc)\n",
    "            motifs.update({c: c_mtfs})\n",
    "            cyclo_mtfs.append(motifs)\n",
    "            \n",
    "    database = open(\"database_%s.pkl\" % motif_size, \"wb\")\n",
    "    pickle.dump(cyclo_mtfs, database)\n",
    "    database.close()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Protein-Target characterization"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This section selects the residues of the PPI interface and characterizes its CA motifs. \n",
    "Name the PPI input file as 'complex.pdb'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Interface "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "def interfaces(molid, cutoff=interface_cutoff, minresidues=4):\n",
    "    I = []\n",
    "    I = atomsel(\"resid {} and same residue as exwithin {} of resid {}\".format(protein, cutoff, target))\n",
    "    if len(set(I.residue)) < minresidues: \n",
    "        I = []\n",
    "    return I\n",
    "\n",
    "\n",
    "def create_interfaces(): \n",
    "    parser = PDBParser()\n",
    "    try:\n",
    "        pdb_fname = (\"{}.pdb\" .format(pdb_name))\n",
    "        molid  = molecule.load(\"pdb\", pdb_fname) #from vmd library \n",
    "        Iall = interfaces(molid) \n",
    "        int_fname = 'interface.pdb'\n",
    "        print('%s -> %s' % (pdb_fname, int_fname))\n",
    "        molecule.write(molid, 'pdb', int_fname, selection=Iall)\n",
    "        molecule.delete(molid)\n",
    "    except RuntimeError as e:\n",
    "        warn(e)\n",
    "        pass\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1e03.pdb -> interface.pdb\n"
     ]
    }
   ],
   "source": [
    "if CREATE_INTERFACES:\n",
    "    os.chdir('{}{}/' .format(my_location, pdb_name))\n",
    "    create_interfaces()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "f9c4635efde845589cd2b0cbc91ce208",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "view = nv.show_file('{}{}/interface.pdb' .format(my_location, pdb_name))\n",
    "view.clear_representations()\n",
    "view.add_representation('ball+stick', selection='protein')\n",
    "view"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Motif construction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "def int_list(lyst, sep=' ', quote=None):\n",
    "    \"\"\"Return the list of numbers in 'lyst' as a string separated by 'sep'.\"\"\"\n",
    "    f = str if quote is None else lambda i: quote+str(i)+quote\n",
    "    return sep.join(map(f, lyst))\n",
    "\n",
    "def load_interface(pdb_fname): # process_all_interfaces(load_interface,...)\n",
    "    \"\"\"Return a vmd molecule for the interface found in pdb_fname in a tuple of (vmd-mol, pdb-code, tag, list-of-chains).\"\"\"\n",
    "    d,f = os.path.split(pdb_fname) # d = path/.../, f = x.pdb\n",
    "    pdb,tag,chains,ext = f.split('.') # e.g. 4gko, pp, B-G, pdb\n",
    "    chains = chains.split('-')\n",
    "    mol = molecule.load(\"pdb\", pdb_fname) # What is molecule in that context? \n",
    "    #vmd.render.render('PostScript', pdb_fname + '.ps')\n",
    "    #molecule.ssrecalc(int(mol)) # recalculate secondary structure\n",
    "    return (mol, pdb, tag, chains) # (12, '4gko', 'pp', ['D', 'J'])\n",
    "\n",
    "\n",
    "def distance_matrix_consecutive(nres, R, fname):\n",
    "    M = dict()\n",
    "    for i,res in enumerate(R[:-nres+1]):\n",
    "        resids = R[i:i+nres]\n",
    "        if not np.all(np.diff(resids)==1): continue\n",
    "        rlist = int_list(resids, quote='\"')      \n",
    "        resid_query = atomsel(\"name CA and resid {}\" .format(rlist))\n",
    "        clz = resid_query.structure\n",
    "        r = np.array(resid_query.centerperresidue())\n",
    "        \n",
    "        # Ignore sequences that are either near the ends of a chain, or are near ends of a broken chain\n",
    "        \n",
    "        if len(r) != nres: continue\n",
    "        assert len(r) == nres, (len(r), r, nres, res, rnbrs, resid_query, fname)\n",
    "        \n",
    "        dr = spatial.distance_matrix(r,r)\n",
    "        m = dr[np.triu_indices(nres,k=2)]\n",
    "        M[resids[0]] = dict(\n",
    "            m=m,\n",
    "            resid=resids,\n",
    "        )\n",
    "    return M\n",
    "\n",
    "\n",
    "def distance_matrix(nres, R, fname):\n",
    "    M = dict()\n",
    "    combination = combinations(R, nres)\n",
    "    #residue_combination = list(combination)\n",
    "    count = 1\n",
    "    for i in combination:\n",
    "        resids = i\n",
    "        #if not np.all(np.diff(resids)==1): continue\n",
    "        rlist = int_list(resids, quote='\"')\n",
    "        #if not rnbrs: continue\n",
    "        resid_query = atomsel(\"name CA and resid {}\" .format(rlist))\n",
    "        r = np.array(resid_query.centerperresidue())\n",
    "        # Ignore sequences that are either near the ends of a chain, or are near ends of a broken chain\n",
    "        #if len(r) != nres: continue\n",
    "        #assert len(r) == nres, (len(r), r, nres, res, rnbrs, resid_query, fname)\n",
    "        dr = spatial.distance_matrix(r,r)\n",
    "        m = dr[np.triu_indices(nres,k=1)]\n",
    "        M[str(count)] = dict(m=m, resid=resids)\n",
    "        count = count + 1\n",
    "    return M\n",
    "        \n",
    "def motifs(molid):\n",
    "    P = atomsel(\"all\")\n",
    "    P_res = list(sorted(set(P.resid)))\n",
    "    if consecutive:\n",
    "        MP = distance_matrix_consecutive(motif_size, P_res, inter_fname)\n",
    "    else:\n",
    "        MP = distance_matrix(motif_size, P_res, inter_fname)\n",
    "    return (MP)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{3: {'m': array([5.38666371, 7.81653292, 6.73579162]), 'resid': [3, 4, 5, 6]},\n",
       " 4: {'m': array([ 6.73579162, 10.28086966,  7.04468341]),\n",
       "  'resid': [4, 5, 6, 7]},\n",
       " 5: {'m': array([7.04468341, 8.36494192, 5.68353729]), 'resid': [5, 6, 7, 8]},\n",
       " 6: {'m': array([5.68353729, 5.15543138, 5.44578758]), 'resid': [6, 7, 8, 9]},\n",
       " 7: {'m': array([5.44578758, 6.22028731, 5.43225326]), 'resid': [7, 8, 9, 10]},\n",
       " 27: {'m': array([6.64064317, 8.87923684, 6.6118616 ]),\n",
       "  'resid': [27, 28, 29, 30]},\n",
       " 28: {'m': array([6.6118616 , 5.95431307, 5.5173707 ]),\n",
       "  'resid': [28, 29, 30, 31]},\n",
       " 29: {'m': array([5.5173707 , 9.07257982, 7.03192473]),\n",
       "  'resid': [29, 30, 31, 32]},\n",
       " 30: {'m': array([7.03192473, 9.41077431, 6.21125978]),\n",
       "  'resid': [30, 31, 32, 33]},\n",
       " 31: {'m': array([6.21125978, 8.7194961 , 5.42618795]),\n",
       "  'resid': [31, 32, 33, 34]},\n",
       " 32: {'m': array([5.42618795, 5.08121134, 5.41419375]),\n",
       "  'resid': [32, 33, 34, 35]},\n",
       " 33: {'m': array([5.41419375, 4.98696946, 5.37523153]),\n",
       "  'resid': [33, 34, 35, 36]},\n",
       " 34: {'m': array([5.37523153, 5.39554064, 5.71690109]),\n",
       "  'resid': [34, 35, 36, 37]},\n",
       " 99: {'m': array([5.83726425, 5.4216955 , 5.50729478]),\n",
       "  'resid': [99, 100, 101, 102]},\n",
       " 100: {'m': array([5.50729478, 5.09099765, 5.45180124]),\n",
       "  'resid': [100, 101, 102, 103]},\n",
       " 101: {'m': array([5.45180124, 5.34876502, 5.39044793]),\n",
       "  'resid': [101, 102, 103, 104]},\n",
       " 102: {'m': array([5.39044793, 4.94847699, 5.71720416]),\n",
       "  'resid': [102, 103, 104, 105]}}"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "if PROCESS_INTERFACE:\n",
    "    os.chdir('{}{}/'.format(my_location, pdb_name))\n",
    "    parser = PDBParser()\n",
    "    inter_fname = \"interface.pdb\"\n",
    "    molid  = molecule.load(\"pdb\", inter_fname)\n",
    "    mtfs = motifs(molid)\n",
    "    \n",
    "mtfs"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Backbone Match"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This section matches the CA distance motifs of the PPI and the cyclic peptide database. 18:35"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('1cnl', 0.3671, [9, 10, 11, 12], [7, 8, 9, 10]),\n",
       " ('1cnl', 0.3912, [1, 2, 3, 4], [33, 34, 35, 36]),\n",
       " ('1cnl', 0.4108, [1, 2, 3, 4], [6, 7, 8, 9]),\n",
       " ('1cnl', 0.4194, [1, 2, 3, 4], [100, 101, 102, 103]),\n",
       " ('1cnl', 0.4492, [1, 2, 3, 4], [32, 33, 34, 35]),\n",
       " ('1cnl', 0.4605, [5, 6, 7, 8], [7, 8, 9, 10]),\n",
       " ('1cnl', 0.4733, [7, 8, 9, 10], [31, 32, 33, 34]),\n",
       " ('1cnl', 0.4818, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('1cnl', 0.5807, [1, 2, 3, 4], [102, 103, 104, 105]),\n",
       " ('1cnl', 0.5848, [8, 9, 10, 11], [31, 32, 33, 34]),\n",
       " ('1cnl', 0.627, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('1cnl', 0.6417, [1, 2, 3, 4], [101, 102, 103, 104]),\n",
       " ('1cnl', 0.659, [4, 5, 6, 7], [5, 6, 7, 8]),\n",
       " ('1cnl', 0.6857, [1, 2, 3, 4], [99, 100, 101, 102]),\n",
       " ('1cnl', 0.6984, [2, 3, 4, 5], [7, 8, 9, 10]),\n",
       " ('1ebp', 0.3303, [13, 14, 15, 16], [30, 31, 32, 33]),\n",
       " ('1ebp', 0.3828, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('1ebp', 0.4104, [12, 13, 14, 15], [4, 5, 6, 7]),\n",
       " ('1ebp', 0.4519, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('1ebp', 0.4742, [7, 8, 9, 10], [34, 35, 36, 37]),\n",
       " ('1ebp', 0.4861, [13, 14, 15, 16], [27, 28, 29, 30]),\n",
       " ('1ebp', 0.509, [10, 11, 12, 13], [30, 31, 32, 33]),\n",
       " ('1ebp', 0.5212, [4, 5, 6, 7], [30, 31, 32, 33]),\n",
       " ('1ebp', 0.5458, [9, 10, 11, 12], [27, 28, 29, 30]),\n",
       " ('1ebp', 0.5541, [4, 5, 6, 7], [27, 28, 29, 30]),\n",
       " ('1ebp', 0.5541, [7, 8, 9, 10], [102, 103, 104, 105]),\n",
       " ('1ebp', 0.6288, [6, 7, 8, 9], [5, 6, 7, 8]),\n",
       " ('1ebp', 0.6469, [3, 4, 5, 6], [4, 5, 6, 7]),\n",
       " ('1foz', 0.4281, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('1foz', 0.4493, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('1foz', 0.5882, [1, 2, 3, 4], [31, 32, 33, 34]),\n",
       " ('1foz', 0.6144, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('1foz', 0.6155, [3, 4, 5, 6], [29, 30, 31, 32]),\n",
       " ('1foz', 0.6421, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('1foz', 0.6479, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('1jbl', 0.1135, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('1jbl', 0.3515, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('1jbl', 0.429, [4, 5, 6, 7], [3, 4, 5, 6]),\n",
       " ('1jbl', 0.5031, [3, 4, 5, 6], [5, 6, 7, 8]),\n",
       " ('1jbl', 0.5039, [3, 4, 5, 6], [31, 32, 33, 34]),\n",
       " ('1jbl', 0.557, [8, 9, 10, 11], [30, 31, 32, 33]),\n",
       " ('1jbl', 0.5903, [11, 12, 13, 14], [5, 6, 7, 8]),\n",
       " ('1jbl', 0.6049, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('1jbl', 0.6593, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('1jbl', 0.662, [8, 9, 10, 11], [27, 28, 29, 30]),\n",
       " ('1jbl', 0.6712, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('1l5g', 0.5617, [1, 2, 3, 4], [28, 29, 30, 31]),\n",
       " ('1l5g', 0.5967, [1, 2, 3, 4], [99, 100, 101, 102]),\n",
       " ('1npo', 0.2188, [5, 6, 7, 8], [27, 28, 29, 30]),\n",
       " ('1npo', 0.2322, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('1npo', 0.281, [2, 3, 4, 5], [33, 34, 35, 36]),\n",
       " ('1npo', 0.2997, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('1npo', 0.3141, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('1npo', 0.3186, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('1npo', 0.3643, [2, 3, 4, 5], [102, 103, 104, 105]),\n",
       " ('1npo', 0.3728, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('1npo', 0.468, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('1npo', 0.4952, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('1npo', 0.5048, [6, 7, 8, 9], [28, 29, 30, 31]),\n",
       " ('1npo', 0.6133, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('1npo', 0.6233, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('1npo', 0.6273, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('1npo', 0.6305, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('1npo', 0.6375, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('1npo', 0.6528, [1, 2, 3, 4], [31, 32, 33, 34]),\n",
       " ('1npo', 0.6529, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('1npo', 0.6747, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('1npo', 0.6885, [4, 5, 6, 7], [29, 30, 31, 32]),\n",
       " ('1qx9', 0.1226, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('1qx9', 0.2116, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('1qx9', 0.325, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('1qx9', 0.4284, [12, 13, 14, 15], [29, 30, 31, 32]),\n",
       " ('1qx9', 0.4374, [11, 12, 13, 14], [30, 31, 32, 33]),\n",
       " ('1qx9', 0.4828, [1, 2, 3, 4], [5, 6, 7, 8]),\n",
       " ('1qx9', 0.4978, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('1qx9', 0.5846, [4, 5, 6, 7], [5, 6, 7, 8]),\n",
       " ('1qx9', 0.6227, [6, 7, 8, 9], [101, 102, 103, 104]),\n",
       " ('1qx9', 0.6395, [3, 4, 5, 6], [29, 30, 31, 32]),\n",
       " ('1qx9', 0.6528, [8, 9, 10, 11], [30, 31, 32, 33]),\n",
       " ('1qx9', 0.6798, [7, 8, 9, 10], [29, 30, 31, 32]),\n",
       " ('1qx9', 0.6994, [1, 2, 3, 4], [27, 28, 29, 30]),\n",
       " ('1sld', 0.2701, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('1sld', 0.3678, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('1sld', 0.4424, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('1sld', 0.4619, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('1sld', 0.4689, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('1sld', 0.4984, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('1sld', 0.526, [4, 5, 6, 7], [102, 103, 104, 105]),\n",
       " ('1sld', 0.535, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('1sld', 0.5936, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('1sld', 0.6324, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('1sle', 0.3342, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('1sle', 0.3526, [4, 5, 6, 7], [3, 4, 5, 6]),\n",
       " ('1sle', 0.4028, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('1sle', 0.4033, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('1sle', 0.4701, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('1sle', 0.512, [3, 4, 5, 6], [7, 8, 9, 10]),\n",
       " ('1sle', 0.555, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('1sle', 0.5803, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('1sle', 0.6212, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('1sle', 0.6392, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('1sle', 0.6638, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('1urc', 0.6773, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('1vwb', 0.3442, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('1vwb', 0.5007, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('1vwb', 0.5125, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('1vwb', 0.5427, [4, 5, 6, 7], [102, 103, 104, 105]),\n",
       " ('1vwb', 0.5482, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('1vwb', 0.5546, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('1vwb', 0.5636, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('1vwb', 0.5648, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('1vwb', 0.6091, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('1vwb', 0.6412, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.0814, [10, 11, 12, 13], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.1066, [5, 6, 7, 8], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.1135, [5, 6, 7, 8], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.1193, [7, 8, 9, 10], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.1215, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.1358, [6, 7, 8, 9], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.14, [14, 15, 16, 17], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.1438, [6, 7, 8, 9], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.1452, [5, 6, 7, 8], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.1497, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.1617, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.188, [6, 7, 8, 9], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.2035, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('2ajw', 0.2272, [5, 6, 7, 8], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.231, [14, 15, 16, 17], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.2323, [7, 8, 9, 10], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.2398, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.2408, [13, 14, 15, 16], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.2468, [14, 15, 16, 17], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.2529, [6, 7, 8, 9], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.2709, [12, 13, 14, 15], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.2791, [14, 15, 16, 17], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.29, [5, 6, 7, 8], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.3187, [1, 2, 3, 4], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.3291, [7, 8, 9, 10], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.3291, [14, 15, 16, 17], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.3296, [7, 8, 9, 10], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.3355, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.3362, [14, 15, 16, 17], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.3382, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.3495, [19, 20, 21, 22], [7, 8, 9, 10]),\n",
       " ('2ajw', 0.3613, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.3738, [7, 8, 9, 10], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.377, [10, 11, 12, 13], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.3782, [15, 16, 17, 18], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.383, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.3907, [13, 14, 15, 16], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.3909, [10, 11, 12, 13], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.3967, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.3975, [6, 7, 8, 9], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.3977, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.4014, [12, 13, 14, 15], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.4161, [10, 11, 12, 13], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.417, [5, 6, 7, 8], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.4184, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.4213, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.4297, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.4299, [7, 8, 9, 10], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.4326, [7, 8, 9, 10], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.4385, [12, 13, 14, 15], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.4389, [13, 14, 15, 16], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.4398, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.4455, [2, 3, 4, 5], [7, 8, 9, 10]),\n",
       " ('2ajw', 0.451, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.4555, [13, 14, 15, 16], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.4559, [12, 13, 14, 15], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.4594, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.4658, [1, 2, 3, 4], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.4665, [12, 13, 14, 15], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.4857, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.4903, [10, 11, 12, 13], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.509, [7, 8, 9, 10], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.5105, [14, 15, 16, 17], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.5156, [1, 2, 3, 4], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.5293, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.5355, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.5433, [14, 15, 16, 17], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.546, [10, 11, 12, 13], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.5532, [17, 18, 19, 20], [28, 29, 30, 31]),\n",
       " ('2ajw', 0.5551, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.5589, [12, 13, 14, 15], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.5626, [5, 6, 7, 8], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.5691, [1, 2, 3, 4], [7, 8, 9, 10]),\n",
       " ('2ajw', 0.583, [1, 2, 3, 4], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.5846, [1, 2, 3, 4], [32, 33, 34, 35]),\n",
       " ('2ajw', 0.5886, [1, 2, 3, 4], [100, 101, 102, 103]),\n",
       " ('2ajw', 0.5927, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.599, [13, 14, 15, 16], [6, 7, 8, 9]),\n",
       " ('2ajw', 0.6119, [11, 12, 13, 14], [31, 32, 33, 34]),\n",
       " ('2ajw', 0.6303, [12, 13, 14, 15], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.6415, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.6654, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('2ajw', 0.6691, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.6703, [12, 13, 14, 15], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.6726, [13, 14, 15, 16], [34, 35, 36, 37]),\n",
       " ('2ajw', 0.6746, [1, 2, 3, 4], [33, 34, 35, 36]),\n",
       " ('2ajw', 0.6863, [13, 14, 15, 16], [101, 102, 103, 104]),\n",
       " ('2ajw', 0.6938, [10, 11, 12, 13], [99, 100, 101, 102]),\n",
       " ('2ajw', 0.6946, [15, 16, 17, 18], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.0677, [7, 8, 9, 10], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.1054, [10, 11, 12, 13], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.1642, [10, 11, 12, 13], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.1646, [10, 11, 12, 13], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.1701, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.1789, [6, 7, 8, 9], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.1843, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.1934, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.1946, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.1976, [6, 7, 8, 9], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.2038, [7, 8, 9, 10], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.2038, [13, 14, 15, 16], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.2116, [7, 8, 9, 10], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.2129, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.2161, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.2243, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.2469, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.2482, [5, 6, 7, 8], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.2507, [5, 6, 7, 8], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.2574, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.2609, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('2ak0', 0.2641, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.2665, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.273, [5, 6, 7, 8], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.274, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.276, [7, 8, 9, 10], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.2792, [10, 11, 12, 13], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.288, [6, 7, 8, 9], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.296, [5, 6, 7, 8], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.2966, [7, 8, 9, 10], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.306, [12, 13, 14, 15], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.3128, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.3199, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.321, [5, 6, 7, 8], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.3255, [14, 15, 16, 17], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.335, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.355, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.3594, [7, 8, 9, 10], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.3686, [10, 11, 12, 13], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.3781, [12, 13, 14, 15], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.3812, [14, 15, 16, 17], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.3848, [12, 13, 14, 15], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.3871, [14, 15, 16, 17], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.4038, [12, 13, 14, 15], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.4038, [19, 20, 21, 22], [28, 29, 30, 31]),\n",
       " ('2ak0', 0.4052, [10, 11, 12, 13], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.4199, [5, 6, 7, 8], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.4251, [1, 2, 3, 4], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.4316, [13, 14, 15, 16], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.4335, [7, 8, 9, 10], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.4337, [14, 15, 16, 17], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.4506, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.4558, [5, 6, 7, 8], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.4609, [1, 2, 3, 4], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.4609, [12, 13, 14, 15], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.4645, [6, 7, 8, 9], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.465, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.4684, [14, 15, 16, 17], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.4747, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.4749, [7, 8, 9, 10], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.4758, [1, 2, 3, 4], [7, 8, 9, 10]),\n",
       " ('2ak0', 0.4906, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.4908, [14, 15, 16, 17], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.4965, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.4994, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.5049, [13, 14, 15, 16], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.5134, [14, 15, 16, 17], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.5154, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.5275, [2, 3, 4, 5], [33, 34, 35, 36]),\n",
       " ('2ak0', 0.5379, [1, 2, 3, 4], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.5421, [14, 15, 16, 17], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.5489, [18, 19, 20, 21], [27, 28, 29, 30]),\n",
       " ('2ak0', 0.5731, [20, 21, 22, 23], [31, 32, 33, 34]),\n",
       " ('2ak0', 0.5814, [19, 20, 21, 22], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.5825, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.5859, [13, 14, 15, 16], [100, 101, 102, 103]),\n",
       " ('2ak0', 0.5884, [13, 14, 15, 16], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.5903, [10, 11, 12, 13], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.5929, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.5971, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('2ak0', 0.5984, [6, 7, 8, 9], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.6084, [13, 14, 15, 16], [32, 33, 34, 35]),\n",
       " ('2ak0', 0.6253, [1, 2, 3, 4], [6, 7, 8, 9]),\n",
       " ('2ak0', 0.641, [12, 13, 14, 15], [101, 102, 103, 104]),\n",
       " ('2ak0', 0.6437, [13, 14, 15, 16], [102, 103, 104, 105]),\n",
       " ('2ak0', 0.6562, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('2ak0', 0.6955, [1, 2, 3, 4], [100, 101, 102, 103]),\n",
       " ('2c9t', 0.0703, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('2c9t', 0.0867, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('2c9t', 0.1161, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('2c9t', 0.1536, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('2c9t', 0.1556, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('2c9t', 0.1564, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('2c9t', 0.2197, [2, 3, 4, 5], [33, 34, 35, 36]),\n",
       " ('2c9t', 0.2253, [5, 6, 7, 8], [32, 33, 34, 35]),\n",
       " ('2c9t', 0.2328, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('2c9t', 0.2425, [5, 6, 7, 8], [100, 101, 102, 103]),\n",
       " ('2c9t', 0.246, [8, 9, 10, 11], [7, 8, 9, 10]),\n",
       " ('2c9t', 0.2483, [5, 6, 7, 8], [34, 35, 36, 37]),\n",
       " ('2c9t', 0.2807, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('2c9t', 0.2859, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('2c9t', 0.2939, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('2c9t', 0.3051, [2, 3, 4, 5], [102, 103, 104, 105]),\n",
       " ('2c9t', 0.3224, [5, 6, 7, 8], [33, 34, 35, 36]),\n",
       " ('2c9t', 0.3473, [5, 6, 7, 8], [6, 7, 8, 9]),\n",
       " ('2c9t', 0.3538, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('2c9t', 0.3806, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('2c9t', 0.4001, [5, 6, 7, 8], [102, 103, 104, 105]),\n",
       " ('2c9t', 0.4058, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('2c9t', 0.4064, [1, 2, 3, 4], [99, 100, 101, 102]),\n",
       " ('2c9t', 0.4317, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('2c9t', 0.4468, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('2c9t', 0.4526, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('2c9t', 0.4611, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('2c9t', 0.4939, [5, 6, 7, 8], [99, 100, 101, 102]),\n",
       " ('2c9t', 0.5006, [1, 2, 3, 4], [7, 8, 9, 10]),\n",
       " ('2c9t', 0.5214, [1, 2, 3, 4], [101, 102, 103, 104]),\n",
       " ('2c9t', 0.5232, [1, 2, 3, 4], [34, 35, 36, 37]),\n",
       " ('2c9t', 0.5547, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('2c9t', 0.6163, [3, 4, 5, 6], [3, 4, 5, 6]),\n",
       " ('2c9t', 0.645, [1, 2, 3, 4], [6, 7, 8, 9]),\n",
       " ('2jrw', 0.1932, [11, 12, 13, 14], [4, 5, 6, 7]),\n",
       " ('2jrw', 0.3238, [12, 13, 14, 15], [5, 6, 7, 8]),\n",
       " ('2jrw', 0.3664, [20, 21, 22, 23], [99, 100, 101, 102]),\n",
       " ('2jrw', 0.3952, [5, 6, 7, 8], [4, 5, 6, 7]),\n",
       " ('2jrw', 0.453, [20, 21, 22, 23], [101, 102, 103, 104]),\n",
       " ('2jrw', 0.4665, [7, 8, 9, 10], [28, 29, 30, 31]),\n",
       " ('2jrw', 0.4753, [6, 7, 8, 9], [30, 31, 32, 33]),\n",
       " ('2jrw', 0.4807, [19, 20, 21, 22], [31, 32, 33, 34]),\n",
       " ('2jrw', 0.4839, [3, 4, 5, 6], [27, 28, 29, 30]),\n",
       " ('2jrw', 0.516, [20, 21, 22, 23], [34, 35, 36, 37]),\n",
       " ('2jrw', 0.5227, [16, 17, 18, 19], [31, 32, 33, 34]),\n",
       " ('2jrw', 0.5269, [20, 21, 22, 23], [7, 8, 9, 10]),\n",
       " ('2jrw', 0.5525, [14, 15, 16, 17], [5, 6, 7, 8]),\n",
       " ('2jrw', 0.5527, [18, 19, 20, 21], [27, 28, 29, 30]),\n",
       " ('2jrw', 0.5871, [20, 21, 22, 23], [6, 7, 8, 9]),\n",
       " ('2jrw', 0.5889, [17, 18, 19, 20], [3, 4, 5, 6]),\n",
       " ('2jrw', 0.642, [4, 5, 6, 7], [4, 5, 6, 7]),\n",
       " ('2jrw', 0.6583, [13, 14, 15, 16], [3, 4, 5, 6]),\n",
       " ('2jrw', 0.6696, [20, 21, 22, 23], [100, 101, 102, 103]),\n",
       " ('2lws', 0.2735, [5, 6, 7, 8], [34, 35, 36, 37]),\n",
       " ('2lws', 0.3582, [5, 6, 7, 8], [102, 103, 104, 105]),\n",
       " ('2lws', 0.3729, [5, 6, 7, 8], [32, 33, 34, 35]),\n",
       " ('2lws', 0.3782, [9, 10, 11, 12], [5, 6, 7, 8]),\n",
       " ('2lws', 0.3989, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('2lws', 0.4096, [5, 6, 7, 8], [100, 101, 102, 103]),\n",
       " ('2lws', 0.4344, [5, 6, 7, 8], [33, 34, 35, 36]),\n",
       " ('2lws', 0.5531, [5, 6, 7, 8], [6, 7, 8, 9]),\n",
       " ('2lws', 0.6639, [4, 5, 6, 7], [6, 7, 8, 9]),\n",
       " ('2lws', 0.6801, [4, 5, 6, 7], [33, 34, 35, 36]),\n",
       " ('2lwt', 0.1696, [5, 6, 7, 8], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.2318, [15, 16, 17, 18], [30, 31, 32, 33]),\n",
       " ('2lwt', 0.234, [3, 4, 5, 6], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.3296, [6, 7, 8, 9], [30, 31, 32, 33]),\n",
       " ('2lwt', 0.3658, [4, 5, 6, 7], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.4006, [14, 15, 16, 17], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.4213, [7, 8, 9, 10], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.4487, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('2lwt', 0.4635, [6, 7, 8, 9], [27, 28, 29, 30]),\n",
       " ('2lwt', 0.4762, [1, 2, 3, 4], [3, 4, 5, 6]),\n",
       " ('2lwt', 0.5343, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('2lwt', 0.5521, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('2lwt', 0.5744, [8, 9, 10, 11], [5, 6, 7, 8]),\n",
       " ('2lwt', 0.5784, [11, 12, 13, 14], [30, 31, 32, 33]),\n",
       " ('2lwt', 0.5817, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('2lwt', 0.5924, [13, 14, 15, 16], [4, 5, 6, 7]),\n",
       " ('2lwt', 0.6259, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('2lwt', 0.6342, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('2lwt', 0.6692, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('2lwt', 0.6726, [11, 12, 13, 14], [27, 28, 29, 30]),\n",
       " ('2lwt', 0.6909, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('2lwt', 0.6945, [8, 9, 10, 11], [30, 31, 32, 33]),\n",
       " ('2lwu', 0.2377, [4, 5, 6, 7], [4, 5, 6, 7]),\n",
       " ('2lwu', 0.2812, [10, 11, 12, 13], [30, 31, 32, 33]),\n",
       " ('2lwu', 0.3827, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('2lwu', 0.3951, [12, 13, 14, 15], [31, 32, 33, 34]),\n",
       " ('2lwu', 0.4466, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('2lwu', 0.4515, [6, 7, 8, 9], [33, 34, 35, 36]),\n",
       " ('2lwu', 0.4644, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('2lwu', 0.5086, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('2lwu', 0.5222, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('2lwu', 0.5499, [6, 7, 8, 9], [100, 101, 102, 103]),\n",
       " ('2lwu', 0.5821, [8, 9, 10, 11], [5, 6, 7, 8]),\n",
       " ('2lwu', 0.637, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('2lwu', 0.6421, [7, 8, 9, 10], [29, 30, 31, 32]),\n",
       " ('2lwu', 0.6676, [9, 10, 11, 12], [4, 5, 6, 7]),\n",
       " ('2lwu', 0.69, [1, 2, 3, 4], [27, 28, 29, 30]),\n",
       " ('2lwu', 0.6956, [6, 7, 8, 9], [6, 7, 8, 9]),\n",
       " ('2lwv', 0.22, [4, 5, 6, 7], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.3048, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.3057, [14, 15, 16, 17], [5, 6, 7, 8]),\n",
       " ('2lwv', 0.3409, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.3693, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('2lwv', 0.4254, [9, 10, 11, 12], [3, 4, 5, 6]),\n",
       " ('2lwv', 0.4564, [3, 4, 5, 6], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.4669, [13, 14, 15, 16], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.4911, [11, 12, 13, 14], [30, 31, 32, 33]),\n",
       " ('2lwv', 0.5432, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('2lwv', 0.5556, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('2lwv', 0.5877, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('2lwv', 0.6624, [8, 9, 10, 11], [7, 8, 9, 10]),\n",
       " ('2lwv', 0.6635, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('2lwv', 0.67, [14, 15, 16, 17], [31, 32, 33, 34]),\n",
       " ('2m2g', 0.0999, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('2m2g', 0.1071, [14, 15, 16, 17], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.1669, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('2m2g', 0.256, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('2m2g', 0.3164, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('2m2g', 0.3263, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('2m2g', 0.3493, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('2m2g', 0.3631, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('2m2g', 0.4205, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('2m2g', 0.4229, [12, 13, 14, 15], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.4521, [9, 10, 11, 12], [3, 4, 5, 6]),\n",
       " ('2m2g', 0.5248, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.576, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.5832, [6, 7, 8, 9], [4, 5, 6, 7]),\n",
       " ('2m2g', 0.6577, [10, 11, 12, 13], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.6936, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('2m2g', 0.698, [1, 2, 3, 4], [27, 28, 29, 30]),\n",
       " ('2mgo', 0.291, [2, 3, 4, 5], [102, 103, 104, 105]),\n",
       " ('2mgo', 0.2923, [6, 7, 8, 9], [31, 32, 33, 34]),\n",
       " ('2mgo', 0.3577, [1, 2, 3, 4], [5, 6, 7, 8]),\n",
       " ('2mgo', 0.4493, [2, 3, 4, 5], [33, 34, 35, 36]),\n",
       " ('2mgo', 0.474, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('2mgo', 0.4928, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('2mgo', 0.5618, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('2mgo', 0.5978, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('2mgo', 0.6877, [4, 5, 6, 7], [30, 31, 32, 33]),\n",
       " ('2n07', 0.0904, [7, 8, 9, 10], [101, 102, 103, 104]),\n",
       " ('2n07', 0.1343, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('2n07', 0.1809, [7, 8, 9, 10], [32, 33, 34, 35]),\n",
       " ('2n07', 0.1893, [7, 8, 9, 10], [100, 101, 102, 103]),\n",
       " ('2n07', 0.2201, [11, 12, 13, 14], [31, 32, 33, 34]),\n",
       " ('2n07', 0.224, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('2n07', 0.2267, [5, 6, 7, 8], [34, 35, 36, 37]),\n",
       " ('2n07', 0.2282, [5, 6, 7, 8], [102, 103, 104, 105]),\n",
       " ('2n07', 0.2324, [13, 14, 15, 16], [34, 35, 36, 37]),\n",
       " ('2n07', 0.2364, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('2n07', 0.2372, [12, 13, 14, 15], [34, 35, 36, 37]),\n",
       " ('2n07', 0.244, [14, 15, 16, 17], [102, 103, 104, 105]),\n",
       " ('2n07', 0.2577, [14, 15, 16, 17], [100, 101, 102, 103]),\n",
       " ('2n07', 0.2702, [5, 6, 7, 8], [100, 101, 102, 103]),\n",
       " ('2n07', 0.2708, [7, 8, 9, 10], [6, 7, 8, 9]),\n",
       " ('2n07', 0.2712, [14, 15, 16, 17], [6, 7, 8, 9]),\n",
       " ('2n07', 0.2767, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('2n07', 0.2779, [7, 8, 9, 10], [33, 34, 35, 36]),\n",
       " ('2n07', 0.2788, [13, 14, 15, 16], [101, 102, 103, 104]),\n",
       " ('2n07', 0.284, [13, 14, 15, 16], [99, 100, 101, 102]),\n",
       " ('2n07', 0.2915, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('2n07', 0.2932, [5, 6, 7, 8], [32, 33, 34, 35]),\n",
       " ('2n07', 0.2957, [12, 13, 14, 15], [101, 102, 103, 104]),\n",
       " ('2n07', 0.303, [6, 7, 8, 9], [99, 100, 101, 102]),\n",
       " ('2n07', 0.307, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('2n07', 0.3138, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('2n07', 0.3149, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('2n07', 0.3286, [14, 15, 16, 17], [32, 33, 34, 35]),\n",
       " ('2n07', 0.3305, [19, 20, 21, 22], [7, 8, 9, 10]),\n",
       " ('2n07', 0.3324, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('2n07', 0.3404, [7, 8, 9, 10], [34, 35, 36, 37]),\n",
       " ('2n07', 0.3506, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('2n07', 0.3591, [13, 14, 15, 16], [6, 7, 8, 9]),\n",
       " ('2n07', 0.3642, [5, 6, 7, 8], [6, 7, 8, 9]),\n",
       " ('2n07', 0.3661, [14, 15, 16, 17], [33, 34, 35, 36]),\n",
       " ('2n07', 0.368, [5, 6, 7, 8], [33, 34, 35, 36]),\n",
       " ('2n07', 0.3756, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('2n07', 0.4033, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('2n07', 0.4036, [13, 14, 15, 16], [100, 101, 102, 103]),\n",
       " ('2n07', 0.4105, [6, 7, 8, 9], [101, 102, 103, 104]),\n",
       " ('2n07', 0.414, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('2n07', 0.4371, [12, 13, 14, 15], [99, 100, 101, 102]),\n",
       " ('2n07', 0.4397, [14, 15, 16, 17], [34, 35, 36, 37]),\n",
       " ('2n07', 0.4412, [7, 8, 9, 10], [102, 103, 104, 105]),\n",
       " ('2n07', 0.4419, [7, 8, 9, 10], [99, 100, 101, 102]),\n",
       " ('2n07', 0.4492, [13, 14, 15, 16], [32, 33, 34, 35]),\n",
       " ('2n07', 0.4591, [14, 15, 16, 17], [101, 102, 103, 104]),\n",
       " ('2n07', 0.4634, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('2n07', 0.4802, [14, 15, 16, 17], [99, 100, 101, 102]),\n",
       " ('2n07', 0.5025, [2, 3, 4, 5], [7, 8, 9, 10]),\n",
       " ('2n07', 0.5072, [12, 13, 14, 15], [6, 7, 8, 9]),\n",
       " ('2n07', 0.5076, [12, 13, 14, 15], [100, 101, 102, 103]),\n",
       " ('2n07', 0.5087, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('2n07', 0.5225, [12, 13, 14, 15], [32, 33, 34, 35]),\n",
       " ('2n07', 0.5226, [5, 6, 7, 8], [99, 100, 101, 102]),\n",
       " ('2n07', 0.523, [6, 7, 8, 9], [6, 7, 8, 9]),\n",
       " ('2n07', 0.5409, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('2n07', 0.543, [15, 16, 17, 18], [31, 32, 33, 34]),\n",
       " ('2n07', 0.5469, [13, 14, 15, 16], [102, 103, 104, 105]),\n",
       " ('2n07', 0.5493, [13, 14, 15, 16], [33, 34, 35, 36]),\n",
       " ('2n07', 0.5773, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('2n07', 0.5799, [18, 19, 20, 21], [5, 6, 7, 8]),\n",
       " ('2n07', 0.5954, [6, 7, 8, 9], [7, 8, 9, 10]),\n",
       " ('2n07', 0.5987, [19, 20, 21, 22], [101, 102, 103, 104]),\n",
       " ('2n07', 0.6092, [16, 17, 18, 19], [34, 35, 36, 37]),\n",
       " ('2n07', 0.6139, [3, 4, 5, 6], [3, 4, 5, 6]),\n",
       " ('2n07', 0.6159, [6, 7, 8, 9], [100, 101, 102, 103]),\n",
       " ('2n07', 0.6184, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('2n07', 0.6236, [17, 18, 19, 20], [27, 28, 29, 30]),\n",
       " ('2n07', 0.6245, [12, 13, 14, 15], [33, 34, 35, 36]),\n",
       " ('2n07', 0.646, [12, 13, 14, 15], [102, 103, 104, 105]),\n",
       " ('2n07', 0.654, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('2n07', 0.6589, [12, 13, 14, 15], [7, 8, 9, 10]),\n",
       " ('2n07', 0.6687, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('2n07', 0.6718, [19, 20, 21, 22], [34, 35, 36, 37]),\n",
       " ('2ox2', 0.4928, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('2ox2', 0.6605, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('2ox2', 0.6809, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('3av9', 0.1241, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('3av9', 0.2166, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('3av9', 0.3005, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('3av9', 0.3446, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('3av9', 0.3643, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('3av9', 0.3781, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('3av9', 0.4126, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('3av9', 0.5274, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('3av9', 0.5442, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('3av9', 0.5868, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('3av9', 0.6855, [4, 5, 6, 7], [34, 35, 36, 37]),\n",
       " ('3ava', 0.0587, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('3ava', 0.3019, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('3ava', 0.3026, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('3ava', 0.3058, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('3ava', 0.307, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('3ava', 0.361, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('3ava', 0.3662, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('3ava', 0.4532, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('3ava', 0.4577, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('3ava', 0.505, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('3avb', 0.1846, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('3avb', 0.2299, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('3avb', 0.2976, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('3avb', 0.2986, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('3avb', 0.3415, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('3avb', 0.3605, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('3avb', 0.4052, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('3avb', 0.5126, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('3avb', 0.5892, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('3avb', 0.5956, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('3avb', 0.6956, [4, 5, 6, 7], [34, 35, 36, 37]),\n",
       " ('3avi', 0.1982, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('3avi', 0.248, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('3avi', 0.2899, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('3avi', 0.3236, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('3avi', 0.4435, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('3avi', 0.4912, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('3avi', 0.4973, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('3avi', 0.5366, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('3avi', 0.5616, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('3avi', 0.5718, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('3avi', 0.6295, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('3p8f', 0.141, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('3p8f', 0.2592, [11, 12, 13, 14], [5, 6, 7, 8]),\n",
       " ('3p8f', 0.3154, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('3p8f', 0.3967, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('3p8f', 0.4136, [3, 4, 5, 6], [31, 32, 33, 34]),\n",
       " ('3p8f', 0.4292, [8, 9, 10, 11], [30, 31, 32, 33]),\n",
       " ('3p8f', 0.4932, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('3p8f', 0.5584, [4, 5, 6, 7], [29, 30, 31, 32]),\n",
       " ('3p8f', 0.6025, [6, 7, 8, 9], [33, 34, 35, 36]),\n",
       " ('3p8f', 0.644, [7, 8, 9, 10], [3, 4, 5, 6]),\n",
       " ('3p8f', 0.6771, [8, 9, 10, 11], [27, 28, 29, 30]),\n",
       " ('3p8f', 0.6812, [3, 4, 5, 6], [5, 6, 7, 8]),\n",
       " ('3p8f', 0.685, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('3wbn', 0.1708, [12, 13, 14, 15], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.1901, [6, 7, 8, 9], [31, 32, 33, 34]),\n",
       " ('3wbn', 0.1988, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('3wbn', 0.2023, [7, 8, 9, 10], [34, 35, 36, 37]),\n",
       " ('3wbn', 0.2194, [7, 8, 9, 10], [101, 102, 103, 104]),\n",
       " ('3wbn', 0.2329, [7, 8, 9, 10], [100, 101, 102, 103]),\n",
       " ('3wbn', 0.2373, [7, 8, 9, 10], [32, 33, 34, 35]),\n",
       " ('3wbn', 0.2439, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('3wbn', 0.295, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('3wbn', 0.3035, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('3wbn', 0.3251, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('3wbn', 0.3325, [7, 8, 9, 10], [102, 103, 104, 105]),\n",
       " ('3wbn', 0.3326, [7, 8, 9, 10], [33, 34, 35, 36]),\n",
       " ('3wbn', 0.333, [7, 8, 9, 10], [6, 7, 8, 9]),\n",
       " ('3wbn', 0.3366, [14, 15, 16, 17], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.3996, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('3wbn', 0.4174, [4, 5, 6, 7], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.4188, [15, 16, 17, 18], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.4678, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('3wbn', 0.4731, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('3wbn', 0.4816, [7, 8, 9, 10], [99, 100, 101, 102]),\n",
       " ('3wbn', 0.4869, [11, 12, 13, 14], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.5125, [3, 4, 5, 6], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.5353, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('3wbn', 0.5526, [10, 11, 12, 13], [29, 30, 31, 32]),\n",
       " ('3wbn', 0.5553, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('3wbn', 0.6734, [13, 14, 15, 16], [4, 5, 6, 7]),\n",
       " ('3wbn', 0.6842, [13, 14, 15, 16], [30, 31, 32, 33]),\n",
       " ('3wne', 0.2513, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('3wne', 0.3242, [1, 2, 3, 4], [5, 6, 7, 8]),\n",
       " ('3wne', 0.4488, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('3wne', 0.4543, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('3wne', 0.5283, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('3wne', 0.5697, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('3wne', 0.6426, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('3wne', 0.6677, [2, 3, 4, 5], [102, 103, 104, 105]),\n",
       " ('3wnf', 0.2073, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('3wnf', 0.5067, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('3wnf', 0.51, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('3wnf', 0.5168, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('3wnf', 0.5398, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('3wnf', 0.6284, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('3wnf', 0.6545, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('3wnf', 0.68, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('3zwz', 0.3822, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('3zwz', 0.5356, [3, 4, 5, 6], [31, 32, 33, 34]),\n",
       " ('3zwz', 0.5534, [2, 3, 4, 5], [27, 28, 29, 30]),\n",
       " ('3zwz', 0.5816, [9, 10, 11, 12], [30, 31, 32, 33]),\n",
       " ('3zwz', 0.6159, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('3zwz', 0.6532, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('3zwz', 0.6636, [5, 6, 7, 8], [27, 28, 29, 30]),\n",
       " ('3zwz', 0.6726, [9, 10, 11, 12], [4, 5, 6, 7]),\n",
       " ('4gly', 0.1242, [7, 8, 9, 10], [28, 29, 30, 31]),\n",
       " ('4gly', 0.2316, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('4gly', 0.2909, [8, 9, 10, 11], [29, 30, 31, 32]),\n",
       " ('4gly', 0.4582, [2, 3, 4, 5], [27, 28, 29, 30]),\n",
       " ('4gly', 0.678, [9, 10, 11, 12], [30, 31, 32, 33]),\n",
       " ('4gly', 0.6938, [5, 6, 7, 8], [3, 4, 5, 6]),\n",
       " ('4ib5', 0.1869, [4, 5, 6, 7], [102, 103, 104, 105]),\n",
       " ('4ib5', 0.2276, [4, 5, 6, 7], [32, 33, 34, 35]),\n",
       " ('4ib5', 0.26, [4, 5, 6, 7], [33, 34, 35, 36]),\n",
       " ('4ib5', 0.2618, [4, 5, 6, 7], [100, 101, 102, 103]),\n",
       " ('4ib5', 0.3672, [10, 11, 12, 13], [29, 30, 31, 32]),\n",
       " ('4ib5', 0.3744, [4, 5, 6, 7], [34, 35, 36, 37]),\n",
       " ('4ib5', 0.3842, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('4ib5', 0.3952, [4, 5, 6, 7], [101, 102, 103, 104]),\n",
       " ('4ib5', 0.3989, [3, 4, 5, 6], [28, 29, 30, 31]),\n",
       " ('4ib5', 0.4318, [4, 5, 6, 7], [6, 7, 8, 9]),\n",
       " ('4ib5', 0.5084, [6, 7, 8, 9], [30, 31, 32, 33]),\n",
       " ('4ib5', 0.5314, [1, 2, 3, 4], [3, 4, 5, 6]),\n",
       " ('4ib5', 0.5498, [7, 8, 9, 10], [27, 28, 29, 30]),\n",
       " ('4ib5', 0.5932, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('4ib5', 0.613, [6, 7, 8, 9], [27, 28, 29, 30]),\n",
       " ('4ib5', 0.6148, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('4ib5', 0.6649, [4, 5, 6, 7], [99, 100, 101, 102]),\n",
       " ('4k8y', 0.3956, [3, 4, 5, 6], [31, 32, 33, 34]),\n",
       " ('4k8y', 0.4186, [6, 7, 8, 9], [33, 34, 35, 36]),\n",
       " ('4k8y', 0.4634, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('4k8y', 0.4653, [4, 5, 6, 7], [29, 30, 31, 32]),\n",
       " ('4k8y', 0.5112, [7, 8, 9, 10], [3, 4, 5, 6]),\n",
       " ('4k8y', 0.5182, [6, 7, 8, 9], [32, 33, 34, 35]),\n",
       " ('4k8y', 0.5663, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('4k8y', 0.5723, [6, 7, 8, 9], [100, 101, 102, 103]),\n",
       " ('4k8y', 0.6185, [3, 4, 5, 6], [5, 6, 7, 8]),\n",
       " ('4k8y', 0.6427, [6, 7, 8, 9], [102, 103, 104, 105]),\n",
       " ('4k8y', 0.6935, [8, 9, 10, 11], [30, 31, 32, 33]),\n",
       " ('4k8y', 0.6939, [8, 9, 10, 11], [27, 28, 29, 30]),\n",
       " ('4kts', 0.1152, [11, 12, 13, 14], [34, 35, 36, 37]),\n",
       " ('4kts', 0.2002, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('4kts', 0.2057, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('4kts', 0.2089, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('4kts', 0.2995, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('4kts', 0.3553, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('4kts', 0.3667, [11, 12, 13, 14], [101, 102, 103, 104]),\n",
       " ('4kts', 0.3872, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('4kts', 0.4214, [8, 9, 10, 11], [5, 6, 7, 8]),\n",
       " ('4kts', 0.4368, [10, 11, 12, 13], [7, 8, 9, 10]),\n",
       " ('4kts', 0.4372, [10, 11, 12, 13], [101, 102, 103, 104]),\n",
       " ('4kts', 0.443, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('4kts', 0.5037, [11, 12, 13, 14], [99, 100, 101, 102]),\n",
       " ('4kts', 0.5086, [11, 12, 13, 14], [100, 101, 102, 103]),\n",
       " ('4kts', 0.5171, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('4kts', 0.5243, [11, 12, 13, 14], [32, 33, 34, 35]),\n",
       " ('4kts', 0.5345, [11, 12, 13, 14], [6, 7, 8, 9]),\n",
       " ('4kts', 0.5456, [10, 11, 12, 13], [99, 100, 101, 102]),\n",
       " ('4kts', 0.5512, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('4kts', 0.5614, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('4kts', 0.5617, [11, 12, 13, 14], [102, 103, 104, 105]),\n",
       " ('4kts', 0.6124, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('4kts', 0.6234, [11, 12, 13, 14], [33, 34, 35, 36]),\n",
       " ('4kts', 0.6643, [4, 5, 6, 7], [29, 30, 31, 32]),\n",
       " ('4kts', 0.6752, [10, 11, 12, 13], [6, 7, 8, 9]),\n",
       " ('4kts', 0.6847, [3, 4, 5, 6], [27, 28, 29, 30]),\n",
       " ('4kts', 0.6868, [4, 5, 6, 7], [27, 28, 29, 30]),\n",
       " ('4ktu', 0.2244, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('4ktu', 0.2385, [8, 9, 10, 11], [5, 6, 7, 8]),\n",
       " ('4ktu', 0.2422, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('4ktu', 0.2654, [10, 11, 12, 13], [7, 8, 9, 10]),\n",
       " ('4ktu', 0.2686, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('4ktu', 0.2856, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('4ktu', 0.3042, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('4ktu', 0.311, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('4ktu', 0.3689, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('4ktu', 0.3847, [9, 10, 11, 12], [99, 100, 101, 102]),\n",
       " ('4ktu', 0.6304, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('4ktu', 0.6391, [5, 6, 7, 8], [5, 6, 7, 8]),\n",
       " ('4ktu', 0.6547, [3, 4, 5, 6], [27, 28, 29, 30]),\n",
       " ('4ktu', 0.6774, [4, 5, 6, 7], [27, 28, 29, 30]),\n",
       " ('4ktu', 0.6818, [4, 5, 6, 7], [29, 30, 31, 32]),\n",
       " ('4mq9', 0.3839, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('4mq9', 0.5844, [2, 3, 4, 5], [28, 29, 30, 31]),\n",
       " ('4mq9', 0.6097, [2, 3, 4, 5], [99, 100, 101, 102]),\n",
       " ('5bxo', 0.4995, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('5bxo', 0.5268, [2, 3, 4, 5], [27, 28, 29, 30]),\n",
       " ('5bxo', 0.5894, [6, 7, 8, 9], [4, 5, 6, 7]),\n",
       " ('5bxo', 0.6148, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('5bxo', 0.6688, [3, 4, 5, 6], [27, 28, 29, 30]),\n",
       " ('5bxo', 0.6997, [4, 5, 6, 7], [27, 28, 29, 30]),\n",
       " ('5eoc', 0.3252, [7, 8, 9, 10], [29, 30, 31, 32]),\n",
       " ('5eoc', 0.3537, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('5eoc', 0.3922, [4, 5, 6, 7], [31, 32, 33, 34]),\n",
       " ('5eoc', 0.4081, [9, 10, 11, 12], [30, 31, 32, 33]),\n",
       " ('5eoc', 0.4696, [6, 7, 8, 9], [99, 100, 101, 102]),\n",
       " ('5eoc', 0.5068, [5, 6, 7, 8], [7, 8, 9, 10]),\n",
       " ('5eoc', 0.5529, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('5eoc', 0.5694, [6, 7, 8, 9], [34, 35, 36, 37]),\n",
       " ('5eoc', 0.5873, [6, 7, 8, 9], [7, 8, 9, 10]),\n",
       " ('5eoc', 0.6589, [6, 7, 8, 9], [101, 102, 103, 104]),\n",
       " ('5eoc', 0.6937, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('5glh', 0.1085, [10, 11, 12, 13], [100, 101, 102, 103]),\n",
       " ('5glh', 0.1273, [10, 11, 12, 13], [32, 33, 34, 35]),\n",
       " ('5glh', 0.1546, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('5glh', 0.1872, [10, 11, 12, 13], [101, 102, 103, 104]),\n",
       " ('5glh', 0.1876, [10, 11, 12, 13], [6, 7, 8, 9]),\n",
       " ('5glh', 0.2068, [14, 15, 16, 17], [101, 102, 103, 104]),\n",
       " ('5glh', 0.207, [10, 11, 12, 13], [33, 34, 35, 36]),\n",
       " ('5glh', 0.2184, [12, 13, 14, 15], [32, 33, 34, 35]),\n",
       " ('5glh', 0.2274, [11, 12, 13, 14], [33, 34, 35, 36]),\n",
       " ('5glh', 0.2562, [12, 13, 14, 15], [33, 34, 35, 36]),\n",
       " ('5glh', 0.2685, [9, 10, 11, 12], [33, 34, 35, 36]),\n",
       " ('5glh', 0.2776, [13, 14, 15, 16], [33, 34, 35, 36]),\n",
       " ('5glh', 0.2803, [9, 10, 11, 12], [102, 103, 104, 105]),\n",
       " ('5glh', 0.2973, [12, 13, 14, 15], [100, 101, 102, 103]),\n",
       " ('5glh', 0.2997, [17, 18, 19, 20], [27, 28, 29, 30]),\n",
       " ('5glh', 0.3087, [12, 13, 14, 15], [101, 102, 103, 104]),\n",
       " ('5glh', 0.325, [11, 12, 13, 14], [32, 33, 34, 35]),\n",
       " ('5glh', 0.33, [5, 6, 7, 8], [7, 8, 9, 10]),\n",
       " ('5glh', 0.3375, [9, 10, 11, 12], [32, 33, 34, 35]),\n",
       " ('5glh', 0.3588, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('5glh', 0.3738, [9, 10, 11, 12], [100, 101, 102, 103]),\n",
       " ('5glh', 0.3757, [11, 12, 13, 14], [100, 101, 102, 103]),\n",
       " ('5glh', 0.3799, [13, 14, 15, 16], [32, 33, 34, 35]),\n",
       " ('5glh', 0.386, [14, 15, 16, 17], [32, 33, 34, 35]),\n",
       " ('5glh', 0.393, [14, 15, 16, 17], [34, 35, 36, 37]),\n",
       " ('5glh', 0.4034, [12, 13, 14, 15], [102, 103, 104, 105]),\n",
       " ('5glh', 0.4059, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('5glh', 0.4132, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('5glh', 0.4134, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('5glh', 0.4139, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('5glh', 0.4156, [13, 14, 15, 16], [100, 101, 102, 103]),\n",
       " ('5glh', 0.4222, [10, 11, 12, 13], [102, 103, 104, 105]),\n",
       " ('5glh', 0.4232, [12, 13, 14, 15], [34, 35, 36, 37]),\n",
       " ('5glh', 0.4265, [14, 15, 16, 17], [100, 101, 102, 103]),\n",
       " ('5glh', 0.4271, [7, 8, 9, 10], [31, 32, 33, 34]),\n",
       " ('5glh', 0.4276, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('5glh', 0.4282, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('5glh', 0.4326, [10, 11, 12, 13], [99, 100, 101, 102]),\n",
       " ('5glh', 0.4634, [14, 15, 16, 17], [33, 34, 35, 36]),\n",
       " ('5glh', 0.4666, [12, 13, 14, 15], [6, 7, 8, 9]),\n",
       " ('5glh', 0.4688, [4, 5, 6, 7], [5, 6, 7, 8]),\n",
       " ('5glh', 0.471, [15, 16, 17, 18], [102, 103, 104, 105]),\n",
       " ('5glh', 0.4993, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('5glh', 0.4995, [17, 18, 19, 20], [30, 31, 32, 33]),\n",
       " ('5glh', 0.501, [13, 14, 15, 16], [102, 103, 104, 105]),\n",
       " ('5glh', 0.5031, [11, 12, 13, 14], [6, 7, 8, 9]),\n",
       " ('5glh', 0.5078, [14, 15, 16, 17], [6, 7, 8, 9]),\n",
       " ('5glh', 0.5089, [18, 19, 20, 21], [5, 6, 7, 8]),\n",
       " ('5glh', 0.5138, [11, 12, 13, 14], [102, 103, 104, 105]),\n",
       " ('5glh', 0.5165, [6, 7, 8, 9], [7, 8, 9, 10]),\n",
       " ('5glh', 0.5275, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('5glh', 0.5344, [9, 10, 11, 12], [6, 7, 8, 9]),\n",
       " ('5glh', 0.5346, [13, 14, 15, 16], [6, 7, 8, 9]),\n",
       " ('5glh', 0.555, [11, 12, 13, 14], [101, 102, 103, 104]),\n",
       " ('5glh', 0.5702, [18, 19, 20, 21], [31, 32, 33, 34]),\n",
       " ('5glh', 0.5844, [15, 16, 17, 18], [33, 34, 35, 36]),\n",
       " ('5glh', 0.5951, [14, 15, 16, 17], [99, 100, 101, 102]),\n",
       " ('5glh', 0.599, [9, 10, 11, 12], [101, 102, 103, 104]),\n",
       " ('5glh', 0.6198, [14, 15, 16, 17], [102, 103, 104, 105]),\n",
       " ('5glh', 0.6307, [13, 14, 15, 16], [101, 102, 103, 104]),\n",
       " ('5glh', 0.6529, [15, 16, 17, 18], [32, 33, 34, 35]),\n",
       " ('5glh', 0.6539, [9, 10, 11, 12], [34, 35, 36, 37]),\n",
       " ('5glh', 0.6644, [15, 16, 17, 18], [100, 101, 102, 103]),\n",
       " ('5glh', 0.6739, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('5glh', 0.6749, [5, 6, 7, 8], [101, 102, 103, 104]),\n",
       " ('5glh', 0.6832, [16, 17, 18, 19], [3, 4, 5, 6]),\n",
       " ('5glh', 0.6835, [12, 13, 14, 15], [99, 100, 101, 102]),\n",
       " ('5glh', 0.6991, [16, 17, 18, 19], [29, 30, 31, 32]),\n",
       " ('5lff', 0.3774, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('5lff', 0.5711, [1, 2, 3, 4], [3, 4, 5, 6]),\n",
       " ('5lso', 0.4439, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('5lso', 0.5911, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('5lso', 0.6948, [2, 3, 4, 5], [3, 4, 5, 6]),\n",
       " ('5vav', 0.2679, [1, 2, 3, 4], [4, 5, 6, 7]),\n",
       " ('5vav', 0.3503, [10, 11, 12, 13], [4, 5, 6, 7]),\n",
       " ('5vav', 0.4227, [11, 12, 13, 14], [5, 6, 7, 8]),\n",
       " ('5vav', 0.446, [5, 6, 7, 8], [30, 31, 32, 33]),\n",
       " ('5vav', 0.4723, [5, 6, 7, 8], [27, 28, 29, 30]),\n",
       " ('5vav', 0.4958, [2, 3, 4, 5], [30, 31, 32, 33]),\n",
       " ('5vav', 0.499, [3, 4, 5, 6], [31, 32, 33, 34]),\n",
       " ('5vav', 0.511, [9, 10, 11, 12], [4, 5, 6, 7]),\n",
       " ('5vav', 0.5781, [11, 12, 13, 14], [31, 32, 33, 34]),\n",
       " ('5xco', 0.0629, [14, 15, 16, 17], [34, 35, 36, 37]),\n",
       " ('5xco', 0.1462, [13, 14, 15, 16], [101, 102, 103, 104]),\n",
       " ('5xco', 0.2122, [12, 13, 14, 15], [31, 32, 33, 34]),\n",
       " ('5xco', 0.2518, [10, 11, 12, 13], [99, 100, 101, 102]),\n",
       " ('5xco', 0.2843, [10, 11, 12, 13], [34, 35, 36, 37]),\n",
       " ('5xco', 0.3037, [3, 4, 5, 6], [101, 102, 103, 104]),\n",
       " ('5xco', 0.3114, [14, 15, 16, 17], [101, 102, 103, 104]),\n",
       " ('5xco', 0.3115, [13, 14, 15, 16], [34, 35, 36, 37]),\n",
       " ('5xco', 0.3155, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('5xco', 0.3185, [5, 6, 7, 8], [31, 32, 33, 34]),\n",
       " ('5xco', 0.3492, [15, 16, 17, 18], [7, 8, 9, 10]),\n",
       " ('5xco', 0.3544, [10, 11, 12, 13], [101, 102, 103, 104]),\n",
       " ('5xco', 0.36, [9, 10, 11, 12], [31, 32, 33, 34]),\n",
       " ('5xco', 0.379, [13, 14, 15, 16], [99, 100, 101, 102]),\n",
       " ('5xco', 0.3826, [10, 11, 12, 13], [6, 7, 8, 9]),\n",
       " ('5xco', 0.3937, [13, 14, 15, 16], [6, 7, 8, 9]),\n",
       " ('5xco', 0.3977, [13, 14, 15, 16], [100, 101, 102, 103]),\n",
       " ('5xco', 0.4003, [14, 15, 16, 17], [100, 101, 102, 103]),\n",
       " ('5xco', 0.4085, [13, 14, 15, 16], [32, 33, 34, 35]),\n",
       " ('5xco', 0.4181, [16, 17, 18, 19], [3, 4, 5, 6]),\n",
       " ('5xco', 0.4255, [14, 15, 16, 17], [32, 33, 34, 35]),\n",
       " ('5xco', 0.4256, [3, 4, 5, 6], [32, 33, 34, 35]),\n",
       " ('5xco', 0.4293, [14, 15, 16, 17], [6, 7, 8, 9]),\n",
       " ('5xco', 0.4456, [14, 15, 16, 17], [99, 100, 101, 102]),\n",
       " ('5xco', 0.451, [4, 5, 6, 7], [34, 35, 36, 37]),\n",
       " ('5xco', 0.4529, [14, 15, 16, 17], [102, 103, 104, 105]),\n",
       " ('5xco', 0.4535, [10, 11, 12, 13], [100, 101, 102, 103]),\n",
       " ('5xco', 0.4686, [3, 4, 5, 6], [100, 101, 102, 103]),\n",
       " ('5xco', 0.4737, [7, 8, 9, 10], [27, 28, 29, 30]),\n",
       " ('5xco', 0.4839, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('5xco', 0.5032, [3, 4, 5, 6], [33, 34, 35, 36]),\n",
       " ('5xco', 0.5064, [13, 14, 15, 16], [33, 34, 35, 36]),\n",
       " ('5xco', 0.5092, [10, 11, 12, 13], [32, 33, 34, 35]),\n",
       " ('5xco', 0.5232, [14, 15, 16, 17], [33, 34, 35, 36]),\n",
       " ('5xco', 0.5468, [17, 18, 19, 20], [4, 5, 6, 7]),\n",
       " ('5xco', 0.5555, [7, 8, 9, 10], [30, 31, 32, 33]),\n",
       " ('5xco', 0.5677, [3, 4, 5, 6], [102, 103, 104, 105]),\n",
       " ('5xco', 0.577, [3, 4, 5, 6], [6, 7, 8, 9]),\n",
       " ('5xco', 0.5812, [10, 11, 12, 13], [102, 103, 104, 105]),\n",
       " ('5xco', 0.6061, [10, 11, 12, 13], [33, 34, 35, 36]),\n",
       " ('5xco', 0.6131, [13, 14, 15, 16], [102, 103, 104, 105]),\n",
       " ('5xco', 0.6293, [4, 5, 6, 7], [7, 8, 9, 10]),\n",
       " ('5xco', 0.6478, [4, 5, 6, 7], [99, 100, 101, 102]),\n",
       " ('5xco', 0.6634, [15, 16, 17, 18], [99, 100, 101, 102]),\n",
       " ('5xco', 0.6692, [3, 4, 5, 6], [99, 100, 101, 102]),\n",
       " ('5xco', 0.6757, [4, 5, 6, 7], [101, 102, 103, 104]),\n",
       " ('5xn3', 0.387, [1, 2, 3, 4], [29, 30, 31, 32]),\n",
       " ('5xn3', 0.5628, [3, 4, 5, 6], [34, 35, 36, 37]),\n",
       " ('5xn3', 0.5907, [2, 3, 4, 5], [31, 32, 33, 34]),\n",
       " ('5xn3', 0.6622, [2, 3, 4, 5], [5, 6, 7, 8]),\n",
       " ('5xn3', 0.6759, [3, 4, 5, 6], [7, 8, 9, 10]),\n",
       " ('6awk', 0.2965, [5, 6, 7, 8], [29, 30, 31, 32]),\n",
       " ('6awk', 0.4685, [4, 5, 6, 7], [28, 29, 30, 31]),\n",
       " ('6awk', 0.5771, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('6awk', 0.6478, [4, 5, 6, 7], [99, 100, 101, 102]),\n",
       " ('6awm', 0.3245, [2, 3, 4, 5], [101, 102, 103, 104]),\n",
       " ('6awm', 0.3886, [2, 3, 4, 5], [34, 35, 36, 37]),\n",
       " ('6awm', 0.45, [2, 3, 4, 5], [32, 33, 34, 35]),\n",
       " ('6awm', 0.5009, [2, 3, 4, 5], [100, 101, 102, 103]),\n",
       " ('6awm', 0.5204, [2, 3, 4, 5], [33, 34, 35, 36]),\n",
       " ('6awm', 0.6106, [2, 3, 4, 5], [6, 7, 8, 9]),\n",
       " ('6awm', 0.6213, [2, 3, 4, 5], [102, 103, 104, 105]),\n",
       " ('6axi', 0.3709, [5, 6, 7, 8], [29, 30, 31, 32]),\n",
       " ('6axi', 0.5782, [1, 2, 3, 4], [30, 31, 32, 33]),\n",
       " ('6pin', 0.1353, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('6pin', 0.1476, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('6pin', 0.1483, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('6pin', 0.1567, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('6pin', 0.2139, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('6pin', 0.2561, [11, 12, 13, 14], [4, 5, 6, 7]),\n",
       " ('6pin', 0.2566, [12, 13, 14, 15], [30, 31, 32, 33]),\n",
       " ('6pin', 0.3091, [14, 15, 16, 17], [30, 31, 32, 33]),\n",
       " ('6pin', 0.3218, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('6pin', 0.3315, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('6pin', 0.4006, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('6pin', 0.4528, [7, 8, 9, 10], [99, 100, 101, 102]),\n",
       " ('6pin', 0.4805, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('6pin', 0.5014, [7, 8, 9, 10], [28, 29, 30, 31]),\n",
       " ('6pin', 0.5229, [13, 14, 15, 16], [4, 5, 6, 7]),\n",
       " ('6pin', 0.6029, [15, 16, 17, 18], [31, 32, 33, 34]),\n",
       " ('6pin', 0.6104, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('6pin', 0.6606, [12, 13, 14, 15], [27, 28, 29, 30]),\n",
       " ('6pio', 0.2156, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('6pio', 0.2492, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('6pio', 0.266, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('6pio', 0.2675, [11, 12, 13, 14], [4, 5, 6, 7]),\n",
       " ('6pio', 0.2836, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('6pio', 0.2937, [7, 8, 9, 10], [28, 29, 30, 31]),\n",
       " ('6pio', 0.308, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('6pio', 0.3106, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('6pio', 0.352, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('6pio', 0.3767, [6, 7, 8, 9], [4, 5, 6, 7]),\n",
       " ('6pio', 0.4129, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('6pio', 0.4246, [14, 15, 16, 17], [4, 5, 6, 7]),\n",
       " ('6pio', 0.4301, [13, 14, 15, 16], [4, 5, 6, 7]),\n",
       " ('6pio', 0.4386, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('6pio', 0.4758, [15, 16, 17, 18], [30, 31, 32, 33]),\n",
       " ('6pio', 0.6001, [12, 13, 14, 15], [30, 31, 32, 33]),\n",
       " ('6pio', 0.6097, [8, 9, 10, 11], [99, 100, 101, 102]),\n",
       " ('6pio', 0.6452, [12, 13, 14, 15], [4, 5, 6, 7]),\n",
       " ('6pip', 0.1198, [8, 9, 10, 11], [33, 34, 35, 36]),\n",
       " ('6pip', 0.1582, [8, 9, 10, 11], [32, 33, 34, 35]),\n",
       " ('6pip', 0.2262, [8, 9, 10, 11], [100, 101, 102, 103]),\n",
       " ('6pip', 0.2494, [11, 12, 13, 14], [4, 5, 6, 7]),\n",
       " ('6pip', 0.2516, [3, 4, 5, 6], [30, 31, 32, 33]),\n",
       " ('6pip', 0.2868, [8, 9, 10, 11], [102, 103, 104, 105]),\n",
       " ('6pip', 0.2901, [7, 8, 9, 10], [28, 29, 30, 31]),\n",
       " ('6pip', 0.4043, [8, 9, 10, 11], [101, 102, 103, 104]),\n",
       " ('6pip', 0.4115, [8, 9, 10, 11], [6, 7, 8, 9]),\n",
       " ('6pip', 0.5087, [2, 3, 4, 5], [4, 5, 6, 7]),\n",
       " ('6pip', 0.5092, [8, 9, 10, 11], [34, 35, 36, 37]),\n",
       " ('6pip', 0.5304, [12, 13, 14, 15], [30, 31, 32, 33]),\n",
       " ('6pip', 0.5619, [3, 4, 5, 6], [27, 28, 29, 30]),\n",
       " ('6pip', 0.6921, [12, 13, 14, 15], [5, 6, 7, 8]),\n",
       " ('6pip', 0.6923, [8, 9, 10, 11], [99, 100, 101, 102])]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cyclo_mtfs = [ ]\n",
    "\n",
    "with open(\"{}{}database_{}.pkl\" .format(my_location,database_location, motif_size),'rb') as database:\n",
    "    cyclo_mtfs = pickle.load(database)\n",
    "    database_length = len(cyclo_mtfs)\n",
    "\n",
    "if FIND_MATCH:\n",
    "    match = dict()\n",
    "    all_match = []\n",
    "    matches = []\n",
    "    d_ppi = []\n",
    "    \n",
    "    for pp in mtfs:\n",
    "        m_count = len(mtfs[pp][\"m\"])\n",
    "        d_ppi = []\n",
    "        for j in range(0, m_count):\n",
    "            d_ppi_single = mtfs[pp][\"m\"][j]\n",
    "            d_ppi.append(d_ppi_single)\n",
    "        for i in range(0,database_length):\n",
    "            for c in cyclo_mtfs[i]:\n",
    "                peptide = c\n",
    "                for d in (cyclo_mtfs[i][c]):\n",
    "                    d_cyclo = []\n",
    "                    for k in range(0, m_count):\n",
    "                        d_cyclo_single = cyclo_mtfs[i][c][d][\"m\"][k]\n",
    "                        d_cyclo.append(d_cyclo_single)\n",
    "                    sum = 0\n",
    "                    for r in range(0, m_count):\n",
    "                        dd = (d_ppi[r]-d_cyclo[r])**2\n",
    "                        sum = sum + dd\n",
    "                    rmsd = np.sqrt(sum)\n",
    "                    if rmsd < frmsd_threshold:\n",
    "                        rmsd = rmsd.round(decimals=4)\n",
    "                        match = ( peptide , rmsd,  cyclo_mtfs[i][c][d][\"resid\"], mtfs[pp][\"resid\"] )\n",
    "                        all_match.append(match) \n",
    "\n",
    "\n",
    "all_match.sort()\n",
    "all_match\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Superimpose & Mutate"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This section superimposes the matched structures. Output files are the superimposed pdb's and a full list of the matches."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "#MODELLER optimization\n",
    "\n",
    "def optimize(atmsel, sched):\n",
    "    #conjugate gradient\n",
    "    for step in sched:\n",
    "        step.optimize(atmsel, max_iterations=200, min_atom_shift=0.001)\n",
    "    #md\n",
    "    refine(atmsel)\n",
    "    cg = ConjugateGradients()\n",
    "    cg.optimize(atmsel, max_iterations=200, min_atom_shift=0.001)\n",
    "\n",
    "\n",
    "#molecular dynamics\n",
    "def refine(atmsel):\n",
    "    # at T=1000, max_atom_shift for 4fs is cca 0.15 A.\n",
    "    md = MolecularDynamics(cap_atom_shift=0.39, md_time_step=4.0,\n",
    "                           md_return='FINAL')\n",
    "    init_vel = True\n",
    "    for (its, equil, temps) in ((200, 20, (150.0, 250.0, 400.0, 700.0, 1000.0)),\n",
    "                                (200, 600,\n",
    "                                 (1000.0, 800.0, 600.0, 500.0, 400.0, 300.0))):\n",
    "        for temp in temps:\n",
    "            md.optimize(atmsel, init_velocities=init_vel, temperature=temp,\n",
    "                         max_iterations=its, equilibrate=equil)\n",
    "            init_vel = False\n",
    "\n",
    "\n",
    "#use homologs and dihedral library for dihedral angle restraints\n",
    "def make_restraints(mdl1, aln):\n",
    "   rsr = mdl1.restraints\n",
    "   rsr.clear()\n",
    "   s = Selection(mdl1)\n",
    "   for typ in ('stereo', 'phi-psi_binormal'):\n",
    "       rsr.make(s, restraint_type=typ, aln=aln, spline_on_site=True)\n",
    "   for typ in ('omega', 'chi1', 'chi2', 'chi3', 'chi4'):\n",
    "       rsr.make(s, restraint_type=typ+'_dihedral', spline_range=4.0,\n",
    "                spline_dx=0.3, spline_min_points = 5, aln=aln,\n",
    "                spline_on_site=True)\n",
    "    \n",
    "    \n",
    "    \n",
    "def overlap_finder(match_path, complex_fname, matched_sequence):\n",
    "    \n",
    "    parser = PDBParser()\n",
    "    ppi = molecule.load(\"pdb\", complex_fname)    \n",
    "    receptor_chain = atomsel(\"resid {}\" .format(target)) #change this if the receptor has a different chain name.\n",
    "    #receptor_chain = ' '.join(str(receptor_chain).split())\n",
    "    \n",
    "    parser = PDBParser()\n",
    "    cyclo = molecule.load(\"pdb\", (match_path + 'cpep_aligned.pdb'))\n",
    "    cyclo_chain = atomsel('chain M and not resid {}'.format(matched_sequence))\n",
    "    #cyclo_chain = ' '.join(str(cyclo_chain).split())\n",
    "    \n",
    "    overlap = atomsel.contacts(cyclo_chain, receptor_chain, 1)\n",
    "    \n",
    "    return overlap         "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Match folder already exist\n",
      "\n",
      "                         MODELLER 10.1, 2021/03/12, r12156\n",
      "\n",
      "     PROTEIN STRUCTURE MODELLING BY SATISFACTION OF SPATIAL RESTRAINTS\n",
      "\n",
      "\n",
      "                     Copyright(c) 1989-2021 Andrej Sali\n",
      "                            All Rights Reserved\n",
      "\n",
      "                             Written by A. Sali\n",
      "                               with help from\n",
      "              B. Webb, M.S. Madhusudhan, M-Y. Shen, G.Q. Dong,\n",
      "          M.A. Marti-Renom, N. Eswar, F. Alber, M. Topf, B. Oliva,\n",
      "             A. Fiser, R. Sanchez, B. Yerkovich, A. Badretdinov,\n",
      "                     F. Melo, J.P. Overington, E. Feyfant\n",
      "                 University of California, San Francisco, USA\n",
      "                    Rockefeller University, New York, USA\n",
      "                      Harvard University, Cambridge, USA\n",
      "                   Imperial Cancer Research Fund, London, UK\n",
      "              Birkbeck College, University of London, London, UK\n",
      "\n",
      "\n",
      "Kind, OS, HostName, Kernel, Processor: 4, Linux gecko 5.3.0-51-generic x86_64\n",
      "Date and time of compilation         : 2021/03/12 00:04:35\n",
      "MODELLER executable type             : x86_64-intel8\n",
      "Job starting time (YY/MM/DD HH:MM:SS): 2022/01/20 14:21:42\n",
      "\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       191566     187.076     0.183\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       192094     187.592     0.183\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       240694     235.053     0.230\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:       253994     248.041     0.242\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       393438     384.217     0.375\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       395038     385.779     0.377\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       399018     389.666     0.381\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       401018     391.619     0.382\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       404018     394.549     0.385\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       408518     398.943     0.390\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       414090     404.385     0.395\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       416890     407.119     0.398\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       421090     411.221     0.402\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       427390     417.373     0.408\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       430162     420.080     0.410\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:       431562     421.447     0.412\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       442599     432.226     0.422\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       442675     432.300     0.422\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       442846     432.467     0.422\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       443696     433.297     0.423\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       444954     434.525     0.424\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       454794     444.135     0.434\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       456123     445.433     0.435\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       463298     452.439     0.442\n",
      "mutate_mode_> Model residue 9 changed from ALA to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       463983     453.108     0.442\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       471158     460.115     0.449\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       472462     461.389     0.451\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    91    93\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    91    88    93    92\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     95      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       476530     465.361     0.454\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     95    128       160        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       480786     469.518     0.459\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     96    130       162        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     96    130       162        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       491823     480.296     0.469\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       491899     480.370     0.469\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       492070     480.537     0.469\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       492920     481.367     0.470\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       494178     482.596     0.471\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504018     492.205     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdbPRO9.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/cpep_aligned.pdbPRO9.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504018     492.205     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610486     596.178     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618678     604.178     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       96     130     164      33\n",
      "              Total number of restraints before, now         :        0      423\n",
      "make_re_422_> Number of previous, current restraints         :        0      423\n",
      "make_re_423_> Number of previous, current selected restraints:        0      423\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618780     604.277     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      423      430\n",
      "make_re_423_> Number of previous, current selected restraints:      423      430\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618796     604.293     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       13\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      430      438\n",
      "make_re_423_> Number of previous, current selected restraints:      430      425\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618796     604.293     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      438      445\n",
      "make_re_423_> Number of previous, current selected restraints:      425      432\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618796     604.293     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      445      451\n",
      "make_re_423_> Number of previous, current selected restraints:      432      438\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618796     604.293     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      451      455\n",
      "make_re_423_> Number of previous, current selected restraints:      438      442\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618796     604.293     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      455      457\n",
      "make_re_423_> Number of previous, current selected restraints:      442      444\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       63\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645832     630.695     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940692     918.645     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      457      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       39\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1378.6934\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   0.704   0.704      994.53       1.000\n",
      " 2 Bond angle potential               :      15       2      2  23.193  23.193      171.49       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       1      1  45.305  45.305      15.201       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.129   0.129     0.86452E-03   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      39       0      6   0.753   0.753      108.24       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.131   0.131     0.20322E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  56.089  56.089    -0.63142E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   3.544   3.544    -0.30910       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      1  58.385  58.385      10.271       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       2      2 114.632 131.925      79.331       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     62   9P   9P CG  CD     59   57    3.65    1.54    2.11   57.98    1.54    2.11   57.98\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    336   9P   9P CA  CB     56   58  -37.03   34.59  -71.62    6.85   34.59  -71.62    6.85\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    428   8C   9P C   N      53   55   53.38  -58.70  131.41   18.48  -64.50  160.61   10.47\n",
      "    1          9P   9P N   CA     55   56   38.11  -30.50                  147.20\n",
      "    2    429   9P  10W C   N      60   62  -90.08  -63.00   94.94   10.62  -63.00   94.94   10.62\n",
      "    2         10W  10W N   CA     62   63   46.79  -44.20                  -44.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.020\n",
      "  serious non-bonded atom clash:    54   59   2.285\n",
      "  serious non-bonded atom clash:    91   94   1.305\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    1    0    1    2    1    5    7   14   18    8   17    8   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       94        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      457      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       40\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         120.5696\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      1   0.053   0.053      6.6342       1.000\n",
      " 2 Bond angle potential               :      15       1      5   9.713   9.713      29.270       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      2  44.619  44.619      12.828       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      1   6.415   6.415      3.2421       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      40       0      5   0.665   0.665      20.639       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   8.343   8.343     0.82146       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  21.969  21.969      2.2015       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  56.902  56.902     0.52343       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  28.782  28.782      1.9342       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       1      2  85.404  85.404      42.475       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    429   9P  10W C   N      60   62  -96.78  -63.00   97.06   10.72  -63.00   97.06   10.72\n",
      "    1         10W  10W N   CA     62   63   46.79  -44.20                  -44.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.020\n",
      "  serious non-bonded atom clash:    91   94   1.305\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    2    7   16   20    8   15   12   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132258    1105.721     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132786    1106.236     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181386    1153.697     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194686    1166.686     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333938    1302.674     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335538    1304.236     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339518    1308.123     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341518    1310.076     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344518    1313.006     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349018    1317.400     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354590    1322.842     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357390    1325.576     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361590    1329.678     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367890    1335.830     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370662    1338.537     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372062    1339.904     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383099    1350.683     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383175    1350.757     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383346    1350.924     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384196    1351.754     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385454    1352.982     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395294    1362.592     1.331\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380141    1347.794     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1387418    1354.900     1.323\n",
      "mutate_mode_> Model residue 10 changed from TRP to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918687     897.155     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925964     904.262     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927132     905.402     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    88    90\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    88    85    90    89\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     90      0         0        28\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930820     909.004     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     90    119       143        28\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935076     913.160     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     91    121       145        29\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     91    121       145        29\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       86        5\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946113     923.938     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946189     924.013     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946360     924.180     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947210     925.010     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948468     926.238     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958308     935.848     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdbARG10.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdbARG10.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503706     491.900     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610174     595.873     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618366     603.873     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       91     121     146      27\n",
      "              Total number of restraints before, now         :        0      385\n",
      "make_re_422_> Number of previous, current restraints         :        0      385\n",
      "make_re_423_> Number of previous, current selected restraints:        0      385\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618366     603.873     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      385      392\n",
      "make_re_423_> Number of previous, current selected restraints:      385      392\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       13\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      392      400\n",
      "make_re_423_> Number of previous, current selected restraints:      392      387\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      400      407\n",
      "make_re_423_> Number of previous, current selected restraints:      387      394\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      407      413\n",
      "make_re_423_> Number of previous, current selected restraints:      394      400\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      413      418\n",
      "make_re_423_> Number of previous, current selected restraints:      400      405\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      418      421\n",
      "make_re_423_> Number of previous, current selected restraints:      405      408\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       64\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644554     629.447     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939342     917.326     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      421      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       15\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          75.6148\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.035   0.035      5.1805       1.000\n",
      " 2 Bond angle potential               :      17       0      1   4.462   4.462      7.9153       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      2  63.459  63.459      8.5499       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.556   0.556     0.18674E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      15       0      1   0.503   0.503      4.4674       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.875   5.875     0.81469       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  12.049  12.049     0.80429       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.81966E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.13744       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.036  90.036     0.73350       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      2  71.791  71.791      47.075       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    391   9P  10R C   N      60   62  -96.76  -63.00   94.15   10.91  -63.00   94.15   10.91\n",
      "    1         10R  10R N   CA     62   63   46.79  -41.10                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.305\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    2    7   13   18    8   12   10   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       91       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      421      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          43.8821\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.010   0.010     0.42629       1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.994   3.994      5.0447       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      2  68.264  68.264      9.5285       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.896   1.896     0.30834       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      1   0.428   0.428      2.6896       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  10.988  10.988      2.8498       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.858   0.858    -0.69157E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  69.980  69.980      1.7235       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  16.135  16.135     0.54687       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 169.919 169.919     0.82581       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      3  56.707  56.707      20.008       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.305\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    3    6   11   17    8    9   12   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1130908    1104.402     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131436    1104.918     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180036    1152.379     1.125\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193336    1165.367     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332588    1301.355     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334188    1302.918     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338168    1306.805     1.276\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340168    1308.758     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343168    1311.688     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347668    1316.082     1.285\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353240    1321.523     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356040    1324.258     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360240    1328.359     1.297\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366540    1334.512     1.303\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369312    1337.219     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370712    1338.586     1.307\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381749    1349.364     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1381825    1349.438     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381996    1349.605     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382846    1350.436     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384104    1351.664     1.320\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1393944    1361.273     1.329\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1378689    1346.376     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1385864    1353.383     1.322\n",
      "mutate_mode_> Model residue 11 changed from ARG to ASP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       926904     905.180     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    85    87\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    85    82    87    86\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     87     87      0         0        28\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930508     908.699     0.887\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     87     87    116       140        28\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       934764     912.855     0.891\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     88     88    118       142        29\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     88     88    118       142        29\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       86        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       945801     923.634     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       945877     923.708     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946048     923.875     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946898     924.705     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948156     925.934     0.904\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       957996     935.543     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdbASP11.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       88       88\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutated_match.pdbASP11.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503394     491.596     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       88\n",
      "read_mo_298_> Segment:        1  1:M  13:M       88\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       609862     595.568     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618054     603.568     0.589\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       88     118     144      27\n",
      "              Total number of restraints before, now         :        0      377\n",
      "make_re_422_> Number of previous, current restraints         :        0      377\n",
      "make_re_423_> Number of previous, current selected restraints:        0      377\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618054     603.568     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      377      384\n",
      "make_re_423_> Number of previous, current selected restraints:      377      384\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618070     603.584     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       13\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      384      392\n",
      "make_re_423_> Number of previous, current selected restraints:      384      379\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618070     603.584     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      392      399\n",
      "make_re_423_> Number of previous, current selected restraints:      379      386\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618070     603.584     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      399      405\n",
      "make_re_423_> Number of previous, current selected restraints:      386      392\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618070     603.584     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      405      409\n",
      "make_re_423_> Number of previous, current selected restraints:      392      396\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618070     603.584     0.589\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match1_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      409      411\n",
      "make_re_423_> Number of previous, current selected restraints:      396      398\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       50\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       643378     628.299     0.614\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       938094     916.107     0.895\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       88       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      411      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          45.5227\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.037   0.037      3.6282       1.000\n",
      " 2 Bond angle potential               :      14       0      0   3.551   3.551      3.3606       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      3  85.717  85.717      11.371       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.800   2.800     0.92529       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      1   0.571   0.571      7.7515       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  10.813  10.813      2.7595       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  97.452  97.452     0.29335       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 166.123 166.123      1.0690       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      2  51.003  51.003      14.364       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   84   2.020\n",
      "  serious non-bonded atom clash:    85   88   1.305\n",
      "  serious non-bonded atom clash:    86   88   2.165\n",
      "  serious non-bonded atom clash:    87   88   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    3    3    6   11   18    8    9   12   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       88        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       88       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      411      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          30.8662\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.007   0.007     0.15656       1.000\n",
      " 2 Bond angle potential               :      14       0      0   3.915   3.915      3.6989       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      3  84.036  84.036      11.341       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.504   2.504     0.64493       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      0   0.403   0.403     0.84843       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0  10.009  10.009      2.3648       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   4.359   4.359     0.35383E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  20.073  20.073     0.53929       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      2  39.568  39.568      11.237       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   84   2.020\n",
      "  serious non-bonded atom clash:    85   88   1.305\n",
      "  serious non-bonded atom clash:    86   88   2.165\n",
      "  serious non-bonded atom clash:    87   88   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    1    7   10   17    8    9   12   15\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match1_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       88       88\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1129660    1103.184     1.077\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1130188    1103.699     1.078\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178788    1151.160     1.124\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1192088    1164.148     1.137\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1331340    1300.137     1.270\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332940    1301.699     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336920    1305.586     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338920    1307.539     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341920    1310.469     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1346420    1314.863     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1351992    1320.305     1.289\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354792    1323.039     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358992    1327.141     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1365292    1333.293     1.302\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368064    1336.000     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369464    1337.367     1.306\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1380501    1348.146     1.317\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1380577    1348.220     1.317\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1380748    1348.387     1.317\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381598    1349.217     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382856    1350.445     1.319\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1392696    1360.055     1.328\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1377645    1345.356     1.314\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1384820    1352.363     1.321\n",
      "mutate_mode_> Model residue 1 changed from GLY to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927096     905.367     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931140     909.316     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLU  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    130       157        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935396     913.473     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    132       159        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    132       159        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        6\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946433     924.251     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946509     924.325     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946680     924.492     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947530     925.322     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948788     926.551     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958628     936.160     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdbGLU1.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/cpep_aligned.pdbGLU1.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504026     492.213     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610494     596.186     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618686     604.186     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       98     132     153      35\n",
      "              Total number of restraints before, now         :        0      418\n",
      "make_re_422_> Number of previous, current restraints         :        0      418\n",
      "make_re_423_> Number of previous, current selected restraints:        0      418\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618890     604.385     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      418      425\n",
      "make_re_423_> Number of previous, current selected restraints:      418      425\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618906     604.400     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      425      433\n",
      "make_re_423_> Number of previous, current selected restraints:      425      421\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618906     604.400     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      433      440\n",
      "make_re_423_> Number of previous, current selected restraints:      421      428\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618906     604.400     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      440      446\n",
      "make_re_423_> Number of previous, current selected restraints:      428      434\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618906     604.400     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      446      450\n",
      "make_re_423_> Number of previous, current selected restraints:      434      438\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618906     604.400     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      450      452\n",
      "make_re_423_> Number of previous, current selected restraints:      438      440\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       40\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646806     631.646     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941738     919.666     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      452      40\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.7425\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.022   0.022      1.2901       1.000\n",
      " 2 Bond angle potential               :      12       0      0   2.845   2.845      1.5389       1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      0  27.212  27.212     0.86211       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.773   0.773     0.33885E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      3   0.590   0.590      44.529       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.138   0.138     0.22631E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.910 112.910     0.56148       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000     0.59865E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.069   0.069     0.86663       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     6   74   2.262\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    1    0    2    0    1    2    8   14   18    8   14    9   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      452      40\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        9\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           6.6898\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.001   0.001     0.68573E-02   1.000\n",
      " 2 Bond angle potential               :      12       0      0   2.313   2.313      1.0295       1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      1  69.833  69.833      4.6110       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.238   0.238     0.42924E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       9       0      0   0.290   0.290    -0.61647       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.860   1.860     0.40832E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   9.018   9.018    -0.28887E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  96.822  96.822     0.76650       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   4.901   4.901     0.87614       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   17    8   14    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133304    1106.742     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133832    1107.258     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182432    1154.719     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195732    1167.707     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334984    1303.695     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336584    1305.258     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340564    1309.145     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342564    1311.098     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345564    1314.027     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350064    1318.422     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355636    1323.863     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358436    1326.598     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362636    1330.699     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368936    1336.852     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371708    1339.559     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373108    1340.926     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384145    1351.704     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384221    1351.778     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384392    1351.945     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385242    1352.775     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386500    1354.004     1.322\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396340    1363.613     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381085    1348.716     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388464    1355.922     1.324\n",
      "mutate_mode_> Model residue 4 changed from SER to SER\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918789     897.255     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926168     904.461     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927504     905.766     0.885\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931548     909.715     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLU  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    130       157        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935804     913.871     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    132       159        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    132       159        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       97        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946841     924.649     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946917     924.724     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947088     924.891     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947938     925.721     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       949196     926.949     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       959036     936.559     0.915\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdbSER4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutated_match.pdbSER4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504434     492.611     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610902     596.584     0.583\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       619094     604.584     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       98     132     153      35\n",
      "              Total number of restraints before, now         :        0      418\n",
      "make_re_422_> Number of previous, current restraints         :        0      418\n",
      "make_re_423_> Number of previous, current selected restraints:        0      418\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619094     604.584     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      418      425\n",
      "make_re_423_> Number of previous, current selected restraints:      418      425\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619110     604.600     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      425      433\n",
      "make_re_423_> Number of previous, current selected restraints:      425      421\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619110     604.600     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      433      440\n",
      "make_re_423_> Number of previous, current selected restraints:      421      428\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619110     604.600     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      440      446\n",
      "make_re_423_> Number of previous, current selected restraints:      428      434\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619110     604.600     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      446      450\n",
      "make_re_423_> Number of previous, current selected restraints:      434      438\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619110     604.600     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match3_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      450      452\n",
      "make_re_423_> Number of previous, current selected restraints:      438      440\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        6       40\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       647010     631.846     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941942     919.865     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       6\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      452      40\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          13.2592\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       7       0      0   0.039   0.039      3.3342       1.000\n",
      " 2 Bond angle potential               :      11       0      0   3.098   3.098      2.2552       1.000\n",
      " 3 Stereochemical cosine torsion poten:      15       0      0  71.449  71.449      4.7028       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.091   0.091     0.49031E-03   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      1   0.386   0.386      2.5960       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.622   0.622     0.45634E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   2.443   2.443     0.17871E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.500  22.500     0.34798       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   17    8   14    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97        6        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       6\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      452      40\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           8.4583\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       7       0      0   0.008   0.008     0.13714       1.000\n",
      " 2 Bond angle potential               :      11       0      0   2.499   2.499      1.2658       1.000\n",
      " 3 Stereochemical cosine torsion poten:      15       0      0  73.170  73.170      4.9399       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.784   0.784     0.56963E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.381   0.381      1.6550       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   2.649   2.649     0.82813E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.000   0.000    -0.46811E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  24.171  24.171     0.36743       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    1    8   12   17    9   14    8   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match3_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133508    1106.941     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1134036    1107.457     1.082\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182636    1154.918     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195936    1167.906     1.141\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335188    1303.895     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336788    1305.457     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340768    1309.344     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342768    1311.297     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345768    1314.227     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350268    1318.621     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355840    1324.062     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358640    1326.797     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362840    1330.898     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369140    1337.051     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371912    1339.758     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373312    1341.125     1.310\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384349    1351.903     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384425    1351.978     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384596    1352.145     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385446    1352.975     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386704    1354.203     1.322\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396544    1363.812     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381085    1348.716     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388260    1355.723     1.324\n",
      "mutate_mode_> Model residue 1 changed from GLY to ASN\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927104     905.375     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    93    95\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    93    90    95    94\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96      0         0        36\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931120     909.297     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  ASN  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96    129       156        36\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935376     913.453     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    131       158        37\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    131       158        37\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        5\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946413     924.231     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946489     924.306     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946660     924.473     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947510     925.303     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948768     926.531     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958608     936.141     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdbASN1.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       96       96\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/cpep_aligned.pdbASN1.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504006     492.193     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       96\n",
      "read_mo_298_> Segment:        1  1:M  13:M       96\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610474     596.166     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618666     604.166     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       97     131     152      36\n",
      "              Total number of restraints before, now         :        0      416\n",
      "make_re_422_> Number of previous, current restraints         :        0      416\n",
      "make_re_423_> Number of previous, current selected restraints:        0      416\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618836     604.332     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      416      423\n",
      "make_re_423_> Number of previous, current selected restraints:      416      423\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618852     604.348     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      423      431\n",
      "make_re_423_> Number of previous, current selected restraints:      423      419\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618852     604.348     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      431      438\n",
      "make_re_423_> Number of previous, current selected restraints:      419      426\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618852     604.348     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      438      444\n",
      "make_re_423_> Number of previous, current selected restraints:      426      432\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618852     604.348     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      444      447\n",
      "make_re_423_> Number of previous, current selected restraints:      432      435\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618852     604.348     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      447      449\n",
      "make_re_423_> Number of previous, current selected restraints:      435      437\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       37\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646464     631.312     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941372     919.309     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       96       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      449      37\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       27\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         196.7206\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.023   0.023      1.2376       1.000\n",
      " 2 Bond angle potential               :      11       0      0   2.652   2.652      1.2927       1.000\n",
      " 3 Stereochemical cosine torsion poten:      11       0      0  28.555  28.555     0.87585       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.652   0.652     0.27552E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      27       0      3   0.572   0.572      191.50       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.107   0.107     0.13395E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 110.499 110.499     0.40872       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 144.842 144.842      1.3788       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     6    9   2.084\n",
      "  serious non-bonded atom clash:    12   56   2.017\n",
      "  serious non-bonded atom clash:    18   92   2.020\n",
      "  serious non-bonded atom clash:    93   96   1.306\n",
      "  serious non-bonded atom clash:    94   96   2.166\n",
      "  serious non-bonded atom clash:    95   96   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    0    0    1    0    1    2    9   14   18    8   15   10   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       96        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       96       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      449      37\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        5\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           3.1886\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.002   0.002     0.79801E-02   1.000\n",
      " 2 Bond angle potential               :      11       0      0   1.042   1.042     0.19229       1.000\n",
      " 3 Stereochemical cosine torsion poten:      11       0      0  48.041  48.041      2.3690       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.442   0.442     0.15627E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       5       0      0   0.430   0.430    -0.55749E-01   1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.525   0.525     0.32515E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   5.194   5.194     0.42674E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  11.542  11.542     0.61345       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    12   56   2.017\n",
      "  serious non-bonded atom clash:    18   92   2.020\n",
      "  serious non-bonded atom clash:    93   96   1.306\n",
      "  serious non-bonded atom clash:    94   96   2.166\n",
      "  serious non-bonded atom clash:    95   96   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   17    8   12    8   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       96       96\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132938    1106.385     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133466    1106.900     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182066    1154.361     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195366    1167.350     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334618    1303.338     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336218    1304.900     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340198    1308.787     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342198    1310.740     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345198    1313.670     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349698    1318.064     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355270    1323.506     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358070    1326.240     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362270    1330.342     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368570    1336.494     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371342    1339.201     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372742    1340.568     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383779    1351.347     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383855    1351.421     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384026    1351.588     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384876    1352.418     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386134    1353.646     1.322\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395974    1363.256     1.331\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       96\n",
      "read_mo_298_> Segment:        1  1:M  13:M       96\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380753    1348.392     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388098    1355.564     1.324\n",
      "mutate_mode_> Model residue 4 changed from SER to VAL\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918755     897.222     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926100     904.395     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927468     905.730     0.885\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97      0         0        37\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931540     909.707     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  ASN  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    131       158        37\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935796     913.863     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    133       160        38\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    133       160        38\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       95        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946833     924.642     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946909     924.716     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947080     924.883     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947930     925.713     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       949188     926.941     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       959028     936.551     0.915\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       96\n",
      "read_mo_298_> Segment:        1  1:M  13:M       96\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdbVAL4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutated_match.pdbVAL4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504426     492.604     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610894     596.576     0.583\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       619086     604.576     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       98     133     154      37\n",
      "              Total number of restraints before, now         :        0      422\n",
      "make_re_422_> Number of previous, current restraints         :        0      422\n",
      "make_re_423_> Number of previous, current selected restraints:        0      422\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619120     604.609     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      422      429\n",
      "make_re_423_> Number of previous, current selected restraints:      422      429\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619136     604.625     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      429      437\n",
      "make_re_423_> Number of previous, current selected restraints:      429      425\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619136     604.625     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      437      444\n",
      "make_re_423_> Number of previous, current selected restraints:      425      432\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619136     604.625     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      444      450\n",
      "make_re_423_> Number of previous, current selected restraints:      432      438\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619136     604.625     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      450      453\n",
      "make_re_423_> Number of previous, current selected restraints:      438      441\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619136     604.625     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match4_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      453      455\n",
      "make_re_423_> Number of previous, current selected restraints:      441      443\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       46\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       647036     631.871     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941968     919.891     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      455      46\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       22\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          62.4867\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.038   0.038      3.5762       1.000\n",
      " 2 Bond angle potential               :      13       0      0   3.324   3.324      2.6408       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.146  67.146      4.7164       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.393   2.393     0.38371       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      22       0      2   0.626   0.626      50.476       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.609   0.609     0.43739E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.000   0.000    -0.29862       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.750  23.750     0.98757       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    12   57   2.017\n",
      "  serious non-bonded atom clash:    18   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    2    0    1    2    7   13   17    9   13    8   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      455      46\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          11.5770\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.007   0.007     0.12357       1.000\n",
      " 2 Bond angle potential               :      13       0      0   2.524   2.524      1.6858       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.285  67.285      4.7714       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.708   1.708     0.27611       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      1   0.466   0.466      4.0740       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   3.107   3.107     0.11395       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   9.187   9.187    -0.23406       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.699  22.699     0.76622       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    12   57   2.017\n",
      "  serious non-bonded atom clash:    18   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.166\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   12   17    9   12   10   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match4_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133534    1106.967     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1134062    1107.482     1.082\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182662    1154.943     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195962    1167.932     1.141\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335214    1303.920     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336814    1305.482     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340794    1309.369     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342794    1311.322     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345794    1314.252     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350294    1318.646     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355866    1324.088     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358666    1326.822     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362866    1330.924     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369166    1337.076     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371938    1339.783     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373338    1341.150     1.310\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384375    1351.929     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384451    1352.003     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384622    1352.170     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385472    1353.000     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386730    1354.229     1.322\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396570    1363.838     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381145    1348.774     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388320    1355.781     1.324\n",
      "mutate_mode_> Model residue 5 changed from ASP to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927024     905.297     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    88    90\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    88    85    90    89\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     92      0         0        33\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930980     909.160     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     92    124       156        33\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935236     913.316     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     93    126       158        34\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     93    126       158        34\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       89        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946273     924.095     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946349     924.169     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946520     924.336     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947370     925.166     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948628     926.395     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958468     936.004     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdbPRO5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/cpep_aligned.pdbPRO5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503866     492.057     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610334     596.029     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618526     604.029     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       93     126     160      32\n",
      "              Total number of restraints before, now         :        0      411\n",
      "make_re_422_> Number of previous, current restraints         :        0      411\n",
      "make_re_423_> Number of previous, current selected restraints:        0      411\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618526     604.029     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      411      418\n",
      "make_re_423_> Number of previous, current selected restraints:      411      418\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618542     604.045     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       13\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      418      426\n",
      "make_re_423_> Number of previous, current selected restraints:      418      413\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618542     604.045     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      426      432\n",
      "make_re_423_> Number of previous, current selected restraints:      413      419\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618542     604.045     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      432      437\n",
      "make_re_423_> Number of previous, current selected restraints:      419      424\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618542     604.045     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      437      441\n",
      "make_re_423_> Number of previous, current selected restraints:      424      428\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618542     604.045     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      441      443\n",
      "make_re_423_> Number of previous, current selected restraints:      428      430\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       71\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644714     629.604     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939502     917.482     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      443      71\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       54\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2774.4824\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   0.570   0.570      650.58       1.000\n",
      " 2 Bond angle potential               :      16       2      2  17.579  17.579      103.94       1.000\n",
      " 3 Stereochemical cosine torsion poten:      35       0      0  40.617  40.617      11.858       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.264   0.264     0.35613E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      54       0      8   0.797   0.797      1985.2       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.637   4.637     0.50754       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  27.222  27.222      3.7447       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1  91.512  91.512      2.5224       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      1 111.374 111.374      6.0041       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  33.879  33.879      10.113       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     29   5P   5P CG  CD     27   25    3.24    1.54    1.71   46.89    1.54    1.71   46.89\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   51   2.017\n",
      "  serious non-bonded atom clash:    10   25   1.696\n",
      "  serious non-bonded atom clash:    14   87   2.021\n",
      "  serious non-bonded atom clash:    17   25   2.112\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    2    0    0    4    0    1    2    7   12   22    7   13    9   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       91        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      443      71\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       42\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          44.5921\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.018   0.018     0.77154       1.000\n",
      " 2 Bond angle potential               :      16       0      0   4.653   4.653      5.8769       1.000\n",
      " 3 Stereochemical cosine torsion poten:      35       1      2  48.000  48.000      20.405       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   3.866   3.866      1.6375       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      42       0      1   0.529   0.529      9.7143       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  11.944  11.944      3.3673       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.864   3.864    -0.18244       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  66.278  66.278    -0.15591       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  64.499  64.499     0.77405E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  29.250  29.250      3.0809       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    250   4S   4S O   C      22   21  109.02  180.00  -70.98    4.92  180.00  -70.98    4.92\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   51   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.021\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    4   12   18    7   16   10    8\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131068    1104.559     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131596    1105.074     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180196    1152.535     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193496    1165.523     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332748    1301.512     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334348    1303.074     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338328    1306.961     1.276\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340328    1308.914     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343328    1311.844     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347828    1316.238     1.285\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353400    1321.680     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356200    1324.414     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360400    1328.516     1.297\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366700    1334.668     1.303\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369472    1337.375     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370872    1338.742     1.307\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381909    1349.521     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1381985    1349.595     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382156    1349.762     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383006    1350.592     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384264    1351.820     1.320\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394104    1361.430     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379053    1346.731     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386228    1353.738     1.322\n",
      "mutate_mode_> Model residue 6 changed from PRO to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927048     905.320     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    92    94\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    92    89    94    93\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95      0         0        33\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930980     909.160     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    126       153        33\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935236     913.316     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96    128       155        34\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96    128       155        34\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946273     924.095     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946349     924.169     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946520     924.336     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947370     925.166     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948628     926.395     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958468     936.004     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdbARG6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       95       95\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdbARG6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503866     492.057     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610334     596.029     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618526     604.029     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       96     128     149      33\n",
      "              Total number of restraints before, now         :        0      406\n",
      "make_re_422_> Number of previous, current restraints         :        0      406\n",
      "make_re_423_> Number of previous, current selected restraints:        0      406\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618662     604.162     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      406      413\n",
      "make_re_423_> Number of previous, current selected restraints:      406      413\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618678     604.178     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      413      421\n",
      "make_re_423_> Number of previous, current selected restraints:      413      409\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618678     604.178     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      421      427\n",
      "make_re_423_> Number of previous, current selected restraints:      409      415\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618678     604.178     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      427      432\n",
      "make_re_423_> Number of previous, current selected restraints:      415      420\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618678     604.178     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      432      436\n",
      "make_re_423_> Number of previous, current selected restraints:      420      424\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618678     604.178     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      436      439\n",
      "make_re_423_> Number of previous, current selected restraints:      424      427\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       64\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646002     630.861     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940886     918.834     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       95      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      439      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       41\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       32703.6543\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.034   0.034      5.0825       1.000\n",
      " 2 Bond angle potential               :      17       0      0   4.011   4.011      5.3696       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      1  59.896  59.896      5.6190       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.827   0.827     0.81487E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      41       0     12   1.009   1.009      32666.       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.170   5.170     0.63102       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  81.984  81.984      8.1344       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1 165.070 165.070      2.8548       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.24278       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.064  90.064     0.73345       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  32.444  32.444      9.3365       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   55   2.017\n",
      "  serious non-bonded atom clash:    14   91   2.021\n",
      "  serious non-bonded atom clash:    24   38   1.748\n",
      "  serious non-bonded atom clash:    26   38   1.497\n",
      "  serious non-bonded atom clash:    30   34   1.491\n",
      "  serious non-bonded atom clash:    92   95   1.306\n",
      "  serious non-bonded atom clash:    93   95   2.165\n",
      "  serious non-bonded atom clash:    94   95   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     7    1    0    1    2    1    1    4    7   13   18    9   17   12    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       95       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       95      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      439      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       17\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.3286\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.005   0.005     0.74900E-01   1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.163   3.163      3.0579       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      0  55.366  55.366      4.8421       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.343   1.343     0.23964       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      17       0      0   0.418   0.418      1.0048       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   7.734   7.734      1.4117       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 126.518 126.518     0.65440       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  25.945  25.945     0.86224       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 103.224 103.224     0.49865       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 165.711 165.711     0.97648       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  26.800  26.800      4.7058       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   55   2.017\n",
      "  serious non-bonded atom clash:    14   91   2.021\n",
      "  serious non-bonded atom clash:    92   95   1.306\n",
      "  serious non-bonded atom clash:    93   95   2.165\n",
      "  serious non-bonded atom clash:    94   95   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    3   12   18    8   16   10    7\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       95       95\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132452    1105.910     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132980    1106.426     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181580    1153.887     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194880    1166.875     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334132    1302.863     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335732    1304.426     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339712    1308.312     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341712    1310.266     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344712    1313.195     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349212    1317.590     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354784    1323.031     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357584    1325.766     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361784    1329.867     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368084    1336.020     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370856    1338.727     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372256    1340.094     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383293    1350.872     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383369    1350.946     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383540    1351.113     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384390    1351.943     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385648    1353.172     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395488    1362.781     1.331\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380301    1347.950     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1387612    1355.090     1.323\n",
      "mutate_mode_> Model residue 7 changed from ARG to ASP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918721     897.188     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926032     904.328     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927296     905.562     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    89    91\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    89    86    91    90\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92      0         0        33\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931144     909.320     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    123       150        33\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935400     913.477     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    125       152        34\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    125       152        34\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       90        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946437     924.255     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946513     924.329     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946684     924.496     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947534     925.326     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948792     926.555     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958632     936.164     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdbASP7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutated_match.pdbASP7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504030     492.217     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610498     596.189     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618690     604.189     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       93     125     147      33\n",
      "              Total number of restraints before, now         :        0      398\n",
      "make_re_422_> Number of previous, current restraints         :        0      398\n",
      "make_re_423_> Number of previous, current selected restraints:        0      398\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618690     604.189     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      398      405\n",
      "make_re_423_> Number of previous, current selected restraints:      398      405\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618706     604.205     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      405      413\n",
      "make_re_423_> Number of previous, current selected restraints:      405      401\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618706     604.205     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      413      419\n",
      "make_re_423_> Number of previous, current selected restraints:      401      407\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618706     604.205     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      419      424\n",
      "make_re_423_> Number of previous, current selected restraints:      407      412\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618706     604.205     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      424      427\n",
      "make_re_423_> Number of previous, current selected restraints:      412      415\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618706     604.205     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match5_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      427      429\n",
      "make_re_423_> Number of previous, current selected restraints:      415      417\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       50\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645166     630.045     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939978     917.947     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      429      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       26\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          89.3657\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.033   0.033      2.9416       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.739   2.739      1.9334       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  65.153  65.153      4.5728       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.818   2.818     0.93929       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      26       0      2   0.579   0.579      74.502       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   6.490   6.490     0.99419       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 131.612 131.612     0.34861       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 166.138 166.138      1.0148       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  20.639  20.639      2.1195       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.021\n",
      "  serious non-bonded atom clash:    46   49   2.129\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    2    0    0    1    0    3    3    4   12   19   10   16    8    6\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       92        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      429      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          10.8350\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.008   0.008     0.17899       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.225   2.225      1.3135       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  63.389  63.389      4.3861       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.772   2.772     0.91658       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      0   0.367   0.367      1.1014       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.291   5.291     0.66069       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 133.766 133.766     0.34131       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 128.144 128.144      1.3610       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  16.849  16.849     0.57538       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.021\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    4   11   18    7   18    9   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match5_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131544    1105.023     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132072    1105.539     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180672    1153.000     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193972    1165.988     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333224    1301.977     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334824    1303.539     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338804    1307.426     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340804    1309.379     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343804    1312.309     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348304    1316.703     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353876    1322.145     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356676    1324.879     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360876    1328.980     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367176    1335.133     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369948    1337.840     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371348    1339.207     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382385    1349.985     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382461    1350.060     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382632    1350.227     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383482    1351.057     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384740    1352.285     1.321\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394580    1361.895     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379257    1346.931     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386432    1353.938     1.322\n",
      "mutate_mode_> Model residue 7 changed from ARG to THR\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       926992     905.266     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    85    87\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    85    82    87    86\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     87     88      0         0        33\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930744     908.930     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     87     88    119       147        33\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935000     913.086     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     88     89    121       149        34\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     88     89    121       149        34\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       85        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946037     923.864     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946113     923.938     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946284     924.105     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947134     924.936     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948392     926.164     0.904\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958232     935.773     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdbTHR7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       88       88\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/cpep_aligned.pdbTHR7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503630     491.826     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       88\n",
      "read_mo_298_> Segment:        1  1:M  13:M       88\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610098     595.799     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618290     603.799     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       89     121     145      33\n",
      "              Total number of restraints before, now         :        0      388\n",
      "make_re_422_> Number of previous, current restraints         :        0      388\n",
      "make_re_423_> Number of previous, current selected restraints:        0      388\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618290     603.799     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      388      395\n",
      "make_re_423_> Number of previous, current selected restraints:      388      395\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618306     603.814     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      395      403\n",
      "make_re_423_> Number of previous, current selected restraints:      395      391\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618306     603.814     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      403      409\n",
      "make_re_423_> Number of previous, current selected restraints:      391      397\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618306     603.814     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      409      413\n",
      "make_re_423_> Number of previous, current selected restraints:      397      401\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618306     603.814     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      413      415\n",
      "make_re_423_> Number of previous, current selected restraints:      401      403\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618306     603.814     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      416\n",
      "make_re_423_> Number of previous, current selected restraints:      403      404\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       50\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       643614     628.529     0.614\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       938330     916.338     0.895\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       88       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      416      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          16.8014\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.039   0.039      3.6689       1.000\n",
      " 2 Bond angle potential               :      13       0      0   3.153   3.153      2.4391       1.000\n",
      " 3 Stereochemical cosine torsion poten:      20       0      0  64.526  64.526      4.2118       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.600   2.600     0.45310       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      1   0.443   0.443      4.4671       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   0.322   0.322     0.24480E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 116.902 116.902     0.80375       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.136  23.136     0.75508       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   84   2.020\n",
      "  serious non-bonded atom clash:    85   88   1.306\n",
      "  serious non-bonded atom clash:    86   88   2.165\n",
      "  serious non-bonded atom clash:    87   88   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   14   17    5   13    7    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       88        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       88       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      416      50\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       13\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           9.0181\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.006   0.006     0.79632E-01   1.000\n",
      " 2 Bond angle potential               :      13       0      0   1.743   1.743     0.83332       1.000\n",
      " 3 Stereochemical cosine torsion poten:      20       0      0  65.006  65.006      4.3160       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.207   1.207     0.12698       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      13       0      1   0.467   0.467      3.2183       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.420   1.420     0.47563E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.648   3.648    -0.11488       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.187  22.187     0.51117       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   84   2.020\n",
      "  serious non-bonded atom clash:    85   88   1.306\n",
      "  serious non-bonded atom clash:    86   88   2.165\n",
      "  serious non-bonded atom clash:    87   88   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    5   14   19    4   12    7    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       88       88\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1129896    1103.414     1.078\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1130424    1103.930     1.078\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179024    1151.391     1.124\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1192324    1164.379     1.137\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1331576    1300.367     1.270\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333176    1301.930     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1337156    1305.816     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339156    1307.770     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342156    1310.699     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1346656    1315.094     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1352228    1320.535     1.290\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355028    1323.270     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1359228    1327.371     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1365528    1333.523     1.302\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368300    1336.230     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369700    1337.598     1.306\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1380737    1348.376     1.317\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1380813    1348.450     1.317\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1380984    1348.617     1.317\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381834    1349.447     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383092    1350.676     1.319\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1392932    1360.285     1.328\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       88\n",
      "read_mo_298_> Segment:        1  1:M  13:M       88\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1377881    1345.587     1.314\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1384954    1352.494     1.321\n",
      "mutate_mode_> Model residue 9 changed from ALA to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918483     896.956     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925556     903.863     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       926852     905.129     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    91    93\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    91    88    93    92\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930800     908.984     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    126       154        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935056     913.141     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       156        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       156        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       88        6\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946093     923.919     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946169     923.993     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946340     924.160     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947190     924.990     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948448     926.219     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958288     935.828     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       88\n",
      "read_mo_298_> Segment:        1  1:M  13:M       88\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdbARG9.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdbARG9.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503686     491.881     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610154     595.854     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618346     603.854     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       95     128     151      34\n",
      "              Total number of restraints before, now         :        0      408\n",
      "make_re_422_> Number of previous, current restraints         :        0      408\n",
      "make_re_423_> Number of previous, current selected restraints:        0      408\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618550     604.053     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      408      415\n",
      "make_re_423_> Number of previous, current selected restraints:      408      415\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      415      423\n",
      "make_re_423_> Number of previous, current selected restraints:      415      411\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      423      430\n",
      "make_re_423_> Number of previous, current selected restraints:      411      418\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      430      435\n",
      "make_re_423_> Number of previous, current selected restraints:      418      423\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      435      438\n",
      "make_re_423_> Number of previous, current selected restraints:      423      426\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      438      440\n",
      "make_re_423_> Number of previous, current selected restraints:      426      428\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       59\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645602     630.471     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940462     918.420     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      440      59\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       17\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          56.2541\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.034   0.034      4.6104       1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.088   3.088      3.5170       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  66.479  66.479      7.6610       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.413   0.413     0.86465E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      17       0      1   0.498   0.498      6.5156       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.183   0.183     0.39722E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.955 112.955     0.23452       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.21151E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 114.564 114.564     0.37348       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.036  90.036     0.73350       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       1      1  67.228  67.228      32.621       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    414   9R  10W C   N      60   62  -90.09  -63.00   94.99   10.63  -63.00   94.99   10.63\n",
      "    1         10W  10W N   CA     62   63   46.85  -44.20                  -44.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.020\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    3    5   14   19    4   12    7    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       94       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      440      59\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       20\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.6971\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.008   0.008     0.18637       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.922   2.922      2.5027       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  68.309  68.309      8.2545       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.548   2.548     0.51025       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      20       0      0   0.458   0.458      1.3007       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   4.093   4.093     0.19771       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  86.328  86.328      1.1474       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  71.484  71.484      1.5706       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 129.997 129.997     0.73361       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  68.608  68.608     0.86361       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       1      1  67.238  67.238      32.430       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    414   9R  10W C   N      60   62  -89.03  -63.00   94.69   10.62  -63.00   94.69   10.62\n",
      "    1         10W  10W N   CA     62   63   46.85  -44.20                  -44.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.020\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    5   14   19    5   12    7    7\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132028    1105.496     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132556    1106.012     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181156    1153.473     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194456    1166.461     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333708    1302.449     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335308    1304.012     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339288    1307.898     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341288    1309.852     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344288    1312.781     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348788    1317.176     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354360    1322.617     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357160    1325.352     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361360    1329.453     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367660    1335.605     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370432    1338.312     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371832    1339.680     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382869    1350.458     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382945    1350.532     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383116    1350.699     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383966    1351.529     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385224    1352.758     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395064    1362.367     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380013    1347.669     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1387290    1354.775     1.323\n",
      "mutate_mode_> Model residue 10 changed from TRP to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918687     897.155     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925964     904.262     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927124     905.395     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    88    90\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    88    85    90    89\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     89      0         0        28\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930692     908.879     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     89    117       137        28\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       934948     913.035     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     90    119       139        29\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     90    119       139        29\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       86        5\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       945985     923.813     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946061     923.888     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946232     924.055     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947082     924.885     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948340     926.113     0.904\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958180     935.723     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdbARG10.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutated_match.pdbARG10.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503578     491.775     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610046     595.748     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618238     603.748     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       90     119     133      28\n",
      "              Total number of restraints before, now         :        0      370\n",
      "make_re_422_> Number of previous, current restraints         :        0      370\n",
      "make_re_423_> Number of previous, current selected restraints:        0      370\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618238     603.748     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      370      377\n",
      "make_re_423_> Number of previous, current selected restraints:      370      377\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618254     603.764     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      377      385\n",
      "make_re_423_> Number of previous, current selected restraints:      377      373\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618254     603.764     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      385      392\n",
      "make_re_423_> Number of previous, current selected restraints:      373      380\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618254     603.764     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      392      397\n",
      "make_re_423_> Number of previous, current selected restraints:      380      385\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618254     603.764     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      397      401\n",
      "make_re_423_> Number of previous, current selected restraints:      385      389\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618254     603.764     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match6_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      401      404\n",
      "make_re_423_> Number of previous, current selected restraints:      389      392\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       61\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644426     629.322     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939214     917.201     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      404      61\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          58.4233\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.034   0.034      4.5962       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.188   2.188      1.6103       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  69.730  69.730      8.2640       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.787   0.787     0.62042E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      1   0.523   0.523      4.6075       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.881   2.881     0.19588       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  11.971  11.971     0.79298       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000    -0.82175E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.028   0.028     0.13797       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.016  90.016     0.73236       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  57.462  57.462      37.506       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    376   9R  10R C   N      60   62  -89.02  -63.00   91.72   10.89  -63.00   91.72   10.89\n",
      "    1         10R  10R N   CA     62   63   46.85  -41.10                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    5   11   17    5    9    5    6\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       91       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      404      61\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          33.6697\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.009   0.009     0.29012       1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.800   3.800      4.5101       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  73.674  73.674      9.4675       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.653   1.653     0.21162       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      1   0.505   0.505      2.3170       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  13.156  13.156      4.0855       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 126.932 126.932     0.21214       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  14.857  14.857     0.12692       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   5.195   5.195     0.20658       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   2.173   2.173     0.55946       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  38.958  38.958      11.683       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   48   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.595\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    3    5    9   16    5    7    8    7\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match6_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1130780    1104.277     1.078\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131308    1104.793     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179908    1152.254     1.125\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193208    1165.242     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332460    1301.230     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334060    1302.793     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338040    1306.680     1.276\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340040    1308.633     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343040    1311.562     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347540    1315.957     1.285\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353112    1321.398     1.290\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355912    1324.133     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360112    1328.234     1.297\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366412    1334.387     1.303\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369184    1337.094     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370584    1338.461     1.307\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381621    1349.239     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1381697    1349.313     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381868    1349.480     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382718    1350.311     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383976    1351.539     1.320\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1393816    1361.148     1.329\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1378561    1346.251     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1385736    1353.258     1.322\n",
      "mutate_mode_> Model residue 4 changed from SER to THR\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927048     905.320     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    90    92\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    90    87    92    91\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930952     909.133     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    125       152        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935208     913.289     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    127       154        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    127       154        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       90        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946245     924.067     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946321     924.142     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946492     924.309     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947342     925.139     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948600     926.367     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958440     935.977     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdbTHR4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       93       93\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/cpep_aligned.pdbTHR4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503838     492.029     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610306     596.002     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618498     604.002     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       94     127     149      34\n",
      "              Total number of restraints before, now         :        0      404\n",
      "make_re_422_> Number of previous, current restraints         :        0      404\n",
      "make_re_423_> Number of previous, current selected restraints:        0      404\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618566     604.068     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      404      411\n",
      "make_re_423_> Number of previous, current selected restraints:      404      411\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618582     604.084     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      411      419\n",
      "make_re_423_> Number of previous, current selected restraints:      411      407\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618582     604.084     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      419      425\n",
      "make_re_423_> Number of previous, current selected restraints:      407      413\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618582     604.084     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      425      430\n",
      "make_re_423_> Number of previous, current selected restraints:      413      418\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618582     604.084     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      430      433\n",
      "make_re_423_> Number of previous, current selected restraints:      418      421\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618582     604.084     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      433      435\n",
      "make_re_423_> Number of previous, current selected restraints:      421      423\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       46\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645330     630.205     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940166     918.131     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       93       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      435      46\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       33\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         224.3683\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.038   0.038      3.6155       1.000\n",
      " 2 Bond angle potential               :      13       0      0   3.257   3.257      2.6236       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.112  67.112      4.7000       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.678   2.678     0.48071       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      33       0      5   0.649   0.649      211.05       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.595   0.595     0.41846E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 121.140 121.140     0.66741       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.733  23.733      1.2297       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     4   21   1.998\n",
      "  serious non-bonded atom clash:     8   53   2.016\n",
      "  serious non-bonded atom clash:    14   89   2.020\n",
      "  serious non-bonded atom clash:    90   93   1.305\n",
      "  serious non-bonded atom clash:    91   93   2.165\n",
      "  serious non-bonded atom clash:    92   93   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    0    0    1    1    1    4    7   15   16    9   16    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       93        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       93       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      435      46\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       22\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           9.9018\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.006   0.006     0.98159E-01   1.000\n",
      " 2 Bond angle potential               :      13       0      0   2.212   2.212      1.3582       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.280  67.280      4.6996       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.234   1.234     0.15277       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      22       0      1   0.391   0.391      2.9311       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   2.753   2.753     0.89480E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.303   1.303    -0.29269       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.657  22.657     0.86531       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   53   2.016\n",
      "  serious non-bonded atom clash:    14   89   2.020\n",
      "  serious non-bonded atom clash:    90   93   1.305\n",
      "  serious non-bonded atom clash:    91   93   2.165\n",
      "  serious non-bonded atom clash:    92   93   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   14   17   10   13    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       93       93\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131732    1105.207     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132260    1105.723     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180860    1153.184     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194160    1166.172     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333412    1302.160     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335012    1303.723     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338992    1307.609     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340992    1309.562     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343992    1312.492     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348492    1316.887     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354064    1322.328     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356864    1325.062     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361064    1329.164     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367364    1335.316     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370136    1338.023     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371536    1339.391     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382573    1350.169     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382649    1350.243     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382820    1350.410     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383670    1351.240     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384928    1352.469     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394768    1362.078     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379649    1347.313     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386892    1354.387     1.323\n",
      "mutate_mode_> Model residue 5 changed from ASP to ASN\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918653     897.122     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925896     904.195     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927200     905.469     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    90    92\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    90    87    92    91\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931104     909.281     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    125       152        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935360     913.438     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    127       154        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    127       154        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       92        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946397     924.216     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946473     924.290     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946644     924.457     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947494     925.287     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948752     926.516     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958592     936.125     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbASN5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       93       93\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbASN5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503990     492.178     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610458     596.150     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618650     604.150     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       94     127     149      35\n",
      "              Total number of restraints before, now         :        0      405\n",
      "make_re_422_> Number of previous, current restraints         :        0      405\n",
      "make_re_423_> Number of previous, current selected restraints:        0      405\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618650     604.150     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      405      412\n",
      "make_re_423_> Number of previous, current selected restraints:      405      412\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618666     604.166     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      412      420\n",
      "make_re_423_> Number of previous, current selected restraints:      412      408\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618666     604.166     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      420      426\n",
      "make_re_423_> Number of previous, current selected restraints:      408      414\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618666     604.166     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      426      431\n",
      "make_re_423_> Number of previous, current selected restraints:      414      419\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618666     604.166     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      431      434\n",
      "make_re_423_> Number of previous, current selected restraints:      419      422\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618666     604.166     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      434      436\n",
      "make_re_423_> Number of previous, current selected restraints:      422      424\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       58\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645414     630.287     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940250     918.213     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       93       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      436      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       53\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          30.1191\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.028   0.028      2.0896       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.209   4.209      4.7078       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      0  48.754  48.754      6.7857       1.000\n",
      " 4 Stereochemical improper torsion pot:       5       0      0   0.796   0.796     0.10161       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      53       0      2   0.478   0.478      13.155       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.988   4.988     0.58733       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  16.178  16.178     0.30779       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 123.762 123.762      1.2174       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  20.696  20.696      1.1669       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   53   2.016\n",
      "  serious non-bonded atom clash:    14   89   2.020\n",
      "  serious non-bonded atom clash:    90   93   1.305\n",
      "  serious non-bonded atom clash:    91   93   2.165\n",
      "  serious non-bonded atom clash:    92   93   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   14   17   10   13    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       93        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       93       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      436      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       42\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.4066\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.008   0.008     0.17546       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.607   3.607      2.8517       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      0  44.329  44.329      6.6677       1.000\n",
      " 4 Stereochemical improper torsion pot:       5       0      0   1.277   1.277     0.20270       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      42       0      1   0.429   0.429      4.6851       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   9.596   9.596      2.1735       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.783   1.783    -0.20490       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 147.029 147.029      1.1620       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  23.678  23.678     0.69333       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   53   2.016\n",
      "  serious non-bonded atom clash:    14   89   2.020\n",
      "  serious non-bonded atom clash:    90   93   1.305\n",
      "  serious non-bonded atom clash:    91   93   2.165\n",
      "  serious non-bonded atom clash:    92   93   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    5   13   17    7   13    9   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       93       93\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131816    1105.289     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132344    1105.805     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180944    1153.266     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194244    1166.254     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333496    1302.242     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335096    1303.805     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339076    1307.691     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341076    1309.645     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344076    1312.574     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348576    1316.969     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354148    1322.410     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356948    1325.145     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361148    1329.246     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367448    1335.398     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370220    1338.105     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371620    1339.473     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382657    1350.251     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382733    1350.325     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382904    1350.492     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383754    1351.322     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385012    1352.551     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394852    1362.160     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379665    1347.329     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386908    1354.402     1.323\n",
      "mutate_mode_> Model residue 6 changed from PRO to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918653     897.122     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925896     904.195     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927224     905.492     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     96      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931104     909.281     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     96    127       149        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935360     913.438     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     97    129       151        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     97    129       151        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       93        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946397     924.216     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946473     924.290     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946644     924.457     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947494     925.287     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948752     926.516     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958592     936.125     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       93\n",
      "read_mo_298_> Segment:        1  1:M  13:M       93\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbARG6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbARG6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503990     492.178     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610458     596.150     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618650     604.150     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       97     129     138      36\n",
      "              Total number of restraints before, now         :        0      400\n",
      "make_re_422_> Number of previous, current restraints         :        0      400\n",
      "make_re_423_> Number of previous, current selected restraints:        0      400\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618786     604.283     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      400      407\n",
      "make_re_423_> Number of previous, current selected restraints:      400      407\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618802     604.299     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       11\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      407      415\n",
      "make_re_423_> Number of previous, current selected restraints:      407      404\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618802     604.299     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      421\n",
      "make_re_423_> Number of previous, current selected restraints:      404      410\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618802     604.299     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      421      426\n",
      "make_re_423_> Number of previous, current selected restraints:      410      415\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618802     604.299     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      426      429\n",
      "make_re_423_> Number of previous, current selected restraints:      415      418\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618802     604.299     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      429      432\n",
      "make_re_423_> Number of previous, current selected restraints:      418      421\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       61\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646702     631.545     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941634     919.564     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      432      61\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       47\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       58396.5625\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.032   0.032      4.4806       1.000\n",
      " 2 Bond angle potential               :      17       0      0   4.018   4.018      5.4730       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      1  57.029  57.029      4.8678       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.588   0.588     0.33488E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      47       0     14   1.092   1.092      58361.       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   6.112   6.112     0.88190       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  82.072  82.072      8.1302       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1 165.020 165.020      2.8523       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.028   0.028     0.24303       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.080  90.080     0.73342       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  27.936  27.936      7.4037       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   57   2.016\n",
      "  serious non-bonded atom clash:    14   93   2.020\n",
      "  serious non-bonded atom clash:    25   40   1.807\n",
      "  serious non-bonded atom clash:    26   40   1.578\n",
      "  serious non-bonded atom clash:    27   40   1.565\n",
      "  serious non-bonded atom clash:    29   38   1.917\n",
      "  serious non-bonded atom clash:    29   39   2.008\n",
      "  serious non-bonded atom clash:    29   40   1.383\n",
      "  serious non-bonded atom clash:    32   36   1.491\n",
      "  serious non-bonded atom clash:    94   97   1.305\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    11    1    0    0    2    0    3    3    8   15   17    8   15   12   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      432      61\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       11\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          15.9483\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.004   0.004     0.68006E-01   1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.175   3.175      2.9777       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      0  50.401  50.401      3.9326       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.857   0.857     0.10607       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      11       0      0   0.445   0.445      1.2103       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   9.289   9.289      2.0366       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 126.544 126.544     0.65423       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   2.087   2.087     0.35747E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 125.432 125.432      1.1017       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 137.674 137.674     0.94669       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  21.807  21.807      2.8786       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   57   2.016\n",
      "  serious non-bonded atom clash:    14   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.305\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    4   12   17    9   13   10    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133200    1106.641     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133728    1107.156     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182328    1154.617     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195628    1167.605     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334880    1303.594     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336480    1305.156     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340460    1309.043     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342460    1310.996     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345460    1313.926     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349960    1318.320     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355532    1323.762     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358332    1326.496     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362532    1330.598     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368832    1336.750     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371604    1339.457     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373004    1340.824     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384041    1351.603     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384117    1351.677     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384288    1351.844     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385138    1352.674     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386396    1353.902     1.322\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396236    1363.512     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380913    1348.548     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388292    1355.754     1.324\n",
      "mutate_mode_> Model residue 7 changed from ARG to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918789     897.255     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926168     904.461     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927496     905.758     0.885\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     96      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931376     909.547     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     96    127       149        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935632     913.703     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     97    129       151        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     97    129       151        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       97        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946669     924.481     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946745     924.556     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946916     924.723     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947766     925.553     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       949024     926.781     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958864     936.391     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbARG7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutated_match.pdbARG7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504262     492.443     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610730     596.416     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618922     604.416     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       97     129     138      36\n",
      "              Total number of restraints before, now         :        0      400\n",
      "make_re_422_> Number of previous, current restraints         :        0      400\n",
      "make_re_423_> Number of previous, current selected restraints:        0      400\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618922     604.416     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      400      407\n",
      "make_re_423_> Number of previous, current selected restraints:      400      407\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618938     604.432     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       11\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      407      415\n",
      "make_re_423_> Number of previous, current selected restraints:      407      404\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618938     604.432     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      421\n",
      "make_re_423_> Number of previous, current selected restraints:      404      410\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618938     604.432     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      421      426\n",
      "make_re_423_> Number of previous, current selected restraints:      410      415\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618938     604.432     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      426      429\n",
      "make_re_423_> Number of previous, current selected restraints:      415      418\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618938     604.432     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match7_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      429      432\n",
      "make_re_423_> Number of previous, current selected restraints:      418      421\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       60\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646838     631.678     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941770     919.697     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      432      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       22\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          23.1296\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.040   0.040      7.5028       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.798   2.798      2.4177       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      0  61.491  61.491      4.6671       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.528   0.528     0.40023E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      22       0      1   0.479   0.479      4.0020       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   7.591   7.591      1.3603       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   2.323   2.323    -0.33618       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   7.136   7.136    -0.56368E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 116.097 116.097     0.60861       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 125.276 125.276     0.82564       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  19.470  19.470      2.0981       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   57   2.016\n",
      "  serious non-bonded atom clash:    14   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.305\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    4   12   17    9   13   10    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      432      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       11\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          11.0012\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.007   0.007     0.15819       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.019   2.019      1.3108       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      0  59.474  59.474      4.3717       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.591   2.591     0.83229       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      11       0      0   0.484   0.484      1.9681       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.286   5.286     0.65947       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.917   3.917    -0.25611       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   9.258   9.258    -0.27138E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 121.231 121.231     0.64848       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  92.004  92.004     0.73013       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  15.493  15.493     0.60528       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   57   2.016\n",
      "  serious non-bonded atom clash:    14   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.305\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    5   12   13   11   12    9    8\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match7_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133336    1106.773     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133864    1107.289     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182464    1154.750     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195764    1167.738     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335016    1303.727     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336616    1305.289     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340596    1309.176     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342596    1311.129     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345596    1314.059     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350096    1318.453     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355668    1323.895     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358468    1326.629     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362668    1330.730     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368968    1336.883     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371740    1339.590     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373140    1340.957     1.310\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384177    1351.735     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384253    1351.810     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384424    1351.977     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385274    1352.807     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386532    1354.035     1.322\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396372    1363.645     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380913    1348.548     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388088    1355.555     1.324\n",
      "mutate_mode_> Model residue 1 changed from GLY to THR\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927080     905.352     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    92    94\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    92    89    94    93\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931084     909.262     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       156        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935340     913.418     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96    130       158        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     95     96    130       158        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946377     924.196     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946453     924.271     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946624     924.438     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947474     925.268     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948732     926.496     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958572     936.105     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdbTHR1.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       95       95\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/cpep_aligned.pdbTHR1.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503970     492.158     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610438     596.131     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618630     604.131     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       96     130     152      35\n",
      "              Total number of restraints before, now         :        0      413\n",
      "make_re_422_> Number of previous, current restraints         :        0      413\n",
      "make_re_423_> Number of previous, current selected restraints:        0      413\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618766     604.264     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      413      420\n",
      "make_re_423_> Number of previous, current selected restraints:      413      420\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618782     604.279     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      420      428\n",
      "make_re_423_> Number of previous, current selected restraints:      420      416\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618782     604.279     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      428      435\n",
      "make_re_423_> Number of previous, current selected restraints:      416      423\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618782     604.279     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      435      440\n",
      "make_re_423_> Number of previous, current selected restraints:      423      428\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618782     604.279     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      440      443\n",
      "make_re_423_> Number of previous, current selected restraints:      428      431\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618782     604.279     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      443      445\n",
      "make_re_423_> Number of previous, current selected restraints:      431      433\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       33\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646106     630.963     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940990     918.936     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       95       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      445      33\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       22\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          13.3323\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       7       0      0   0.034   0.034      2.1266       1.000\n",
      " 2 Bond angle potential               :      10       0      0   3.099   3.099      1.6641       1.000\n",
      " 3 Stereochemical cosine torsion poten:      11       0      0  27.827  27.827     0.80735       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   3.700   3.700     0.68799       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      22       0      2   0.522   0.522      6.9689       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.114   0.114     0.15241E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 121.184 121.184      1.0771       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    11   55   2.017\n",
      "  serious non-bonded atom clash:    17   91   2.020\n",
      "  serious non-bonded atom clash:    92   95   1.306\n",
      "  serious non-bonded atom clash:    93   95   2.165\n",
      "  serious non-bonded atom clash:    94   95   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    2    7   15   18    8   15    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       95        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       95       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      445      33\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        8\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           5.7791\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       7       0      0   0.006   0.006     0.60707E-01   1.000\n",
      " 2 Bond angle potential               :      10       0      0   1.944   1.944     0.65231       1.000\n",
      " 3 Stereochemical cosine torsion poten:      11       0      0  54.969  54.969      3.1279       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.223   1.223     0.94164E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       8       0      0   0.511   0.511     0.72573       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   4.577   4.577     0.24723       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 107.409 107.409     0.87106       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    11   55   2.017\n",
      "  serious non-bonded atom clash:    17   91   2.020\n",
      "  serious non-bonded atom clash:    92   95   1.306\n",
      "  serious non-bonded atom clash:    93   95   2.165\n",
      "  serious non-bonded atom clash:    94   95   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   14   17    8   14    7   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       95       95\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132556    1106.012     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133084    1106.527     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181684    1153.988     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194984    1166.977     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334236    1302.965     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335836    1304.527     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339816    1308.414     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341816    1310.367     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344816    1313.297     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349316    1317.691     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354888    1323.133     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357688    1325.867     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361888    1329.969     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368188    1336.121     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370960    1338.828     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372360    1340.195     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383397    1350.974     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383473    1351.048     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383644    1351.215     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384494    1352.045     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385752    1353.273     1.322\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395592    1362.883     1.331\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1380405    1348.052     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1387716    1355.191     1.323\n",
      "mutate_mode_> Model residue 4 changed from SER to GLN\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918721     897.188     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926032     904.328     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927408     905.672     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    95    97\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    95    92    97    96\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98      0         0        37\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931508     909.676     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    132       159        37\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935764     913.832     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     98     99    134       161        38\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     98     99    134       161        38\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       94        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946801     924.610     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946877     924.685     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947048     924.852     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947898     925.682     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       949156     926.910     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958996     936.520     0.915\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       95\n",
      "read_mo_298_> Segment:        1  1:M  13:M       95\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdbGLN4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       98       98\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutated_match.pdbGLN4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504394     492.572     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       98\n",
      "read_mo_298_> Segment:        1  1:M  13:M       98\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610862     596.545     0.583\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       619054     604.545     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       99     134     155      37\n",
      "              Total number of restraints before, now         :        0      425\n",
      "make_re_422_> Number of previous, current restraints         :        0      425\n",
      "make_re_423_> Number of previous, current selected restraints:        0      425\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619156     604.645     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      425      432\n",
      "make_re_423_> Number of previous, current selected restraints:      425      432\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619172     604.660     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      432      440\n",
      "make_re_423_> Number of previous, current selected restraints:      432      428\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619172     604.660     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      440      447\n",
      "make_re_423_> Number of previous, current selected restraints:      428      435\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619172     604.660     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      447      453\n",
      "make_re_423_> Number of previous, current selected restraints:      435      441\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619172     604.660     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      453      457\n",
      "make_re_423_> Number of previous, current selected restraints:      441      445\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619172     604.660     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match8_\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      457      459\n",
      "make_re_423_> Number of previous, current selected restraints:      445      447\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       54\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       647360     632.188     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       942316     920.230     0.899\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       98       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      459      54\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       23\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         110.5389\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.034   0.034      3.5353       1.000\n",
      " 2 Bond angle potential               :      15       0      0   2.559   2.559      1.9161       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  65.215  65.215      4.6945       1.000\n",
      " 4 Stereochemical improper torsion pot:       5       0      0   0.181   0.181     0.23984E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      23       0      2   0.598   0.598      98.389       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.580   0.580     0.39766E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.000   0.000     0.71664E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.56873E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.210 179.210     0.95592       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.418  23.418      1.0274       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     5   26   2.241\n",
      "  serious non-bonded atom clash:    11   58   2.017\n",
      "  serious non-bonded atom clash:    17   94   2.020\n",
      "  serious non-bonded atom clash:    95   98   1.306\n",
      "  serious non-bonded atom clash:    96   98   2.165\n",
      "  serious non-bonded atom clash:    97   98   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    1    0    1    0    1    2    7   14   17    9   14    8   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       98        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       98       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      459      54\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       27\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          12.3886\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.006   0.006     0.10757       1.000\n",
      " 2 Bond angle potential               :      15       0      0   2.178   2.178      1.4209       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  65.892  65.892      5.0420       1.000\n",
      " 4 Stereochemical improper torsion pot:       5       0      0   0.652   0.652     0.66587E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      27       0      1   0.372   0.372      2.2766       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.319   1.319     0.20535E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  13.862  13.862     0.35775       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  88.866  88.866      1.3513       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 130.868 130.868      1.2223       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.068  22.068     0.52306       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    11   58   2.017\n",
      "  serious non-bonded atom clash:    17   94   2.020\n",
      "  serious non-bonded atom clash:    95   98   1.306\n",
      "  serious non-bonded atom clash:    96   98   2.165\n",
      "  serious non-bonded atom clash:    97   98   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   16    9   15   10   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match8_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       98       98\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133882    1107.307     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1134410    1107.822     1.082\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183010    1155.283     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1196310    1168.271     1.141\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335562    1304.260     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1337162    1305.822     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341142    1309.709     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343142    1311.662     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1346142    1314.592     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350642    1318.986     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356214    1324.428     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1359014    1327.162     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1363214    1331.264     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369514    1337.416     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372286    1340.123     1.309\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373686    1341.490     1.310\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384723    1352.269     1.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384799    1352.343     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384970    1352.510     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385820    1353.340     1.322\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1387078    1354.568     1.323\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396918    1364.178     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381493    1349.114     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388668    1356.121     1.324\n",
      "mutate_mode_> Model residue 1 changed from GLY to LYS\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927080     905.352     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    94    96\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    94    91    96    95\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931112     909.289     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  LYS  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     96     97    129       157        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935368     913.445     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    131       159        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     97     98    131       159        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        6\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946405     924.224     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946481     924.298     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946652     924.465     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947502     925.295     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948760     926.523     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958600     936.133     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdbLYS1.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/cpep_aligned.pdbLYS1.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503998     492.186     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610466     596.158     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618658     604.158     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       98     131     153      34\n",
      "              Total number of restraints before, now         :        0      416\n",
      "make_re_422_> Number of previous, current restraints         :        0      416\n",
      "make_re_423_> Number of previous, current selected restraints:        0      416\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618862     604.357     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      416      423\n",
      "make_re_423_> Number of previous, current selected restraints:      416      423\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618878     604.373     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      423      431\n",
      "make_re_423_> Number of previous, current selected restraints:      423      419\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618878     604.373     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      431      438\n",
      "make_re_423_> Number of previous, current selected restraints:      419      426\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618878     604.373     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      438      444\n",
      "make_re_423_> Number of previous, current selected restraints:      426      432\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618878     604.373     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      444      448\n",
      "make_re_423_> Number of previous, current selected restraints:      432      436\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618878     604.373     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      448      451\n",
      "make_re_423_> Number of previous, current selected restraints:      436      439\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       39\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       646778     631.619     0.617\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       941710     919.639     0.898\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      451      39\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       32\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         634.6080\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.022   0.022      1.2711       1.000\n",
      " 2 Bond angle potential               :      11       0      0   1.422   1.422     0.39029       1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      0  27.155  27.155     0.85491       1.000\n",
      " 4 Stereochemical improper torsion pot:       2       0      0   0.879   0.879     0.25863E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      32       0      6   0.814   0.814      631.43       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.110   0.110     0.14318E-03   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.955 112.955     0.32821       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  13.770  13.770     0.56617E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.25728E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.000   0.000     0.22251       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     6   74   2.078\n",
      "  serious non-bonded atom clash:     6   76   2.249\n",
      "  serious non-bonded atom clash:     7   76   2.017\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     6    1    1    0    1    0    2    5    7   15   18    8   14   10   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       97        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__611W> Precision problem; starting from scratch.\n",
      "              Last such message in this model.optimize().\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       97       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      451      39\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        4\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           4.7486\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.004   0.004     0.28248E-01   1.000\n",
      " 2 Bond angle potential               :      11       0      0   1.388   1.388     0.38733       1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      0  46.253  46.253      2.1714       1.000\n",
      " 4 Stereochemical improper torsion pot:       2       0      0   0.336   0.336     0.77393E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       4       0      0   0.571   0.571     0.73017       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.034   0.034     0.13875E-04   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.159   0.159     0.66962E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  89.325  89.325      1.0900       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.25728E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   3.898   3.898     0.24107       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   57   2.017\n",
      "  serious non-bonded atom clash:    19   93   2.020\n",
      "  serious non-bonded atom clash:    94   97   1.306\n",
      "  serious non-bonded atom clash:    95   97   2.165\n",
      "  serious non-bonded atom clash:    96   97   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   17    8   13    7   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       97       97\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133276    1106.715     1.081\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1133804    1107.230     1.081\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1182404    1154.691     1.128\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1195704    1167.680     1.140\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334956    1303.668     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336556    1305.230     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340536    1309.117     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1342536    1311.070     1.280\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345536    1314.000     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1350036    1318.395     1.287\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1355608    1323.836     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1358408    1326.570     1.295\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1362608    1330.672     1.299\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1368908    1336.824     1.305\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371680    1339.531     1.308\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1373080    1340.898     1.309\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384117    1351.677     1.320\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1384193    1351.751     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384364    1351.918     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385214    1352.748     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386472    1353.977     1.322\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1396312    1363.586     1.332\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381057    1348.688     1.317\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388436    1355.895     1.324\n",
      "mutate_mode_> Model residue 4 changed from SER to ASP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918789     897.255     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       926168     904.461     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927520     905.781     0.885\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    96    98\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    96    93    98    97\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     98     99      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931620     909.785     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  LYS  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     98     99    132       159        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935876     913.941     0.893\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     99    100    134       161        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     99    100    134       161        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       96        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946913     924.720     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946989     924.794     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947160     924.961     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948010     925.791     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       949268     927.020     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       959108     936.629     0.915\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       97\n",
      "read_mo_298_> Segment:        1  1:M  13:M       97\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdbASP4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       99       99\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutated_match.pdbASP4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504506     492.682     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       99\n",
      "read_mo_298_> Segment:        1  1:M  13:M       99\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610974     596.654     0.583\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       619166     604.654     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      100     134     155      35\n",
      "              Total number of restraints before, now         :        0      424\n",
      "make_re_422_> Number of previous, current restraints         :        0      424\n",
      "make_re_423_> Number of previous, current selected restraints:        0      424\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619234     604.721     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      424      431\n",
      "make_re_423_> Number of previous, current selected restraints:      424      431\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619250     604.736     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      431      439\n",
      "make_re_423_> Number of previous, current selected restraints:      431      427\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619250     604.736     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      439      446\n",
      "make_re_423_> Number of previous, current selected restraints:      427      434\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619250     604.736     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      446      453\n",
      "make_re_423_> Number of previous, current selected restraints:      434      441\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619250     604.736     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      453      457\n",
      "make_re_423_> Number of previous, current selected restraints:      441      445\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       619250     604.736     0.591\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match11\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      457      460\n",
      "make_re_423_> Number of previous, current selected restraints:      445      448\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       49\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       647726     632.545     0.618\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       942706     920.611     0.899\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       99       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      460      49\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       19\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          17.3823\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.034   0.034      3.4033       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.589   2.589      1.8289       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.129  67.129      4.7167       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.745   2.745     0.88101       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      19       0      1   0.458   0.458      4.6469       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.651   0.651     0.50021E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.391   3.391     0.87362E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 166.138 166.138      1.0576       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.219  23.219     0.75551       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   59   2.017\n",
      "  serious non-bonded atom clash:    19   95   2.020\n",
      "  serious non-bonded atom clash:    96   99   1.306\n",
      "  serious non-bonded atom clash:    97   99   2.165\n",
      "  serious non-bonded atom clash:    98   99   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    2    7   14   16    8   14    7   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       99        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       99       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      460      49\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       19\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          10.5681\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.006   0.006     0.10190       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.100   2.100      1.2363       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.654  67.654      4.8807       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.634   0.634     0.55020E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      19       0      1   0.382   0.382      2.4174       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.493   1.493     0.26291E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   5.408   5.408     0.10072       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  56.419  56.419      1.3212       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.476  22.476     0.42860       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    13   59   2.017\n",
      "  serious non-bonded atom clash:    19   95   2.020\n",
      "  serious non-bonded atom clash:    96   99   1.306\n",
      "  serious non-bonded atom clash:    97   99   2.165\n",
      "  serious non-bonded atom clash:    98   99   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   12   16   10   14    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match11_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       99       99\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1134272    1107.688     1.082\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1134800    1108.203     1.082\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183400    1155.664     1.129\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1196700    1168.652     1.141\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335952    1304.641     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1337552    1306.203     1.276\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341532    1310.090     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343532    1312.043     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1346532    1314.973     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1351032    1319.367     1.288\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356604    1324.809     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1359404    1327.543     1.296\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1363604    1331.645     1.300\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369904    1337.797     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372676    1340.504     1.309\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1374076    1341.871     1.310\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385113    1352.649     1.321\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1385189    1352.724     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385360    1352.891     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1386210    1353.721     1.322\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1387468    1354.949     1.323\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1397308    1364.559     1.333\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1381781    1349.396     1.318\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1388956    1356.402     1.325\n",
      "mutate_mode_> Model residue 4 changed from SER to ALA\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927016     905.289     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    88    90\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    88    85    90    89\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     91      0         0        33\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930820     909.004     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     90     91    122       148        33\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935076     913.160     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    124       150        34\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    124       150        34\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       90        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946113     923.938     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946189     924.013     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946360     924.180     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947210     925.010     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948468     926.238     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958308     935.848     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdbALA4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/cpep_aligned.pdbALA4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503706     491.900     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610174     595.873     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618366     603.873     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       92     124     145      33\n",
      "              Total number of restraints before, now         :        0      394\n",
      "make_re_422_> Number of previous, current restraints         :        0      394\n",
      "make_re_423_> Number of previous, current selected restraints:        0      394\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618366     603.873     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      394      401\n",
      "make_re_423_> Number of previous, current selected restraints:      394      401\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      401      409\n",
      "make_re_423_> Number of previous, current selected restraints:      401      397\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      409      414\n",
      "make_re_423_> Number of previous, current selected restraints:      397      402\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      414      419\n",
      "make_re_423_> Number of previous, current selected restraints:      402      407\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      419      422\n",
      "make_re_423_> Number of previous, current selected restraints:      407      410\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618382     603.889     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      422      424\n",
      "make_re_423_> Number of previous, current selected restraints:      410      412\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        5       35\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644554     629.447     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939342     917.326     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      424      35\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          13.1837\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.042   0.042      3.3734       1.000\n",
      " 2 Bond angle potential               :      10       0      0   2.687   2.687      1.4798       1.000\n",
      " 3 Stereochemical cosine torsion poten:      13       0      0  76.734  76.734      4.6900       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.086   0.086     0.51891E-03   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      1   0.383   0.383      2.5838       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.579   0.579     0.39583E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.341  23.341      1.0521       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   51   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   14   17    8   14    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       91        5        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       91       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      424      35\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       18\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           8.9714\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.005   0.005     0.59652E-01   1.000\n",
      " 2 Bond angle potential               :      10       0      0   2.381   2.381      1.1417       1.000\n",
      " 3 Stereochemical cosine torsion poten:      13       0      0  76.659  76.659      4.6718       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.445   0.445     0.21688E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      18       0      1   0.355   0.355      2.4776       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.069   1.069     0.13482E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  22.124  22.124     0.58536       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   51   2.017\n",
      "  serious non-bonded atom clash:    14   87   2.020\n",
      "  serious non-bonded atom clash:    88   91   1.306\n",
      "  serious non-bonded atom clash:    89   91   2.165\n",
      "  serious non-bonded atom clash:    90   91   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   18    8   13    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       91       91\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1130908    1104.402     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131436    1104.918     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180036    1152.379     1.125\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193336    1165.367     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332588    1301.355     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334188    1302.918     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338168    1306.805     1.276\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340168    1308.758     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343168    1311.688     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347668    1316.082     1.285\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353240    1321.523     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356040    1324.258     1.293\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360240    1328.359     1.297\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366540    1334.512     1.303\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369312    1337.219     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370712    1338.586     1.307\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381749    1349.364     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1381825    1349.438     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1381996    1349.605     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382846    1350.436     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384104    1351.664     1.320\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1393944    1361.273     1.329\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1378893    1346.575     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386068    1353.582     1.322\n",
      "mutate_mode_> Model residue 5 changed from ASP to LYS\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927008     905.281     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    89    91\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    89    86    91    90\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92      0         0        32\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930828     909.012     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    122       149        32\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935084     913.168     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       89        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946121     923.946     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946197     924.021     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946368     924.188     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947218     925.018     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948476     926.246     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958316     935.855     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbLYS5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbLYS5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503714     491.908     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610182     595.881     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618374     603.881     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       93     124     146      32\n",
      "              Total number of restraints before, now         :        0      395\n",
      "make_re_422_> Number of previous, current restraints         :        0      395\n",
      "make_re_423_> Number of previous, current selected restraints:        0      395\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618408     603.914     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      395      402\n",
      "make_re_423_> Number of previous, current selected restraints:      395      402\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618424     603.930     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      402      410\n",
      "make_re_423_> Number of previous, current selected restraints:      402      398\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618424     603.930     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      410      415\n",
      "make_re_423_> Number of previous, current selected restraints:      398      403\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618424     603.930     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      420\n",
      "make_re_423_> Number of previous, current selected restraints:      403      408\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618424     603.930     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      420      424\n",
      "make_re_423_> Number of previous, current selected restraints:      408      412\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618424     603.930     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      424      427\n",
      "make_re_423_> Number of previous, current selected restraints:      412      415\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       60\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644884     629.770     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939696     917.672     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       52\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         527.9212\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.027   0.027      2.0421       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.977   3.977      3.9594       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      0  46.756  46.756      6.5975       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.458   0.458     0.18559E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      52       0      5   0.708   0.708      513.41       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.657   4.657     0.51189       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  13.664  13.664     0.22121       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  13.770  13.770     0.72380E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.19573E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.000   0.000     0.22251       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  19.482  19.482     0.84735       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    26   44   2.223\n",
      "  serious non-bonded atom clash:    26   45   2.284\n",
      "  serious non-bonded atom clash:    26   46   2.089\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    2    0    1    0    1    2    7   12   19    8   14    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       92        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       38\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          20.6679\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.007   0.007     0.14536       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.465   3.465      2.6540       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      0  43.405  43.405      6.8929       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.302   2.302     0.41204       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      38       0      0   0.472   0.472      6.2473       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   9.475   9.475      2.1190       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  15.789  15.789     0.32836       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  93.950  93.950     0.73584       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.079   0.079     0.20520E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 118.519 118.519     0.95568       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  22.602  22.602     0.15685       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    5   12   18    5   13    8   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131262    1104.748     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131790    1105.264     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180390    1152.725     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193690    1165.713     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332942    1301.701     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334542    1303.264     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338522    1307.150     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340522    1309.104     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343522    1312.033     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348022    1316.428     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353594    1321.869     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356394    1324.604     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360594    1328.705     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366894    1334.857     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369666    1337.564     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371066    1338.932     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382103    1349.710     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382179    1349.784     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382350    1349.951     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383200    1350.781     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384458    1352.010     1.320\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394298    1361.619     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379213    1346.888     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386422    1353.928     1.322\n",
      "mutate_mode_> Model residue 6 changed from PRO to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918619     897.089     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925828     904.129     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927076     905.348     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    89    91\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    89    86    91    90\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92      0         0        32\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930896     909.078     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    122       149        32\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935152     913.234     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       92        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946189     924.013     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946265     924.087     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946436     924.254     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947286     925.084     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948544     926.312     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958384     935.922     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbPRO6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbPRO6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503782     491.975     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610250     595.947     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618442     603.947     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       93     124     146      32\n",
      "              Total number of restraints before, now         :        0      395\n",
      "make_re_422_> Number of previous, current restraints         :        0      395\n",
      "make_re_423_> Number of previous, current selected restraints:        0      395\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618442     603.947     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      395      402\n",
      "make_re_423_> Number of previous, current selected restraints:      395      402\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      402      410\n",
      "make_re_423_> Number of previous, current selected restraints:      402      398\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      410      415\n",
      "make_re_423_> Number of previous, current selected restraints:      398      403\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      420\n",
      "make_re_423_> Number of previous, current selected restraints:      403      408\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      420      424\n",
      "make_re_423_> Number of previous, current selected restraints:      408      412\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      424      427\n",
      "make_re_423_> Number of previous, current selected restraints:      412      415\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       65\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644918     629.803     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939730     917.705     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          26.2247\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.030   0.030      2.5831       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.097   4.097      4.1079       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      1  48.423  48.423      9.5549       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.396   0.396     0.15177E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      1   0.498   0.498      4.1437       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   6.466   6.466     0.98690       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  15.384  15.384     0.34732       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  18.520  18.520      1.7039       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  23.252  23.252      1.7233       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  23.148  23.148      1.0585       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    5   12   18    5   13    8   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       92        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.0425\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.008   0.008     0.15721       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.409   4.409      5.1701       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       1      1  47.895  47.895      10.729       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.680   0.680     0.48890E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.505   0.505      1.3687       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.275   4.275     0.43140       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  14.797  14.797     0.30992       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   8.372   8.372     0.20705E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   6.328   6.328    -0.25146       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  21.133  21.133     0.57744E-01   1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    280   6P   6P CA  CB     32   34  -25.11   34.59  -59.70    4.96   34.59  -59.70    4.96\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    4   11   17    7   13    9    8\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131296    1104.781     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131824    1105.297     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180424    1152.758     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193724    1165.746     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332976    1301.734     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334576    1303.297     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338556    1307.184     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340556    1309.137     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343556    1312.066     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348056    1316.461     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353628    1321.902     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356428    1324.637     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360628    1328.738     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366928    1334.891     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369700    1337.598     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371100    1338.965     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382137    1349.743     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382213    1349.817     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382384    1349.984     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383234    1350.814     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384492    1352.043     1.320\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394332    1361.652     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379213    1346.888     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386422    1353.928     1.322\n",
      "mutate_mode_> Model residue 7 changed from ARG to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918619     897.089     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925828     904.129     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927076     905.348     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    89    91\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    89    86    91    90\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92      0         0        32\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930896     909.078     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     91     92    122       149        32\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935152     913.234     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     92     93    124       151        33\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       92        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946189     924.013     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946265     924.087     0.902\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946436     924.254     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947286     925.084     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948544     926.312     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958384     935.922     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbARG7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutated_match.pdbARG7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503782     491.975     0.480\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       92\n",
      "read_mo_298_> Segment:        1  1:M  13:M       92\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610250     595.947     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618442     603.947     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       93     124     146      32\n",
      "              Total number of restraints before, now         :        0      395\n",
      "make_re_422_> Number of previous, current restraints         :        0      395\n",
      "make_re_423_> Number of previous, current selected restraints:        0      395\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618442     603.947     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      395      402\n",
      "make_re_423_> Number of previous, current selected restraints:      395      402\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      402      410\n",
      "make_re_423_> Number of previous, current selected restraints:      402      398\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      410      415\n",
      "make_re_423_> Number of previous, current selected restraints:      398      403\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      415      420\n",
      "make_re_423_> Number of previous, current selected restraints:      403      408\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      420      424\n",
      "make_re_423_> Number of previous, current selected restraints:      408      412\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618458     603.963     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match12\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      424      427\n",
      "make_re_423_> Number of previous, current selected restraints:      412      415\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       63\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       644918     629.803     0.615\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       939730     917.705     0.896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.6489\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.040   0.040      7.4608       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.744   2.744      2.3659       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      0  61.573  61.573      4.3014       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.213   0.213     0.57543E-02   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      1   0.476   0.476      3.7351       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.648   1.648     0.64101E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   2.299   2.299    -0.33674       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   7.146   7.146    -0.56239E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 116.089 116.089     0.60856       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 125.274 125.274     0.82563       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  21.845  21.845    -0.32529       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    4   11   17    7   13    9    8\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       92       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       92      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      427      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       11\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           8.3304\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.004   0.004     0.55299E-01   1.000\n",
      " 2 Bond angle potential               :      17       0      0   1.311   1.311     0.60075       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      0  59.575  59.575      4.0450       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.186   1.186     0.20474       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      11       0      0   0.382   0.382      1.1904       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.011   1.011     0.24143E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 129.286 129.286     0.46559       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 100.026 100.026      1.6555       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 117.942 117.942     0.46849       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 103.293 103.293     0.72272       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  18.784  18.784     -1.1022       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   52   2.017\n",
      "  serious non-bonded atom clash:    14   88   2.020\n",
      "  serious non-bonded atom clash:    89   92   1.306\n",
      "  serious non-bonded atom clash:    90   92   2.165\n",
      "  serious non-bonded atom clash:    91   92   1.593\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    5   11   15    7   12   10    7\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match12_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       92       92\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131296    1104.781     1.079\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1131824    1105.297     1.079\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180424    1152.758     1.126\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1193724    1165.746     1.138\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332976    1301.734     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1334576    1303.297     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1338556    1307.184     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1340556    1309.137     1.278\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343556    1312.066     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348056    1316.461     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1353628    1321.902     1.291\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1356428    1324.637     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1360628    1328.738     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1366928    1334.891     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1369700    1337.598     1.306\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371100    1338.965     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382137    1349.743     1.318\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1382213    1349.817     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382384    1349.984     1.318\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383234    1350.814     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384492    1352.043     1.320\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "rdpdb___459W> Residue type  NH2 not recognized. 'AutoModel' model building\n",
      "              will treat this residue as a rigid body.\n",
      "              To use real parameters, add the residue type to ${LIB}/restyp.lib,\n",
      "              its topology to ${LIB}/top_*.lib, and suitable forcefield\n",
      "              parameters to ${LIB}/par.lib.\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1394332    1361.652     1.330\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379213    1346.888     1.315\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1386388    1353.895     1.322\n",
      "mutate_mode_> Model residue 4 changed from SER to ASP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918585     897.056     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925760     904.062     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927056     905.328     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    91    93\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    91    88    93    92\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       930972     909.152     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    126       152        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935228     913.309     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       154        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       154        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       90        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946265     924.087     0.902\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946341     924.161     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946512     924.328     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947362     925.158     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948620     926.387     0.905\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb, Line: 92\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958460     935.996     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       91\n",
      "read_mo_298_> Segment:        1  1:M  13:M       91\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdbASP4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/cpep_aligned.pdbASP4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       503858     492.049     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610326     596.021     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618518     604.021     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       95     128     149      34\n",
      "              Total number of restraints before, now         :        0      406\n",
      "make_re_422_> Number of previous, current restraints         :        0      406\n",
      "make_re_423_> Number of previous, current selected restraints:        0      406\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618620     604.121     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      406      413\n",
      "make_re_423_> Number of previous, current selected restraints:      406      413\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618636     604.137     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      413      421\n",
      "make_re_423_> Number of previous, current selected restraints:      413      409\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618636     604.137     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      421      427\n",
      "make_re_423_> Number of previous, current selected restraints:      409      415\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618636     604.137     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      427      433\n",
      "make_re_423_> Number of previous, current selected restraints:      415      421\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618636     604.137     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      433      436\n",
      "make_re_423_> Number of previous, current selected restraints:      421      424\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618636     604.137     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      436      438\n",
      "make_re_423_> Number of previous, current selected restraints:      424      426\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       49\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645672     630.539     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940532     918.488     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      438      49\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       27\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          17.1775\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.034   0.034      3.3675       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.580   2.580      1.8149       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  67.141  67.141      4.7176       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.760   2.760     0.89068       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      27       0      1   0.425   0.425      4.4700       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.634   0.634     0.47389E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.419   3.419     0.10312       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 166.158 166.158      1.0574       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.237  23.237     0.75154       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   54   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.021\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    2    2    7   15   17    8   15    8   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       94        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      438      49\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       23\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          10.8160\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.005   0.005     0.70538E-01   1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.253   2.253      1.4447       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  69.409  69.409      5.3800       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.792   1.792     0.23985       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      23       0      1   0.326   0.326      1.3477       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   5.175   5.175     0.31606       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  95.324  95.324     0.46168       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  29.230  29.230     0.66818       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  23.672  23.672     0.88727       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   54   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.021\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    7   13   17   10   13    7   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132098    1105.564     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132626    1106.080     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181226    1153.541     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194526    1166.529     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333778    1302.518     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335378    1304.080     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339358    1307.967     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341358    1309.920     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344358    1312.850     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348858    1317.244     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354430    1322.686     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357230    1325.420     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361430    1329.521     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367730    1335.674     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370502    1338.381     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1371902    1339.748     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1382939    1350.526     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383015    1350.601     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383186    1350.768     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384036    1351.598     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385294    1352.826     1.321\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1395134    1362.436     1.331\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1379981    1347.638     1.316\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1387258    1354.744     1.323\n",
      "mutate_mode_> Model residue 5 changed from ASP to ILE\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       918687     897.155     0.876\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       925964     904.262     0.883\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       927260     905.527     0.884\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     +N\n",
      "              atom indices         :    91    93\n",
      "\n",
      "getf_______W> Skipping RTF peptide bond restraint to a HETATM (NH2)\n",
      "              that does not appear to be a modified amino acid\n",
      "              residue type, indices:    34    12\n",
      "              atom names           : C     CA    +N    O\n",
      "              atom indices         :    91    88    93    92\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94      0         0        34\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       931208     909.383     0.888\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  GLY  ;  GLYP\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     93     94    126       154        34\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       935464     913.539     0.892\n",
      "patch_______> segment topology patched using RTF:  12 ;  CSS  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       156        35\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       13     94     95    128       156        35\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       91        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946501     924.317     0.903\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       946577     924.392     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       946748     924.559     0.903\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       947598     925.389     0.904\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       948856     926.617     0.905\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       958696     936.227     0.914\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdbILE5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutated_match.pdbILE5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       504094     492.279     0.481\n",
      "read_mo_297_> Segments, residues, atoms:        1       13       94\n",
      "read_mo_298_> Segment:        1  1:M  13:M       94\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       610562     596.252     0.582\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       618754     604.252     0.590\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       95     128     151      34\n",
      "              Total number of restraints before, now         :        0      408\n",
      "make_re_422_> Number of previous, current restraints         :        0      408\n",
      "make_re_423_> Number of previous, current selected restraints:        0      408\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618754     604.252     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        1       11  ARG\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      408      415\n",
      "make_re_423_> Number of previous, current selected restraints:      408      415\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618770     604.268     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       12\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      415      423\n",
      "make_re_423_> Number of previous, current selected restraints:      415      411\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618770     604.268     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      423      429\n",
      "make_re_423_> Number of previous, current selected restraints:      411      417\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618770     604.268     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      429      435\n",
      "make_re_423_> Number of previous, current selected restraints:      417      423\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618770     604.268     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      435      438\n",
      "make_re_423_> Number of previous, current selected restraints:      423      426\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       618770     604.268     0.590\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match14\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      438      440\n",
      "make_re_423_> Number of previous, current selected restraints:      426      428\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       59\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       645806     630.670     0.616\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       940666     918.619     0.897\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      440      59\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       62\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       39250.3516\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.029   0.029      2.1756       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.065   4.065      4.1042       1.000\n",
      " 3 Stereochemical cosine torsion poten:      24       0      0  46.730  46.730      6.6970       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   3.554   3.554     0.86512       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      62       0     13   0.850   0.850      39233.       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.875   5.875     0.81458       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 116.229 116.229      1.2875       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  13.798  13.798     0.10898       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  21.683  21.683      1.0373       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   54   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.021\n",
      "  serious non-bonded atom clash:    30   45   1.935\n",
      "  serious non-bonded atom clash:    30   46   1.839\n",
      "  serious non-bonded atom clash:    30   48   1.368\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     7    1    0    0    1    2    1    6    8   13   19    8   11    8   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       94        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       13\n",
      "Number of all, selected real atoms                :       94       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      440      59\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       41\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          27.8300\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.014   0.014     0.47419       1.000\n",
      " 2 Bond angle potential               :      15       0      0   5.538   5.538      7.2542       1.000\n",
      " 3 Stereochemical cosine torsion poten:      24       0      0  45.207  45.207      7.0183       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.529   2.529     0.55148       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      41       0      2   0.520   0.520      9.8343       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0  10.319  10.319      2.5136       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   4.118   4.118    -0.37347       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.558   0.558    -0.20818       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  22.775  22.775     0.76546       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     8   54   2.017\n",
      "  serious non-bonded atom clash:    14   90   2.021\n",
      "  serious non-bonded atom clash:    91   94   1.306\n",
      "  serious non-bonded atom clash:    92   94   2.165\n",
      "  serious non-bonded atom clash:    93   94   1.594\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    1    0    0    1    0    1    2    4   12   19    9   11    6   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match14_1cnl/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       13       94       94\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132232    1105.695     1.080\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1132760    1106.211     1.080\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181360    1153.672     1.127\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1194660    1166.660     1.139\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1333912    1302.648     1.272\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335512    1304.211     1.274\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339492    1308.098     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1341492    1310.051     1.279\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1344492    1312.980     1.282\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348992    1317.375     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1354564    1322.816     1.292\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1357364    1325.551     1.294\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1361564    1329.652     1.298\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1367864    1335.805     1.304\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1370636    1338.512     1.307\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1372036    1339.879     1.308\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383073    1350.657     1.319\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1383149    1350.731     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1383320    1350.898     1.319\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1384170    1351.729     1.320\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1385428    1352.957     1.321\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1387332    1354.816     1.323\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1400756    1367.926     1.336\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1385588    1353.113     1.321\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1395028    1362.332     1.330\n",
      "mutate_mode_> Model residue 12 changed from VAL to THR\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       926452     904.738     0.884\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       935892     913.957     0.893\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       938020     916.035     0.895\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136      0         0        65\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943936     921.812     0.900\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136    186       232        65\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       950032     927.766     0.906\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      129        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       961069     938.544     0.917\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       961145     938.618     0.917\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       961316     938.785     0.917\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       962166     939.615     0.918\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       963424     940.844     0.919\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       965328     942.703     0.921\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       978752     955.812     0.933\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdbTHR12.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/cpep_aligned.pdbTHR12.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524150     511.865     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630618     615.838     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638810     623.838     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655194     639.838     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      137     188     235      64\n",
      "              Total number of restraints before, now         :        0      624\n",
      "make_re_422_> Number of previous, current restraints         :        0      624\n",
      "make_re_423_> Number of previous, current selected restraints:        0      624\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655228     639.871     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      624      636\n",
      "make_re_423_> Number of previous, current selected restraints:      624      636\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655244     639.887     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      636      649\n",
      "make_re_423_> Number of previous, current selected restraints:      636      632\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655244     639.887     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      649      662\n",
      "make_re_423_> Number of previous, current selected restraints:      632      645\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655244     639.887     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      662      671\n",
      "make_re_423_> Number of previous, current selected restraints:      645      654\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655244     639.887     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      671      675\n",
      "make_re_423_> Number of previous, current selected restraints:      654      658\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655244     639.887     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      675      676\n",
      "make_re_423_> Number of previous, current selected restraints:      658      659\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       47\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692936     676.695     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988684     965.512     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      47\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          88.6589\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.026   0.026      1.4146       1.000\n",
      " 2 Bond angle potential               :      13       0      0   2.724   2.724      2.2366       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      1  56.939  56.939      5.0132       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       1      1  34.465  34.465      79.934       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      0   0.260   0.260     -1.9914       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.772   1.772     0.74092E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 121.140 121.140      1.0351       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  30.737  56.636     0.94260       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    3    7    7    6   11   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      47\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       20\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           8.6579\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.005   0.005     0.58519E-01   1.000\n",
      " 2 Bond angle potential               :      13       0      0   1.892   1.892     0.92444       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      1  56.652  56.652      5.2757       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.208   2.208     0.43300       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      20       0      0   0.294   0.294    -0.56070       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   5.944   5.944     0.83405       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 111.616 111.616     0.82595       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  30.560  56.662     0.86692       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    4    3    8    6    9    9   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180250    1152.588     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180778    1153.104     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229378    1200.564     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242678    1213.553     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381930    1349.541     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383530    1351.104     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387510    1354.990     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389510    1356.943     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392510    1359.873     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397010    1364.268     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402582    1369.709     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405382    1372.443     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409582    1376.545     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415882    1382.697     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418654    1385.404     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420054    1386.771     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431091    1397.550     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431167    1397.624     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431338    1397.791     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432188    1398.621     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433446    1399.850     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435350    1401.709     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448774    1414.818     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429132    1395.637     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438606    1404.889     1.372\n",
      "mutate_mode_> Model residue 14 changed from LYS to LYS\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943576     921.461     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136      0         0        65\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949492     927.238     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136    186       232        65\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955588     933.191     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      131        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966625     943.970     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966701     944.044     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966872     944.211     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967722     945.041     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968980     946.270     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970884     948.129     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984308     961.238     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdbLYS14.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdbLYS14.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524218     511.932     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630686     615.904     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638878     623.904     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655262     639.904     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      137     188     235      64\n",
      "              Total number of restraints before, now         :        0      624\n",
      "make_re_422_> Number of previous, current restraints         :        0      624\n",
      "make_re_423_> Number of previous, current selected restraints:        0      624\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655262     639.904     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      624      636\n",
      "make_re_423_> Number of previous, current selected restraints:      624      636\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      636      649\n",
      "make_re_423_> Number of previous, current selected restraints:      636      632\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      649      662\n",
      "make_re_423_> Number of previous, current selected restraints:      632      645\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      662      671\n",
      "make_re_423_> Number of previous, current selected restraints:      645      654\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      671      675\n",
      "make_re_423_> Number of previous, current selected restraints:      654      658\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      675      676\n",
      "make_re_423_> Number of previous, current selected restraints:      658      659\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       58\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692970     676.729     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988718     965.545     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       26\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          29.2965\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.026   0.026      1.6829       1.000\n",
      " 2 Bond angle potential               :      15       0      1   4.419   4.419      5.2399       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      1  52.688  52.688      7.9754       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   3.180   3.180     0.95357       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      26       0      1   0.495   0.495      7.7964       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.769   0.769     0.69881E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  14.059  14.059     0.99266E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  25.672  25.672      1.3266       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 111.853 111.853      1.0516       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 115.607 115.607      1.1194       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  39.405  39.405      2.0446       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    4    3    8    6    9    9   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          16.8752\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.009   0.009     0.26681       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.116   4.116      3.7725       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      1  45.976  45.976      6.9469       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.722   1.722     0.26214       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      1   0.439   0.439      3.0373       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   4.650   4.650     0.25523       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 119.705 119.705      1.2493       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.805   0.805    -0.25299       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.15241       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.040   0.040     0.22268       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  35.324  35.324     0.96283       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    0    1    3    4    6    7    7   10   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180284    1152.621     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180812    1153.137     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229412    1200.598     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242712    1213.586     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381964    1349.574     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383564    1351.137     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387544    1355.023     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389544    1356.977     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392544    1359.906     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397044    1364.301     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402616    1369.742     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405416    1372.477     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409616    1376.578     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415916    1382.730     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418688    1385.438     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420088    1386.805     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431125    1397.583     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431201    1397.657     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431372    1397.824     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432222    1398.654     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433480    1399.883     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435384    1401.742     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448808    1414.852     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429132    1395.637     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438606    1404.889     1.372\n",
      "mutate_mode_> Model residue 15 changed from PRO to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943576     921.461     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136      0         0        65\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949492     927.238     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136    186       232        65\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955588     933.191     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      131        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966625     943.970     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966701     944.044     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966872     944.211     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967722     945.041     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968980     946.270     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970884     948.129     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984308     961.238     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdbPRO15.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutated_match.pdbPRO15.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524218     511.932     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630686     615.904     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638878     623.904     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655262     639.904     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      137     188     235      64\n",
      "              Total number of restraints before, now         :        0      624\n",
      "make_re_422_> Number of previous, current restraints         :        0      624\n",
      "make_re_423_> Number of previous, current selected restraints:        0      624\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655262     639.904     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      624      636\n",
      "make_re_423_> Number of previous, current selected restraints:      624      636\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      636      649\n",
      "make_re_423_> Number of previous, current selected restraints:      636      632\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      649      662\n",
      "make_re_423_> Number of previous, current selected restraints:      632      645\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      662      671\n",
      "make_re_423_> Number of previous, current selected restraints:      645      654\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      671      675\n",
      "make_re_423_> Number of previous, current selected restraints:      654      658\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655278     639.920     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match17\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      675      676\n",
      "make_re_423_> Number of previous, current selected restraints:      658      659\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       64\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692970     676.729     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988718     965.545     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          31.7783\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      1   0.041   0.041      3.6668       1.000\n",
      " 2 Bond angle potential               :      15       0      1   6.080   6.080      10.279       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       1      1  38.553  38.553      14.537       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.068   2.068     0.22906       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      0   0.476   0.476      1.0244       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.505   3.505     0.29004       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.790   1.790    -0.29880       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  14.348  14.348     0.46749       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  12.447  12.447     0.62131       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  35.316  35.316     0.96241       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    547  15P  15P CA  CB    116  118  -47.83   34.59  -82.42    8.68   34.59  -82.42    8.68\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    0    1    3    4    6    7    7   10   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      676      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          12.5984\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.005   0.005     0.54888E-01   1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.216   4.216      4.5158       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      0  38.791  38.791      6.1756       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.915   0.915     0.88333E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.417   0.417     0.48426       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.152   3.152     0.23458       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  53.416  53.416     0.83005E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  62.874  62.874     0.11579       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  56.836  56.836     0.26990       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  34.604  34.604     0.57623       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    0    1    3    4    5    6    8   11   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match17_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180284    1152.621     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180812    1153.137     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229412    1200.598     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242712    1213.586     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381964    1349.574     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383564    1351.137     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387544    1355.023     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389544    1356.977     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392544    1359.906     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397044    1364.301     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402616    1369.742     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405416    1372.477     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409616    1376.578     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415916    1382.730     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418688    1385.438     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420088    1386.805     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431125    1397.583     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431201    1397.657     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431372    1397.824     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432222    1398.654     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433480    1399.883     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435384    1401.742     1.369\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448808    1414.852     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429132    1395.637     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438572    1404.855     1.372\n",
      "mutate_mode_> Model residue 7 changed from GLY to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943596     921.480     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    137    143      0         0        67\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949780     927.520     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    137    143    195       242        67\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955876     933.473     0.912\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    138    144    197       244        68\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    138    144    197       244        68\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      130        8\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966913     944.251     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966989     944.325     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967160     944.492     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968010     945.322     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969268     946.551     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       971172     948.410     0.926\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984596     961.520     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdbARG7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      138      138\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/cpep_aligned.pdbARG7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524506     512.213     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630974     616.186     0.602\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       639166     624.186     0.610\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655550     640.186     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      144     197     243      66\n",
      "              Total number of restraints before, now         :        0      650\n",
      "make_re_422_> Number of previous, current restraints         :        0      650\n",
      "make_re_423_> Number of previous, current selected restraints:        0      650\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655822     640.451     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      650      662\n",
      "make_re_423_> Number of previous, current selected restraints:      650      662\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655838     640.467     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      662      675\n",
      "make_re_423_> Number of previous, current selected restraints:      662      658\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655838     640.467     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      675      689\n",
      "make_re_423_> Number of previous, current selected restraints:      658      672\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655838     640.467     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      689      699\n",
      "make_re_423_> Number of previous, current selected restraints:      672      682\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655838     640.467     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      699      704\n",
      "make_re_423_> Number of previous, current selected restraints:      682      687\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655838     640.467     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      704      706\n",
      "make_re_423_> Number of previous, current selected restraints:      687      689\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       67\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       695546     679.244     0.663\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       991462     968.225     0.946\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      138      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      706      67\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         123.3166\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.022   0.022      2.1987       1.000\n",
      " 2 Bond angle potential               :      18       0      0   3.329   3.329      2.8718       1.000\n",
      " 3 Stereochemical cosine torsion poten:      24       0      0  29.254  29.254      5.6421       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.971   1.971     0.42697       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      6   0.809   0.809      105.84       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.525   1.525     0.54873E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.955 112.955     0.58554       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000    -0.42340E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 114.592 114.592     0.50548       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.056  90.056     0.73346       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  34.795  46.200      4.5045       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  110   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    1    0    1    0    1    5    4    7    8    7   11   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      138       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      138      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      706      67\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       28\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          42.5839\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.011   0.011     0.41261       1.000\n",
      " 2 Bond angle potential               :      18       0      0   4.961   4.961      6.4362       1.000\n",
      " 3 Stereochemical cosine torsion poten:      24       0      0  37.546  37.546      6.8580       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   5.329   5.329      1.9486       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      28       0      2   0.639   0.639      14.403       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  12.194  12.194      3.5100       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  39.554  39.554      3.0700       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  21.443  21.443     0.41428       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 164.137 164.137     0.99967       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 127.761 127.761     0.84396       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  33.805  53.130      3.6876       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  110   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    4    5    6    7    7   16   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      138      138\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183028    1155.301     1.128\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183556    1155.816     1.129\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1232156    1203.277     1.175\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1245456    1216.266     1.188\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1384708    1352.254     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386308    1353.816     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390288    1357.703     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392288    1359.656     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395288    1362.586     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1399788    1366.980     1.335\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405360    1372.422     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408160    1375.156     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1412360    1379.258     1.347\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418660    1385.410     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1421432    1388.117     1.356\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1422832    1389.484     1.357\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433869    1400.263     1.367\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1433945    1400.337     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434116    1400.504     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434966    1401.334     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1436224    1402.562     1.370\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1438128    1404.422     1.372\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1451552    1417.531     1.384\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1431672    1398.117     1.365\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1441384    1407.602     1.375\n",
      "mutate_mode_> Model residue 8 changed from PRO to VAL\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       932212     910.363     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941924     919.848     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       944132     922.004     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    137    142      0         0        67\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       950196     927.926     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    137    142    193       236        67\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       956292     933.879     0.912\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    138    143    195       238        68\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    138    143    195       238        68\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      136        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967329     944.657     0.923\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       967405     944.731     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967576     944.898     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968426     945.729     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969684     946.957     0.925\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       971588     948.816     0.927\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       985012     961.926     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbVAL8.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      138      138\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbVAL8.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524922     512.619     0.501\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       631390     616.592     0.602\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       639582     624.592     0.610\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655966     640.592     0.626\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      143     195     230      67\n",
      "              Total number of restraints before, now         :        0      635\n",
      "make_re_422_> Number of previous, current restraints         :        0      635\n",
      "make_re_423_> Number of previous, current selected restraints:        0      635\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655966     640.592     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      635      647\n",
      "make_re_423_> Number of previous, current selected restraints:      635      647\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655982     640.607     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       16\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      647      660\n",
      "make_re_423_> Number of previous, current selected restraints:      647      644\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655982     640.607     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      660      674\n",
      "make_re_423_> Number of previous, current selected restraints:      644      658\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655982     640.607     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      674      683\n",
      "make_re_423_> Number of previous, current selected restraints:      658      667\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655982     640.607     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      683      687\n",
      "make_re_423_> Number of previous, current selected restraints:      667      671\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655982     640.607     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      687      689\n",
      "make_re_423_> Number of previous, current selected restraints:      671      673\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       48\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       695690     679.385     0.663\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       991606     968.365     0.946\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      138       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      689      48\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          45.0920\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.025   0.025      2.4565       1.000\n",
      " 2 Bond angle potential               :      13       0      1   4.922   4.922      6.3631       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  57.623  57.623      4.6086       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.906   2.906     0.62204       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.374   0.374     0.68007       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  11.721  11.721      3.2430       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.028   0.028    -0.11496       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      2  44.311  52.397      27.234       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    642   8V   9L C   N      69   71 -109.29  -63.50   48.05    8.77  -63.50   48.05    8.77\n",
      "    1          9L   9L N   CA     71   72  -55.75  -41.20                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  110   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    5    6    8    7   16   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      138        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      138       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      689      48\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          25.2476\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.012   0.012     0.36845       1.000\n",
      " 2 Bond angle potential               :      13       0      0   3.391   3.391      3.1789       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  48.023  48.023      3.4952       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.044   2.044     0.55532       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      1   0.411   0.411      4.5827       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0  11.326  11.326      3.0278       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   0.000   0.000    -0.15224       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  33.577  43.736      10.192       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  110   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    3    4    7    8    7   17   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      138      138\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183172    1155.441     1.128\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1183700    1155.957     1.129\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1232300    1203.418     1.175\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1245600    1216.406     1.188\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1384852    1352.395     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386452    1353.957     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390432    1357.844     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392432    1359.797     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395432    1362.727     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1399932    1367.121     1.335\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405504    1372.562     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408304    1375.297     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1412504    1379.398     1.347\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418804    1385.551     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1421576    1388.258     1.356\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1422976    1389.625     1.357\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434013    1400.403     1.368\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1434089    1400.478     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434260    1400.645     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435110    1401.475     1.369\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1436368    1402.703     1.370\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1438272    1404.562     1.372\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1451696    1417.672     1.384\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1431544    1397.992     1.365\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1441256    1407.477     1.374\n",
      "mutate_mode_> Model residue 9 changed from LEU to TRP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       932212     910.363     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941924     919.848     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       944292     922.160     0.901\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    143    150      0         0        73\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       950820     928.535     0.907\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    143    150    205       256        73\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       956916     934.488     0.913\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    144    151    207       258        74\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    144    151    207       258        74\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      138        6\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967953     945.267     0.923\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       968029     945.341     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968200     945.508     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969050     946.338     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970308     947.566     0.925\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       972212     949.426     0.927\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       985636     962.535     0.940\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      138\n",
      "read_mo_298_> Segment:        1  1:M  16:M      138\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbTRP9.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      144      144\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbTRP9.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       525546     513.229     0.501\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      144\n",
      "read_mo_298_> Segment:        1  1:M  16:M      144\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       632014     617.201     0.603\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       640206     625.201     0.611\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       656590     641.201     0.626\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      151     207     250      73\n",
      "              Total number of restraints before, now         :        0      681\n",
      "make_re_422_> Number of previous, current restraints         :        0      681\n",
      "make_re_423_> Number of previous, current selected restraints:        0      681\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656794     641.400     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      681      693\n",
      "make_re_423_> Number of previous, current selected restraints:      681      693\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656810     641.416     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       16\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      693      706\n",
      "make_re_423_> Number of previous, current selected restraints:      693      690\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656810     641.416     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      706      720\n",
      "make_re_423_> Number of previous, current selected restraints:      690      704\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656810     641.416     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      720      729\n",
      "make_re_423_> Number of previous, current selected restraints:      704      713\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656810     641.416     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      729      733\n",
      "make_re_423_> Number of previous, current selected restraints:      713      717\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       656810     641.416     0.626\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      733      735\n",
      "make_re_423_> Number of previous, current selected restraints:      717      719\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       14       97\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       698246     681.881     0.666\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       994306     971.002     0.948\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      144      14\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      735      97\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       25\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         230.7905\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      17       2      3   0.058   0.058      17.443       1.000\n",
      " 2 Bond angle potential               :      26       1      3   5.883   5.883      18.498       1.000\n",
      " 3 Stereochemical cosine torsion poten:      37       1      4  52.549  52.549      20.468       1.000\n",
      " 4 Stereochemical improper torsion pot:      10       1      1  10.636  10.636      31.018       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      25       0      3   0.719   0.719      131.44       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   7.693   7.693      1.3971       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  86.141  86.141      2.3303       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 168.406 168.406     0.49473       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  37.277  37.277      7.7053       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     76   9W   9W CD1 CG     76   74    1.50    1.37    0.13    4.54    1.37    0.13    4.54\n",
      "    2     78   9W   9W NE1 CD1    77   76    1.53    1.37    0.16    4.78    1.37    0.16    4.78\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    473   9W   9W NE1 CD1    77   76  124.39  180.00  -55.61    4.77  180.00  -55.61    4.77\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  116   2.016\n",
      "  serious non-bonded atom clash:    67   81   2.184\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    1    0    0    0    1    1    1    3    5    8    9    9   18   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      144       14        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      144      14\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      735      97\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          12.3458\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      17       0      0   0.006   0.006     0.18940       1.000\n",
      " 2 Bond angle potential               :      26       0      0   1.602   1.602      1.4997       1.000\n",
      " 3 Stereochemical cosine torsion poten:      37       0      0  49.002  49.002      6.4315       1.000\n",
      " 4 Stereochemical improper torsion pot:      10       0      0   1.609   1.609     0.91595       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      0   0.308   0.308    -0.65657       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.534   4.534     0.48526       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 107.989 107.989    -0.35042E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 179.550 179.550     0.39316       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  34.445  34.445      3.1225       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  116   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    3    4    7    6    7   17   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      144      144\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1185872    1158.078     1.131\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1186400    1158.594     1.131\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1235000    1206.055     1.178\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1248300    1219.043     1.190\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387552    1355.031     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389152    1356.594     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1393132    1360.480     1.329\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395132    1362.434     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1398132    1365.363     1.333\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402632    1369.758     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408204    1375.199     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1411004    1377.934     1.346\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415204    1382.035     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1421504    1388.188     1.356\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1424276    1390.895     1.358\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1425676    1392.262     1.360\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1436713    1403.040     1.370\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1436789    1403.114     1.370\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1436960    1403.281     1.370\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1437810    1404.111     1.371\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1439068    1405.340     1.372\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1440972    1407.199     1.374\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1454396    1420.309     1.387\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      144\n",
      "read_mo_298_> Segment:        1  1:M  16:M      144\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1434040    1400.430     1.368\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1443956    1410.113     1.377\n",
      "mutate_mode_> Model residue 10 changed from THR to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       932416     910.562     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       942332     920.246     0.899\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       944716     922.574     0.901\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    145    152      0         0        73\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       951284     928.988     0.907\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    145    152    207       257        73\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       957380     934.941     0.913\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    146    153    209       259        74\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    146    153    209       259        74\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      142        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968417     945.720     0.924\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       968493     945.794     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968664     945.961     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969514     946.791     0.925\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970772     948.020     0.926\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       972676     949.879     0.928\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       986100     962.988     0.940\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      144\n",
      "read_mo_298_> Segment:        1  1:M  16:M      144\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbGLU10.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      146      146\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutated_match.pdbGLU10.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       526010     513.682     0.502\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      146\n",
      "read_mo_298_> Segment:        1  1:M  16:M      146\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       632478     617.654     0.603\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       640670     625.654     0.611\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       657054     641.654     0.627\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      153     209     251      73\n",
      "              Total number of restraints before, now         :        0      686\n",
      "make_re_422_> Number of previous, current restraints         :        0      686\n",
      "make_re_423_> Number of previous, current selected restraints:        0      686\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657122     641.721     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      686      698\n",
      "make_re_423_> Number of previous, current selected restraints:      686      698\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657138     641.736     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       16\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      698      711\n",
      "make_re_423_> Number of previous, current selected restraints:      698      695\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657138     641.736     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      711      725\n",
      "make_re_423_> Number of previous, current selected restraints:      695      709\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657138     641.736     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      725      735\n",
      "make_re_423_> Number of previous, current selected restraints:      709      719\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657138     641.736     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      735      740\n",
      "make_re_423_> Number of previous, current selected restraints:      719      724\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       657138     641.736     0.627\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match19\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      740      742\n",
      "make_re_423_> Number of previous, current selected restraints:      724      726\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       55\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       699150     682.764     0.667\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       995258     971.932     0.949\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      146       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      742      55\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       13\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.1729\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.019   0.019     0.97094       1.000\n",
      " 2 Bond angle potential               :      15       0      0   2.966   2.966      2.2378       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  53.915  53.915      3.9489       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.494   0.494     0.28981E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      13       0      1   0.451   0.451      3.8431       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.303   4.303     0.43708       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.955 112.955     0.39720       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000    -0.33968E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.254 179.254      1.1129       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  40.037  55.766      5.2299       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  118   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    3    4    7    6    7   16   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      146        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      146       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      742      55\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          11.1214\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.008   0.008     0.24116       1.000\n",
      " 2 Bond angle potential               :      15       0      0   1.765   1.765     0.78763       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  54.405  54.405      3.8675       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.879   0.879     0.89532E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      0   0.375   0.375    -0.21598       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.017   3.017     0.21481       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.034   1.034    -0.10037       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.33439E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  88.326  88.326      1.2214       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  41.711  55.778      5.0492       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  118   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    2    4    7    7    6   17   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match19_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      146      146\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1186824    1159.008     1.132\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1187352    1159.523     1.132\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1235952    1206.984     1.179\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1249252    1219.973     1.191\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388504    1355.961     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390104    1357.523     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1394084    1361.410     1.330\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1396084    1363.363     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1399084    1366.293     1.334\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403584    1370.688     1.339\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409156    1376.129     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1411956    1378.863     1.347\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416156    1382.965     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1422456    1389.117     1.357\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1425228    1391.824     1.359\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1426628    1393.191     1.361\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1437665    1403.970     1.371\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1437741    1404.044     1.371\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1437912    1404.211     1.371\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1438762    1405.041     1.372\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1440020    1406.270     1.373\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1441924    1408.129     1.375\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1455348    1421.238     1.388\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1434720    1401.094     1.368\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1444160    1410.312     1.377\n",
      "mutate_mode_> Model residue 14 changed from LYS to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943516     921.402     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    128    135      0         0        66\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949524     927.270     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    128    135    187       237        66\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955620     933.223     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    136    189       239        67\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    136    189       239        67\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      128        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966657     944.001     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966733     944.075     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966904     944.242     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967754     945.072     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969012     946.301     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970916     948.160     0.926\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984340     961.270     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdbPRO14.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      129      129\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/cpep_aligned.pdbPRO14.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524250     511.963     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      129\n",
      "read_mo_298_> Segment:        1  1:M  16:M      129\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630718     615.936     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638910     623.936     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655294     639.936     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      136     189     247      64\n",
      "              Total number of restraints before, now         :        0      636\n",
      "make_re_422_> Number of previous, current restraints         :        0      636\n",
      "make_re_423_> Number of previous, current selected restraints:        0      636\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655294     639.936     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      636      648\n",
      "make_re_423_> Number of previous, current selected restraints:      636      648\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655310     639.951     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      648      661\n",
      "make_re_423_> Number of previous, current selected restraints:      648      643\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655310     639.951     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      661      674\n",
      "make_re_423_> Number of previous, current selected restraints:      643      656\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655310     639.951     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      674      683\n",
      "make_re_423_> Number of previous, current selected restraints:      656      665\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655310     639.951     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      683      687\n",
      "make_re_423_> Number of previous, current selected restraints:      665      669\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655310     639.951     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      687      687\n",
      "make_re_423_> Number of previous, current selected restraints:      669      669\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       69\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692426     676.197     0.660\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988126     964.967     0.942\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      129       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      687      69\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4009.7922\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   1.121   1.121      3615.3       1.000\n",
      " 2 Bond angle potential               :      16       2      4  26.253  26.253      281.31       1.000\n",
      " 3 Stereochemical cosine torsion poten:      35       1      3  47.376  47.376      32.651       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   3.188   3.188     0.95718       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      1   0.504   0.504      7.7571       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.708   0.708     0.59212E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1 165.296 165.296      14.706       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1 134.575 134.575      20.848       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  33.920  33.920      1.7761       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       1      1  75.283  75.283      34.443       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    119  14P  14P N   CD    106  108    4.82    1.46    3.36  110.67    1.46    3.36  110.67\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    540  14P  14P CA  CB    107  109 -168.06   34.59  157.35   19.23   34.59  157.35   19.23\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    647  13C  14P C   N     104  106 -168.63  -64.50  104.13   10.65  -64.50  104.13   10.65\n",
      "    1         14P  14P N   CA    106  107  146.54  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    3    6    7    6   11   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      129        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      129       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      687      69\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       26\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          73.5774\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.021   0.021      1.1671       1.000\n",
      " 2 Bond angle potential               :      16       0      3   7.204   7.204      14.193       1.000\n",
      " 3 Stereochemical cosine torsion poten:      35       3      4  46.822  46.822      29.874       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       1      1   8.830   8.830      8.5493       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      26       0      2   0.494   0.494      7.0828       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.321   1.321     0.20600E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   5.898   5.898    -0.24257E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   5.655   5.655    -0.10981       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   7.170   7.170    -0.14044       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  50.822  50.822      12.965       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    513  13C  13C O   C     105  104   65.36    0.00   65.36    4.54    0.00   65.36    4.54\n",
      "    2    517  13C  13C CA  C     101  104  -89.49    0.00  -89.49    5.50    0.00  -89.49    5.50\n",
      "    3    540  14P  14P CA  CB    107  109  -27.83   34.59  -62.42    5.37   34.59  -62.42    5.37\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    627  13C  13C C   CA    104  101   15.25    0.00   15.25    5.37    0.00   15.25    5.37\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    1    2    3    6    6    8   11   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      129      129\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179692    1152.043     1.125\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180220    1152.559     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1228820    1200.020     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242120    1213.008     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381372    1348.996     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382972    1350.559     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386952    1354.445     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388952    1356.398     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1391952    1359.328     1.327\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1396452    1363.723     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402024    1369.164     1.337\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1404824    1371.898     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409024    1376.000     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415324    1382.152     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418096    1384.859     1.352\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419496    1386.227     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430533    1397.005     1.364\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1430609    1397.079     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430780    1397.246     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431630    1398.076     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432888    1399.305     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434792    1401.164     1.368\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448216    1414.273     1.381\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      129\n",
      "read_mo_298_> Segment:        1  1:M  16:M      129\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1428608    1395.125     1.362\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438014    1404.311     1.371\n",
      "mutate_mode_> Model residue 15 changed from PRO to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931906     910.064     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941312     919.250     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943456     921.344     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136      0         0        66\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949384     927.133     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136    187       232        66\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955480     933.086     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    189       234        67\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    189       234        67\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      129        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966517     943.864     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966593     943.938     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966764     944.105     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967614     944.936     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968872     946.164     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970776     948.023     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984200     961.133     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      129\n",
      "read_mo_298_> Segment:        1  1:M  16:M      129\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdbGLU15.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdbGLU15.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524110     511.826     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630578     615.799     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638770     623.799     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655154     639.799     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      137     189     235      65\n",
      "              Total number of restraints before, now         :        0      626\n",
      "make_re_422_> Number of previous, current restraints         :        0      626\n",
      "make_re_423_> Number of previous, current selected restraints:        0      626\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655222     639.865     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      626      638\n",
      "make_re_423_> Number of previous, current selected restraints:      626      638\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655238     639.881     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      638      651\n",
      "make_re_423_> Number of previous, current selected restraints:      638      634\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655238     639.881     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      664\n",
      "make_re_423_> Number of previous, current selected restraints:      634      647\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655238     639.881     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      664      673\n",
      "make_re_423_> Number of previous, current selected restraints:      647      656\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655238     639.881     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      673      677\n",
      "make_re_423_> Number of previous, current selected restraints:      656      660\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655238     639.881     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      677      677\n",
      "make_re_423_> Number of previous, current selected restraints:      660      660\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       57\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692930     676.689     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988678     965.506     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      677      57\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       36\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      328869.0312\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      1   0.039   0.039      3.8050       1.000\n",
      " 2 Bond angle potential               :      15       0      2   5.553   5.553      8.8736       1.000\n",
      " 3 Stereochemical cosine torsion poten:      21       0      1  47.518  47.518      3.8902       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.888   1.888     0.28344       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      36       0     10   1.003   1.003     0.32884E+06   1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.575   1.575     0.58517E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  99.186  99.186      2.2993       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 132.132 132.132      1.0397       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.254 179.254     0.96853       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  49.640  49.640      12.128       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "  serious non-bonded atom clash:   111  119   1.761\n",
      "  serious non-bonded atom clash:   113  117   1.466\n",
      "  serious non-bonded atom clash:   113  119   1.058\n",
      "  serious non-bonded atom clash:   114  119   2.036\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    0    0    0    2    1    1    3    4    5    6    6    8   13   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "conmin__610W> Precision problem; starting from scratch.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      677      57\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.1774\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.007   0.007     0.13074       1.000\n",
      " 2 Bond angle potential               :      15       0      0   1.658   1.658     0.87145       1.000\n",
      " 3 Stereochemical cosine torsion poten:      21       0      0  45.095  45.095      3.3631       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.663   1.663     0.38593       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.409   0.409     0.22795       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.710   4.710     0.52374       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 122.580 122.580     0.43331       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000    -0.16880E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  20.128  20.128     0.87099       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  49.261  49.261      11.387       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    1    2    3    6    6    6   13   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180244    1152.582     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180772    1153.098     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229372    1200.559     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242672    1213.547     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381924    1349.535     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383524    1351.098     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387504    1354.984     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389504    1356.938     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392504    1359.867     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397004    1364.262     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402576    1369.703     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405376    1372.438     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409576    1376.539     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415876    1382.691     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418648    1385.398     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420048    1386.766     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431085    1397.544     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431161    1397.618     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431332    1397.785     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432182    1398.615     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433440    1399.844     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435344    1401.703     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448768    1414.812     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429160    1395.664     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438634    1404.916     1.372\n",
      "mutate_mode_> Model residue 16 changed from GLN to ALA\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943528     921.414     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    132      0         0        64\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949316     927.066     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    132    182       227        64\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955412     933.020     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  ALA  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    133    184       229        65\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    133    184       229        65\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      127        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966449     943.798     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966525     943.872     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966696     944.039     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967546     944.869     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968804     946.098     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970708     947.957     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984132     961.066     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdbALA16.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutated_match.pdbALA16.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524042     511.760     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      127\n",
      "read_mo_298_> Segment:        1  1:M  16:M      127\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630510     615.732     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638702     623.732     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655086     639.732     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      133     184     230      63\n",
      "              Total number of restraints before, now         :        0      610\n",
      "make_re_422_> Number of previous, current restraints         :        0      610\n",
      "make_re_423_> Number of previous, current selected restraints:        0      610\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655086     639.732     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      610      622\n",
      "make_re_423_> Number of previous, current selected restraints:      610      622\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655102     639.748     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      622      635\n",
      "make_re_423_> Number of previous, current selected restraints:      622      618\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655102     639.748     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      635      647\n",
      "make_re_423_> Number of previous, current selected restraints:      618      630\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655102     639.748     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      647      655\n",
      "make_re_423_> Number of previous, current selected restraints:      630      638\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655102     639.748     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      655      658\n",
      "make_re_423_> Number of previous, current selected restraints:      638      641\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655102     639.748     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match20\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      658      658\n",
      "make_re_423_> Number of previous, current selected restraints:      641      641\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        6       30\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       691642     675.432     0.660\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       987294     964.154     0.942\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       6\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      658      30\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          39.8178\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.016   0.016     0.63988       1.000\n",
      " 2 Bond angle potential               :       9       1      1   7.121   7.121      13.363       1.000\n",
      " 3 Stereochemical cosine torsion poten:      10       0      0  31.481  31.481     0.64346       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       1      1  17.018  17.018      25.351       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.306   0.306    -0.20490       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   6.497   6.497     0.49818       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       1       0      0   3.388   3.388    -0.47228       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    610  16A  16A O   CA    126  123   29.39    0.00   29.39    9.25    0.00   29.39    9.25\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    1    1    2    6    6    6   12   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      127        6        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       6\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      658      30\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           0.1301\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.002   0.002     0.67279E-02   1.000\n",
      " 2 Bond angle potential               :       9       0      0   0.773   0.773     0.88773E-01   1.000\n",
      " 3 Stereochemical cosine torsion poten:      10       0      0  33.383  33.383     0.62651       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.542   0.542     0.31702E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.315   0.315    -0.14296       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.895   0.895     0.94453E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       1       0      0   3.219   3.219    -0.49007       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    1    1    2    6    7    7   10   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match20_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178860    1151.230     1.124\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179388    1151.746     1.125\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227988    1199.207     1.171\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1241288    1212.195     1.184\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1380540    1348.184     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382140    1349.746     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386120    1353.633     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388120    1355.586     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1391120    1358.516     1.327\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395620    1362.910     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1401192    1368.352     1.336\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403992    1371.086     1.339\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408192    1375.188     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1414492    1381.340     1.349\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417264    1384.047     1.352\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418664    1385.414     1.353\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429701    1396.192     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1429777    1396.267     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429948    1396.434     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430798    1397.264     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432056    1398.492     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433960    1400.352     1.368\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1447384    1413.461     1.380\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1427708    1394.246     1.362\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1437148    1403.465     1.371\n",
      "mutate_mode_> Model residue 9 changed from LEU to ILE\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943508     921.395     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136      0         0        65\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949456     927.203     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    136    186       234        65\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955552     933.156     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       236        66\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       236        66\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      128        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966589     943.935     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966665     944.009     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966836     944.176     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967686     945.006     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968944     946.234     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970848     948.094     0.926\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984272     961.203     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdbILE9.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/cpep_aligned.pdbILE9.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524182     511.896     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630650     615.869     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638842     623.869     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655226     639.869     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      137     188     237      64\n",
      "              Total number of restraints before, now         :        0      626\n",
      "make_re_422_> Number of previous, current restraints         :        0      626\n",
      "make_re_423_> Number of previous, current selected restraints:        0      626\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655260     639.902     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      626      638\n",
      "make_re_423_> Number of previous, current selected restraints:      626      638\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655276     639.918     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      638      651\n",
      "make_re_423_> Number of previous, current selected restraints:      638      634\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655276     639.918     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      664\n",
      "make_re_423_> Number of previous, current selected restraints:      634      647\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655276     639.918     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      664      673\n",
      "make_re_423_> Number of previous, current selected restraints:      647      656\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655276     639.918     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      673      677\n",
      "make_re_423_> Number of previous, current selected restraints:      656      660\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655276     639.918     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      677      678\n",
      "make_re_423_> Number of previous, current selected restraints:      660      661\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        8       56\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692968     676.727     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988716     965.543     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      678      56\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.5343\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      1   0.037   0.037      3.0843       1.000\n",
      " 2 Bond angle potential               :      14       0      1   4.504   4.504      5.3428       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      1  72.611  72.611      7.1747       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   3.564   3.564     0.86524       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      1   0.467   0.467      3.1420       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.063   1.063     0.26650E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 116.229 116.229      1.2028       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1  57.093  57.093      5.7529       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  44.777  44.777      22.943       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    633   8P   9I C   N      62   64 -109.28  -63.40   47.47    8.81  -63.40   47.47    8.81\n",
      "    1          9I   9I N   CA     64   65  -55.78  -43.60                  -43.60\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    4    3    7    7    6   10   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        8        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       8\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      678      56\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          25.2357\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.016   0.016     0.65659       1.000\n",
      " 2 Bond angle potential               :      14       0      0   3.470   3.470      3.2957       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      1  69.687  69.687      7.0570       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   4.166   4.166      1.8639       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      0   0.441   0.441     0.60982       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.300   3.300     0.25712       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 114.702 114.702      1.1211       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   1.895   1.895    -0.19575       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  39.873  39.873      10.570       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    3    7    8    4   12   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180282    1152.619     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180810    1153.135     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229410    1200.596     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242710    1213.584     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381962    1349.572     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383562    1351.135     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387542    1355.021     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389542    1356.975     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392542    1359.904     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397042    1364.299     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402614    1369.740     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405414    1372.475     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409614    1376.576     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415914    1382.729     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418686    1385.436     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420086    1386.803     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431123    1397.581     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431199    1397.655     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431370    1397.822     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432220    1398.652     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433478    1399.881     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435382    1401.740     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448806    1414.850     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429164    1395.668     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438638    1404.920     1.372\n",
      "mutate_mode_> Model residue 10 changed from THR to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943584     921.469     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    137      0         0        65\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949652     927.395     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    137    188       240        65\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955748     933.348     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    138    190       242        66\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    138    190       242        66\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      129        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966785     944.126     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966861     944.200     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967032     944.367     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967882     945.197     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969140     946.426     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       971044     948.285     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984468     961.395     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbPRO10.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbPRO10.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524378     512.088     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630846     616.061     0.602\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       639038     624.061     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655422     640.061     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      138     190     250      63\n",
      "              Total number of restraints before, now         :        0      641\n",
      "make_re_422_> Number of previous, current restraints         :        0      641\n",
      "make_re_423_> Number of previous, current selected restraints:        0      641\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655422     640.061     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      641      653\n",
      "make_re_423_> Number of previous, current selected restraints:      641      653\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655438     640.076     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      653      666\n",
      "make_re_423_> Number of previous, current selected restraints:      653      648\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655438     640.076     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      666      679\n",
      "make_re_423_> Number of previous, current selected restraints:      648      661\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655438     640.076     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      679      689\n",
      "make_re_423_> Number of previous, current selected restraints:      661      671\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655438     640.076     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      689      694\n",
      "make_re_423_> Number of previous, current selected restraints:      671      676\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655438     640.076     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      694      695\n",
      "make_re_423_> Number of previous, current selected restraints:      676      677\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       65\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       693130     676.885     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988878     965.701     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      695      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       28\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         911.7311\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   0.502   0.502      505.37       1.000\n",
      " 2 Bond angle potential               :      15       2      3  15.229  15.229      73.395       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      0  46.390  46.390      8.2867       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.623   0.623     0.38770E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      28       0      3   0.721   0.721      285.20       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.618   2.618     0.16176       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.252   1.252    -0.23137       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1  36.683  36.683      3.4116       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  75.918  75.918     0.81392       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      2  64.815  75.586      35.281       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     82  10P  10P CG  CD     76   74    3.04    1.54    1.50   41.31    1.54    1.50   41.31\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    649   9I  10P C   N      70   72 -163.23  -64.50  107.40    8.86  -64.50  107.40    8.86\n",
      "    1         10P  10P N   CA     72   73 -170.51  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "  serious non-bonded atom clash:    74   78   1.986\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    0    1    1    0    0    3    3    7    9    4   12   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      695      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          91.1333\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      1   0.042   0.042      4.3476       1.000\n",
      " 2 Bond angle potential               :      15       1      1   6.626   6.626      14.037       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       2      2  55.942  55.942      22.009       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.481   2.481     0.66648       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      1   0.644   0.644      10.412       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      1  11.888  11.888      3.3359       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  20.269  20.269     0.50387       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  50.905  50.905      1.6489       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  72.393  72.393     0.57272       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      2  65.645  73.195      33.600       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    456   9I   9I O   C      71   70   99.07  180.00  -80.93    5.36  180.00  -80.93    5.36\n",
      "    2    460   9I   9I CA  C      65   70  -73.86    0.00  -73.86    5.07    0.00  -73.86    5.07\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    649   9I  10P C   N      70   72 -146.76  -64.50  108.42    7.24  -64.50  108.42    7.24\n",
      "    1         10P  10P N   CA     72   73 -142.16  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  103   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    6    9    6   12   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180444    1152.777     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180972    1153.293     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229572    1200.754     1.173\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242872    1213.742     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382124    1349.730     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383724    1351.293     1.320\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387704    1355.180     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389704    1357.133     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392704    1360.062     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397204    1364.457     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402776    1369.898     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405576    1372.633     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409776    1376.734     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416076    1382.887     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418848    1385.594     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420248    1386.961     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431285    1397.739     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431361    1397.813     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431532    1397.980     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432382    1398.811     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433640    1400.039     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435544    1401.898     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448968    1415.008     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429292    1395.793     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438766    1405.045     1.372\n",
      "mutate_mode_> Model residue 11 changed from TRP to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943432     921.320     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130      0         0        59\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949064     926.820     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130    177       221        59\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955160     932.773     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    179       223        60\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    179       223        60\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      123        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966197     943.552     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966273     943.626     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966444     943.793     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967294     944.623     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968552     945.852     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970456     947.711     0.925\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983880     960.820     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbGLU11.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      126      126\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbGLU11.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523790     511.514     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630258     615.486     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638450     623.486     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654834     639.486     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      131     179     231      57\n",
      "              Total number of restraints before, now         :        0      598\n",
      "make_re_422_> Number of previous, current restraints         :        0      598\n",
      "make_re_423_> Number of previous, current selected restraints:        0      598\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654834     639.486     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      598      610\n",
      "make_re_423_> Number of previous, current selected restraints:      598      610\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654850     639.502     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      610      623\n",
      "make_re_423_> Number of previous, current selected restraints:      610      605\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654850     639.502     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      623      636\n",
      "make_re_423_> Number of previous, current selected restraints:      605      618\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654850     639.502     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      636      646\n",
      "make_re_423_> Number of previous, current selected restraints:      618      628\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654850     639.502     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      646      652\n",
      "make_re_423_> Number of previous, current selected restraints:      628      634\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654850     639.502     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      652      653\n",
      "make_re_423_> Number of previous, current selected restraints:      634      635\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       58\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       691102     674.904     0.659\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       986730     963.604     0.941\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      126       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      653      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       15\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          36.8179\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.030   0.030      2.8710       1.000\n",
      " 2 Bond angle potential               :      15       0      1   3.879   3.879      4.3960       1.000\n",
      " 3 Stereochemical cosine torsion poten:      21       0      0  53.052  53.052      5.2314       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.792   0.792     0.74561E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      15       0      0   0.279   0.279     -1.2670       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.532   2.532     0.15132       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  10.354  10.354     0.15988       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000     0.16762E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.254 179.254      1.2606       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  66.868  66.868      23.923       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    606   9I  10P C   N      70   72 -146.78  -64.50  108.43    7.25  -64.50  108.43    7.25\n",
      "    1         10P  10P N   CA     72   73 -142.18  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   98   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    5    9    5   12   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      126        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      126       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      653      58\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       15\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          31.0599\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.012   0.012     0.44402       1.000\n",
      " 2 Bond angle potential               :      15       0      0   2.762   2.762      2.2850       1.000\n",
      " 3 Stereochemical cosine torsion poten:      21       0      0  51.959  51.959      4.9058       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.892   0.892     0.10262       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      15       0      0   0.290   0.290     -1.2385       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.975   2.975     0.20893       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   4.413   4.413    -0.69959E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   8.134   8.134     0.14469       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  59.726  59.726     0.96475       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  65.084  65.084      23.313       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    606   9I  10P C   N      70   72 -146.78  -64.50  105.31    7.21  -64.50  105.31    7.21\n",
      "    1         10P  10P N   CA     72   73 -147.07  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   98   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    5   10    4   13   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      126      126\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178296    1150.680     1.124\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178824    1151.195     1.124\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227424    1198.656     1.171\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1240724    1211.645     1.183\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1379976    1347.633     1.316\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381576    1349.195     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1385556    1353.082     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387556    1355.035     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390556    1357.965     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395056    1362.359     1.330\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1400628    1367.801     1.336\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403428    1370.535     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1407628    1374.637     1.342\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1413928    1380.789     1.348\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416700    1383.496     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418100    1384.863     1.352\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429137    1395.642     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1429213    1395.716     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429384    1395.883     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430234    1396.713     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431492    1397.941     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433396    1399.801     1.367\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1446820    1412.910     1.380\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1427144    1393.695     1.361\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1436448    1402.781     1.370\n",
      "mutate_mode_> Model residue 12 changed from VAL to ALA\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931804     909.965     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941108     919.051     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943060     920.957     0.899\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    123    128      0         0        58\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948592     926.359     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    123    128    174       217        58\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       954688     932.312     0.910\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    124    129    176       219        59\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    124    129    176       219        59\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      124        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       965725     943.091     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       965801     943.165     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       965972     943.332     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966822     944.162     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968080     945.391     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       969984     947.250     0.925\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983408     960.359     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbALA12.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      124      124\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutated_match.pdbALA12.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523318     511.053     0.499\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      124\n",
      "read_mo_298_> Segment:        1  1:M  16:M      124\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       629786     615.025     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       637978     623.025     0.608\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654362     639.025     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      129     176     227      56\n",
      "              Total number of restraints before, now         :        0      588\n",
      "make_re_422_> Number of previous, current restraints         :        0      588\n",
      "make_re_423_> Number of previous, current selected restraints:        0      588\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654362     639.025     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      588      600\n",
      "make_re_423_> Number of previous, current selected restraints:      588      600\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654378     639.041     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      600      613\n",
      "make_re_423_> Number of previous, current selected restraints:      600      595\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654378     639.041     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      613      625\n",
      "make_re_423_> Number of previous, current selected restraints:      595      607\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654378     639.041     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      625      635\n",
      "make_re_423_> Number of previous, current selected restraints:      607      617\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654378     639.041     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      635      641\n",
      "make_re_423_> Number of previous, current selected restraints:      617      623\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654378     639.041     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match23\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      641      642\n",
      "make_re_423_> Number of previous, current selected restraints:      623      624\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        5       36\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       690054     673.881     0.658\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       985634     962.533     0.940\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      124       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      642      36\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           8.8433\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.017   0.017     0.57758       1.000\n",
      " 2 Bond angle potential               :      10       0      0   3.335   3.335      2.3046       1.000\n",
      " 3 Stereochemical cosine torsion poten:      13       0      1  65.520  65.520      5.0062       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.369   2.369     0.61490       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.284   0.284     -1.5884       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   0.848   0.848     0.16983E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  32.300  32.300      1.9114       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   96   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    5   10    4   13   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      124        5        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      124       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      642      36\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           6.1276\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.005   0.005     0.44897E-01   1.000\n",
      " 2 Bond angle potential               :      10       0      0   1.961   1.961     0.88177       1.000\n",
      " 3 Stereochemical cosine torsion poten:      13       0      1  64.592  64.592      5.0656       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.248   1.248     0.16959       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.284   0.284     -1.5397       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.715   4.715     0.52479       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  28.429  28.429     0.98065       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   96   2.016\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    5   10    4   13   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match23_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      124      124\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1177200    1149.609     1.123\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1177728    1150.125     1.123\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1226328    1197.586     1.170\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1239628    1210.574     1.182\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1378880    1346.562     1.315\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1380480    1348.125     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1384460    1352.012     1.320\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386460    1353.965     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389460    1356.895     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1393960    1361.289     1.329\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1399532    1366.730     1.335\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402332    1369.465     1.337\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1406532    1373.566     1.341\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1412832    1379.719     1.347\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415604    1382.426     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417004    1383.793     1.351\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1428041    1394.571     1.362\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1428117    1394.646     1.362\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1428288    1394.812     1.362\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429138    1395.643     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430396    1396.871     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432300    1398.730     1.366\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1445724    1411.840     1.379\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1426388    1392.957     1.360\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1435828    1402.176     1.369\n",
      "mutate_mode_> Model residue 5 changed from HIS to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943404     921.293     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    133      0         0        60\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949268     927.020     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    133    183       231        60\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955364     932.973     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    128    134    185       233        61\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    128    134    185       233        61\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      126        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966401     943.751     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966477     943.825     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966648     943.992     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967498     944.822     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968756     946.051     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970660     947.910     0.926\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984084     961.020     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdbPRO5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      128      128\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/cpep_aligned.pdbPRO5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523994     511.713     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      128\n",
      "read_mo_298_> Segment:        1  1:M  16:M      128\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630462     615.686     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638654     623.686     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655038     639.686     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      134     185     240      58\n",
      "              Total number of restraints before, now         :        0      617\n",
      "make_re_422_> Number of previous, current restraints         :        0      617\n",
      "make_re_423_> Number of previous, current selected restraints:        0      617\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655038     639.686     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      617      629\n",
      "make_re_423_> Number of previous, current selected restraints:      617      629\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655054     639.701     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      629      642\n",
      "make_re_423_> Number of previous, current selected restraints:      629      624\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655054     639.701     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      642      655\n",
      "make_re_423_> Number of previous, current selected restraints:      624      637\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655054     639.701     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      655      664\n",
      "make_re_423_> Number of previous, current selected restraints:      637      646\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655054     639.701     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      664      669\n",
      "make_re_423_> Number of previous, current selected restraints:      646      651\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655054     639.701     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      669      670\n",
      "make_re_423_> Number of previous, current selected restraints:      651      652\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       63\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       691882     675.666     0.660\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       987558     964.412     0.942\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      128       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      670      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         917.4330\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   0.565   0.565      638.44       1.000\n",
      " 2 Bond angle potential               :      15       2      2  13.385  13.385      57.025       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       1      1  41.078  41.078      11.507       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.766   2.766     0.62323       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      3   0.723   0.723      182.97       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.764   0.764     0.68912E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  93.730  93.730      5.5582       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  57.164  57.164     0.53524       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  45.246  45.246      1.1648       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       1      1  57.644  68.442      19.600       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     39   5P   5P CG  CD     36   34    3.23    1.54    1.69   46.49    1.54    1.69   46.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    389   5P   5P CA  CB     33   35  -23.53   34.59  -58.12    4.72   34.59  -58.12    4.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    620   4C   5P C   N      30   32 -142.82  -64.50   79.49    7.52  -64.50   79.49    7.52\n",
      "    1          5P   5P N   CA     32   33  160.79  147.20                  147.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  100   2.017\n",
      "  serious non-bonded atom clash:    34   38   2.232\n",
      "  serious non-bonded atom clash:    34   96   2.098\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    1    0    0    1    0    0    3    3    9    8    6   10   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      128        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      128       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      670      63\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       19\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          33.9554\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.016   0.016     0.65764       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.984   3.984      4.5451       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       1      2  49.475  49.475      18.951       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      1   4.836   4.836      2.4518       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      19       0      0   0.325   0.325    -0.37185       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0  10.962  10.962      1.4181       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   9.258   9.258     0.26598E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   5.052   5.052    -0.18535       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  63.852  63.852     0.24839       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      1  40.391  51.004      6.2134       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  3                           : Stereochemical cosine torsion potential \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    367   4C   4C O   C      31   30  103.57  180.00  -76.43    5.20  180.00  -76.43    5.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  100   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    7    5    8    9   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      128      128\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179124    1151.488     1.125\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179652    1152.004     1.125\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1228252    1199.465     1.171\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1241552    1212.453     1.184\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1380804    1348.441     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382404    1350.004     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386384    1353.891     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388384    1355.844     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1391384    1358.773     1.327\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395884    1363.168     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1401456    1368.609     1.337\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1404256    1371.344     1.339\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408456    1375.445     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1414756    1381.598     1.349\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417528    1384.305     1.352\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418928    1385.672     1.353\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429965    1396.450     1.364\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1430041    1396.524     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430212    1396.691     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431062    1397.521     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432320    1398.750     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434224    1400.609     1.368\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1447648    1413.719     1.381\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      128\n",
      "read_mo_298_> Segment:        1  1:M  16:M      128\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1428040    1394.570     1.362\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1437412    1403.723     1.371\n",
      "mutate_mode_> Model residue 6 changed from PHE to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931872     910.031     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941244     919.184     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943164     921.059     0.899\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130      0         0        55\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948868     926.629     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130    179       224        55\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       954964     932.582     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    181       226        56\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    181       226        56\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      123        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966001     943.360     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966077     943.435     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966248     943.602     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967098     944.432     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968356     945.660     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970260     947.520     0.925\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983684     960.629     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      128\n",
      "read_mo_298_> Segment:        1  1:M  16:M      128\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdbGLU6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      126      126\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdbGLU6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523594     511.322     0.499\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630062     615.295     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638254     623.295     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654638     639.295     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      131     181     233      53\n",
      "              Total number of restraints before, now         :        0      598\n",
      "make_re_422_> Number of previous, current restraints         :        0      598\n",
      "make_re_423_> Number of previous, current selected restraints:        0      598\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654638     639.295     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      598      610\n",
      "make_re_423_> Number of previous, current selected restraints:      598      610\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654654     639.311     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      610      623\n",
      "make_re_423_> Number of previous, current selected restraints:      610      605\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654654     639.311     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      623      636\n",
      "make_re_423_> Number of previous, current selected restraints:      605      618\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654654     639.311     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      636      645\n",
      "make_re_423_> Number of previous, current selected restraints:      618      627\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654654     639.311     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      645      651\n",
      "make_re_423_> Number of previous, current selected restraints:      627      633\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654654     639.311     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      652\n",
      "make_re_423_> Number of previous, current selected restraints:      633      634\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       57\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       690906     674.713     0.659\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       986534     963.412     0.941\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      126       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      652      57\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        5\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          15.8616\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.022   0.022      1.2594       1.000\n",
      " 2 Bond angle potential               :      15       0      0   2.808   2.808      1.9297       1.000\n",
      " 3 Stereochemical cosine torsion poten:      20       0      0  50.107  50.107      3.8715       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.621   1.621     0.37561       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       5       0      0   0.281   0.281    -0.37063       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   7.847   7.847      1.4534       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   5.617   5.617    -0.11006       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.46324E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.274 179.274      1.0357       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  36.309  40.989      6.4635       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   98   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    6    5    7    9    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      126        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      126       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      652      57\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        5\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          11.8771\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.006   0.006     0.10050       1.000\n",
      " 2 Bond angle potential               :      15       0      0   1.294   1.294     0.40531       1.000\n",
      " 3 Stereochemical cosine torsion poten:      20       0      0  51.695  51.695      3.9263       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   1.219   1.219     0.21260       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       5       0      0   0.263   0.263    -0.37619       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.519   4.519     0.48193       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   2.996   2.996    -0.17150       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.028   0.028    -0.46324E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  93.002  93.002      1.2959       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  37.495  37.495      6.0486       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   98   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    4    6    5    7    9    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      126      126\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178100    1150.488     1.124\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178628    1151.004     1.124\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227228    1198.465     1.170\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1240528    1211.453     1.183\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1379780    1347.441     1.316\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381380    1349.004     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1385360    1352.891     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387360    1354.844     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390360    1357.773     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1394860    1362.168     1.330\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1400432    1367.609     1.336\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403232    1370.344     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1407432    1374.445     1.342\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1413732    1380.598     1.348\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416504    1383.305     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417904    1384.672     1.352\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1428941    1395.450     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1429017    1395.524     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429188    1395.691     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430038    1396.521     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431296    1397.750     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433200    1399.609     1.367\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1446624    1412.719     1.380\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1427152    1393.703     1.361\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1436456    1402.789     1.370\n",
      "mutate_mode_> Model residue 7 changed from GLY to ALA\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931804     909.965     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941108     919.051     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943052     920.949     0.899\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131      0         0        56\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948828     926.590     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    181       227        56\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       954924     932.543     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    132    183       229        57\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    132    183       229        57\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      126        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       965961     943.321     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966037     943.396     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966208     943.562     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967058     944.393     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968316     945.621     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970220     947.480     0.925\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983644     960.590     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      126\n",
      "read_mo_298_> Segment:        1  1:M  16:M      126\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdbALA7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutated_match.pdbALA7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523554     511.283     0.499\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      127\n",
      "read_mo_298_> Segment:        1  1:M  16:M      127\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630022     615.256     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638214     623.256     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654598     639.256     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      132     183     235      54\n",
      "              Total number of restraints before, now         :        0      604\n",
      "make_re_422_> Number of previous, current restraints         :        0      604\n",
      "make_re_423_> Number of previous, current selected restraints:        0      604\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654632     639.289     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      604      616\n",
      "make_re_423_> Number of previous, current selected restraints:      604      616\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654648     639.305     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      616      629\n",
      "make_re_423_> Number of previous, current selected restraints:      616      611\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654648     639.305     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      629      642\n",
      "make_re_423_> Number of previous, current selected restraints:      611      624\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654648     639.305     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      642      651\n",
      "make_re_423_> Number of previous, current selected restraints:      624      633\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654648     639.305     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      651      657\n",
      "make_re_423_> Number of previous, current selected restraints:      633      639\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654648     639.305     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match24\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      657      658\n",
      "make_re_423_> Number of previous, current selected restraints:      639      640\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        5       43\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       691188     674.988     0.659\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       986840     963.711     0.941\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      658      43\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          24.2778\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.017   0.017     0.50028       1.000\n",
      " 2 Bond angle potential               :      11       0      0   4.139   4.139      2.6338       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  32.903  32.903      5.4761       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.274   1.274     0.98744E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      2   0.607   0.607      11.159       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.222   2.222     0.11649       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  36.115  36.115      4.2930       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   99   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    1    4    4    6    6    7    9    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      127        5        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      658      43\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          20.9541\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.018   0.018     0.56945       1.000\n",
      " 2 Bond angle potential               :      11       0      0   4.471   4.471      3.1884       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  31.666  31.666      4.7623       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.969   0.969     0.10018       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      3   0.610   0.610      9.3191       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.425   2.425     0.13881       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  33.412  33.412      2.8760       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   99   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    1    3    6    6    6    7    8    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match24_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178406    1150.787     1.124\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178934    1151.303     1.124\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227534    1198.764     1.171\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1240834    1211.752     1.183\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1380086    1347.740     1.316\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381686    1349.303     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1385666    1353.189     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387666    1355.143     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390666    1358.072     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395166    1362.467     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1400738    1367.908     1.336\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403538    1370.643     1.339\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1407738    1374.744     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1414038    1380.896     1.349\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416810    1383.604     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418210    1384.971     1.353\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429247    1395.749     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1429323    1395.823     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429494    1395.990     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430344    1396.820     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431602    1398.049     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433506    1399.908     1.367\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1446930    1413.018     1.380\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1427560    1394.102     1.361\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1437000    1403.320     1.370\n",
      "mutate_mode_> Model residue 6 changed from PHE to ALA\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943356     921.246     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    124    129      0         0        59\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948972     926.730     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    124    129    177       220        59\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955068     932.684     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130    179       222        60\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    125    130    179       222        60\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      124        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966105     943.462     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966181     943.536     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966352     943.703     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967202     944.533     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968460     945.762     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970364     947.621     0.925\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983788     960.730     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdbALA6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      125      125\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/cpep_aligned.pdbALA6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523698     511.424     0.499\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      125\n",
      "read_mo_298_> Segment:        1  1:M  16:M      125\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630166     615.396     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638358     623.396     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654742     639.396     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      130     179     223      58\n",
      "              Total number of restraints before, now         :        0      590\n",
      "make_re_422_> Number of previous, current restraints         :        0      590\n",
      "make_re_423_> Number of previous, current selected restraints:        0      590\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654742     639.396     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      590      602\n",
      "make_re_423_> Number of previous, current selected restraints:      590      602\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654758     639.412     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      602      615\n",
      "make_re_423_> Number of previous, current selected restraints:      602      598\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654758     639.412     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      615      627\n",
      "make_re_423_> Number of previous, current selected restraints:      598      610\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654758     639.412     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      627      635\n",
      "make_re_423_> Number of previous, current selected restraints:      610      618\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654758     639.412     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      635      639\n",
      "make_re_423_> Number of previous, current selected restraints:      618      622\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654758     639.412     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      639      640\n",
      "make_re_423_> Number of previous, current selected restraints:      622      623\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        5       36\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       690722     674.533     0.659\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       986326     963.209     0.941\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      125       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      640      36\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           5.9030\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.024   0.024      1.0306       1.000\n",
      " 2 Bond angle potential               :      10       0      0   1.356   1.356     0.37386       1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      0  51.293  51.293      2.7891       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.470   2.470     0.66264       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.307   0.307    -0.50509       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.427   1.427     0.48061E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  25.136  25.136      1.5038       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   97   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    3    6    7    5   11   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      125        5        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      125       5\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      640      36\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           3.9761\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       6       0      0   0.001   0.001     0.39252E-02   1.000\n",
      " 2 Bond angle potential               :      10       0      0   0.615   0.615     0.67423E-01   1.000\n",
      " 3 Stereochemical cosine torsion poten:      12       0      0  51.273  51.273      2.6155       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.194   1.194     0.14957       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.309   0.309    -0.44533       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.755   1.755     0.72694E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  25.491  25.491      1.5124       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29   97   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    3    3    6    7    4   12   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      125      125\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1177892    1150.285     1.123\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178420    1150.801     1.124\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227020    1198.262     1.170\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1240320    1211.250     1.183\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1379572    1347.238     1.316\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381172    1348.801     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1385152    1352.688     1.321\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387152    1354.641     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390152    1357.570     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1394652    1361.965     1.330\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1400224    1367.406     1.335\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403024    1370.141     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1407224    1374.242     1.342\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1413524    1380.395     1.348\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416296    1383.102     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417696    1384.469     1.352\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1428733    1395.247     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1428809    1395.321     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1428980    1395.488     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429830    1396.318     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431088    1397.547     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432992    1399.406     1.367\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1446416    1412.516     1.379\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      125\n",
      "read_mo_298_> Segment:        1  1:M  16:M      125\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1426808    1393.367     1.361\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1436078    1402.420     1.370\n",
      "mutate_mode_> Model residue 7 changed from GLY to LYS\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931770     909.932     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941040     918.984     0.897\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943072     920.969     0.899\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    134      0         0        60\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948888     926.648     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    134    183       228        60\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       954984     932.602     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135    185       230        61\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135    185       230        61\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      125        5\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966021     943.380     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966097     943.454     0.921\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966268     943.621     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967118     944.451     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968376     945.680     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970280     947.539     0.925\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983704     960.648     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      125\n",
      "read_mo_298_> Segment:        1  1:M  16:M      125\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbLYS7.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      130      130\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbLYS7.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523614     511.342     0.499\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630082     615.314     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638274     623.314     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654658     639.314     0.624\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      135     185     230      59\n",
      "              Total number of restraints before, now         :        0      609\n",
      "make_re_422_> Number of previous, current restraints         :        0      609\n",
      "make_re_423_> Number of previous, current selected restraints:        0      609\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654828     639.480     0.624\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      609      621\n",
      "make_re_423_> Number of previous, current selected restraints:      609      621\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654844     639.496     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      621      634\n",
      "make_re_423_> Number of previous, current selected restraints:      621      617\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654844     639.496     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      634      647\n",
      "make_re_423_> Number of previous, current selected restraints:      617      630\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654844     639.496     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      647      656\n",
      "make_re_423_> Number of previous, current selected restraints:      630      639\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654844     639.496     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      656      661\n",
      "make_re_423_> Number of previous, current selected restraints:      639      644\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654844     639.496     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      661      663\n",
      "make_re_423_> Number of previous, current selected restraints:      644      646\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       60\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692248     676.023     0.660\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       987972     964.816     0.942\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      130       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      663      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         113.0431\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.017   0.017     0.70950       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.459   3.459      2.5470       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      0  29.974  29.974      5.5929       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.934   0.934     0.53373E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      5   0.808   0.808      99.277       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.886   1.886     0.83918E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 112.955 112.955     0.34781       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  13.770  13.770     0.56617E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.31336       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.000   0.000     0.22251       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  33.848  37.131      3.8389       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  102   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    1    0    1    0    2    4    4    6    9    4   12   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      130        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      130       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      663      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       27\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          47.6614\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.020   0.020      1.1554       1.000\n",
      " 2 Bond angle potential               :      15       1      1   6.669   6.669      11.316       1.000\n",
      " 3 Stereochemical cosine torsion poten:      23       0      0  30.301  30.301      4.9719       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.844   0.844     0.46139E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      27       0      3   0.729   0.729      25.360       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.285   2.285     0.12321       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 126.247 126.247      1.0027       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  87.722  87.722      1.1679       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.179   0.179     0.30648       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   2.167   2.167     0.23226       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  27.445  32.673      1.9789       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    204   7K   7K CA  CB     48   49  133.91  113.50   20.41    5.01  113.50   20.41    5.01\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  102   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    4    5    5   12    6   10   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      130      130\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179538    1151.893     1.125\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180066    1152.408     1.125\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1228666    1199.869     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1241966    1212.857     1.184\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381218    1348.846     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382818    1350.408     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386798    1354.295     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388798    1356.248     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1391798    1359.178     1.327\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1396298    1363.572     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1401870    1369.014     1.337\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1404670    1371.748     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408870    1375.850     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415170    1382.002     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417942    1384.709     1.352\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419342    1386.076     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430379    1396.854     1.364\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1430455    1396.929     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430626    1397.096     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431476    1397.926     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432734    1399.154     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434638    1401.014     1.368\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448062    1414.123     1.381\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1428624    1395.141     1.362\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438064    1404.359     1.371\n",
      "mutate_mode_> Model residue 8 changed from PRO to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943412     921.301     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    134      0         0        60\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949228     926.980     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    129    134    183       228        60\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955324     932.934     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135    185       230        61\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135    185       230        61\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      130        0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966361     943.712     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966437     943.786     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966608     943.953     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967458     944.783     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968716     946.012     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970620     947.871     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984044     960.980     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbPRO8.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      130      130\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbPRO8.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523954     511.674     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630422     615.646     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638614     623.646     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654998     639.646     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      135     185     230      59\n",
      "              Total number of restraints before, now         :        0      609\n",
      "make_re_422_> Number of previous, current restraints         :        0      609\n",
      "make_re_423_> Number of previous, current selected restraints:        0      609\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654998     639.646     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      609      621\n",
      "make_re_423_> Number of previous, current selected restraints:      609      621\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655014     639.662     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      621      634\n",
      "make_re_423_> Number of previous, current selected restraints:      621      617\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655014     639.662     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      634      647\n",
      "make_re_423_> Number of previous, current selected restraints:      617      630\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655014     639.662     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      647      656\n",
      "make_re_423_> Number of previous, current selected restraints:      630      639\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655014     639.662     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      656      661\n",
      "make_re_423_> Number of previous, current selected restraints:      639      644\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655014     639.662     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      661      663\n",
      "make_re_423_> Number of previous, current selected restraints:      644      646\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       65\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692418     676.189     0.660\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988142     964.982     0.942\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      130       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      663      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          48.5046\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.029   0.029      2.7791       1.000\n",
      " 2 Bond angle potential               :      15       0      0   4.744   4.744      6.0997       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      0  52.645  52.645      6.9738       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.504   1.504     0.13543       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      1   0.692   0.692      9.2005       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.378   2.378     0.13346       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   3.429   3.429    -0.29743       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   1.404   1.404    -0.39916       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.697   0.697    -0.14240       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  37.445  41.454      24.022       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    616   8P   9L C   N      61   63 -109.20  -63.50   47.98    8.76  -63.50   47.98    8.76\n",
      "    1          9L   9L N   CA     63   64  -55.80  -41.20                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  102   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    0    4    5    5   12    6   10   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      130        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      130       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      663      65\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       14\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          27.6508\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.006   0.006     0.82269E-01   1.000\n",
      " 2 Bond angle potential               :      15       0      0   5.425   5.425      6.4822       1.000\n",
      " 3 Stereochemical cosine torsion poten:      30       0      0  51.251  51.251      8.3000       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.150   2.150     0.50554       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      14       0      1   0.511   0.511      3.4595       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.193   4.193     0.41503       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.586   1.586    -0.38605       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   3.929   3.929    -0.29414       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   7.804   7.804     0.85104E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  26.960  32.361      9.0013       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  102   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    1    4    3    5    9    7   12   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      130      130\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179708    1152.059     1.125\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180236    1152.574     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1228836    1200.035     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242136    1213.023     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381388    1349.012     1.317\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382988    1350.574     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1386968    1354.461     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388968    1356.414     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1391968    1359.344     1.327\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1396468    1363.738     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402040    1369.180     1.337\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1404840    1371.914     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409040    1376.016     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415340    1382.168     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418112    1384.875     1.352\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419512    1386.242     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430549    1397.021     1.364\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1430625    1397.095     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430796    1397.262     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431646    1398.092     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432904    1399.320     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434808    1401.180     1.368\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448232    1414.289     1.381\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1428624    1395.141     1.362\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438064    1404.359     1.371\n",
      "mutate_mode_> Model residue 9 changed from LEU to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943436     921.324     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    132    137      0         0        60\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949336     927.086     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    132    137    186       231        60\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955432     933.039     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    133    138    188       233        61\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    133    138    188       233        61\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      128        5\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966469     943.817     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966545     943.892     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966716     944.059     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967566     944.889     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968824     946.117     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970728     947.977     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984152     961.086     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbARG9.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      133      133\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutated_match.pdbARG9.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524062     511.779     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      133\n",
      "read_mo_298_> Segment:        1  1:M  16:M      133\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630530     615.752     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638722     623.752     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655106     639.752     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      138     188     232      59\n",
      "              Total number of restraints before, now         :        0      617\n",
      "make_re_422_> Number of previous, current restraints         :        0      617\n",
      "make_re_423_> Number of previous, current selected restraints:        0      617\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655208     639.852     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      617      629\n",
      "make_re_423_> Number of previous, current selected restraints:      617      629\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655224     639.867     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      629      642\n",
      "make_re_423_> Number of previous, current selected restraints:      629      625\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655224     639.867     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      642      655\n",
      "make_re_423_> Number of previous, current selected restraints:      625      638\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655224     639.867     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      655      664\n",
      "make_re_423_> Number of previous, current selected restraints:      638      647\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655224     639.867     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      664      670\n",
      "make_re_423_> Number of previous, current selected restraints:      647      653\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655224     639.867     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match26\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      670      673\n",
      "make_re_423_> Number of previous, current selected restraints:      653      656\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       64\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       693492     677.238     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       989288     966.102     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      133      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      673      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       10\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          23.3331\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.021   0.021      2.1309       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.462   2.462      2.0389       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      1  69.043  69.043      6.7794       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.708   0.708     0.56045E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      10       0      0   0.382   0.382     0.26932       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.256   4.256     0.42765       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  26.809  26.809      1.2838       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   0.000   0.000     0.62459E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.028   0.028     0.62125E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.102  90.102     0.73338       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  37.507  37.507      9.4892       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  105   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    1    4    3    5    9    7   11   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      133       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      133      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      673      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       13\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          17.6991\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.007   0.007     0.21366       1.000\n",
      " 2 Bond angle potential               :      17       0      0   2.275   2.275      1.7690       1.000\n",
      " 3 Stereochemical cosine torsion poten:      22       0      1  66.935  66.935      6.3259       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.474   2.474     0.89132       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      13       0      0   0.349   0.349    -0.30900       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.041   3.041     0.21829       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 108.986 108.986     0.62875       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 103.512 103.512     0.85705       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 108.262 108.262     0.97065       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 137.334 137.334     0.94199       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  37.326  37.326      5.1915       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    29  105   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    1    1    4    3    5    8    6   13   14\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match26_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      133      133\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180854    1153.178     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181382    1153.693     1.127\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229982    1201.154     1.173\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1243282    1214.143     1.186\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382534    1350.131     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1384134    1351.693     1.320\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1388114    1355.580     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390114    1357.533     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1393114    1360.463     1.329\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397614    1364.857     1.333\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403186    1370.299     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405986    1373.033     1.341\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1410186    1377.135     1.345\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416486    1383.287     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419258    1385.994     1.354\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420658    1387.361     1.355\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431695    1398.140     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431771    1398.214     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431942    1398.381     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432792    1399.211     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1434050    1400.439     1.368\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435954    1402.299     1.369\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1449378    1415.408     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429668    1396.160     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1439108    1405.379     1.372\n",
      "mutate_mode_> Model residue 3 changed from SER to THR\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931940     910.098     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941380     919.316     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943532     921.418     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137      0         0        66\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949504     927.250     0.906\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    137    188       234        66\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955600     933.203     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    132    138    190       236        67\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    132    138    190       236        67\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      129        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966637     943.981     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966713     944.056     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966884     944.223     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967734     945.053     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968992     946.281     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970896     948.141     0.926\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb, Line: 131\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984320     961.250     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      130\n",
      "read_mo_298_> Segment:        1  1:M  16:M      130\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdbTHR3.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      132      132\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/cpep_aligned.pdbTHR3.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524230     511.943     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      132\n",
      "read_mo_298_> Segment:        1  1:M  16:M      132\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630698     615.916     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638890     623.916     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655274     639.916     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      138     190     237      65\n",
      "              Total number of restraints before, now         :        0      630\n",
      "make_re_422_> Number of previous, current restraints         :        0      630\n",
      "make_re_423_> Number of previous, current selected restraints:        0      630\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655342     639.982     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      630      642\n",
      "make_re_423_> Number of previous, current selected restraints:      630      642\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655358     639.998     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      642      655\n",
      "make_re_423_> Number of previous, current selected restraints:      642      638\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655358     639.998     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      655      668\n",
      "make_re_423_> Number of previous, current selected restraints:      638      651\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655358     639.998     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      668      677\n",
      "make_re_423_> Number of previous, current selected restraints:      651      660\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655358     639.998     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      677      681\n",
      "make_re_423_> Number of previous, current selected restraints:      660      664\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655358     639.998     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      681      682\n",
      "make_re_423_> Number of previous, current selected restraints:      664      665\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       47\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       693338     677.088     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       989110     965.928     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      132       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      682      47\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       24\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          18.9666\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.026   0.026      1.5392       1.000\n",
      " 2 Bond angle potential               :      13       0      0   2.762   2.762      1.7252       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      1  57.481  57.481      5.1630       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.865   2.865     0.59823       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      24       0      1   0.442   0.442      7.3921       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.917   2.917     0.20090       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 121.140 121.140      1.1870       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  28.673  28.673      1.1610       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30  104   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    1    0    0    2    3    7    8    6   10   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      132        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      132       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      682      47\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       21\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          13.4603\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.005   0.005     0.50535E-01   1.000\n",
      " 2 Bond angle potential               :      13       0      0   2.641   2.641      1.6823       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      0  56.007  56.007      5.0125       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.535   2.535     0.56359       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      21       0      1   0.397   0.397      3.5811       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.539   4.539     0.48639       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 121.140 121.140      1.1870       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  25.264  25.264     0.89687       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30  104   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    0    2    3    7    7    6   11   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      132      132\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180676    1153.004     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1181204    1153.520     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229804    1200.980     1.173\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1243104    1213.969     1.186\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1382356    1349.957     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383956    1351.520     1.320\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387936    1355.406     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389936    1357.359     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392936    1360.289     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1397436    1364.684     1.333\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403008    1370.125     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405808    1372.859     1.341\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1410008    1376.961     1.345\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1416308    1383.113     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419080    1385.820     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1420480    1387.188     1.355\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431517    1397.966     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431593    1398.040     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431764    1398.207     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432614    1399.037     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433872    1400.266     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435776    1402.125     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1449200    1415.234     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      132\n",
      "read_mo_298_> Segment:        1  1:M  16:M      132\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1429524    1396.020     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1439032    1405.305     1.372\n",
      "mutate_mode_> Model residue 5 changed from HIS to LYS\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       932008     910.164     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941516     919.449     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943556     921.441     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135      0         0        60\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949384     927.133     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    130    135    184       228        60\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955480     933.086     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    136    186       230        61\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    131    136    186       230        61\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      128        3\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966517     943.864     0.922\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966593     943.938     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966764     944.105     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967614     944.936     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968872     946.164     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970776     948.023     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       984200     961.133     0.939\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      132\n",
      "read_mo_298_> Segment:        1  1:M  16:M      132\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdbLYS5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdbLYS5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       524110     511.826     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630578     615.799     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638770     623.799     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       655154     639.799     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      136     186     230      59\n",
      "              Total number of restraints before, now         :        0      611\n",
      "make_re_422_> Number of previous, current restraints         :        0      611\n",
      "make_re_423_> Number of previous, current selected restraints:        0      611\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655154     639.799     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      611      623\n",
      "make_re_423_> Number of previous, current selected restraints:      611      623\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655170     639.814     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       17\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      623      636\n",
      "make_re_423_> Number of previous, current selected restraints:      623      619\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655170     639.814     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      636      649\n",
      "make_re_423_> Number of previous, current selected restraints:      619      632\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655170     639.814     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      649      658\n",
      "make_re_423_> Number of previous, current selected restraints:      632      641\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655170     639.814     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      658      663\n",
      "make_re_423_> Number of previous, current selected restraints:      641      646\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       655170     639.814     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      663      665\n",
      "make_re_423_> Number of previous, current selected restraints:      646      648\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       52\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       692862     676.623     0.661\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       988610     965.439     0.943\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      665      52\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           6.7767\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.018   0.018     0.88297       1.000\n",
      " 2 Bond angle potential               :      14       0      0   2.120   2.120      1.1749       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  46.929  46.929      3.4028       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   2.760   2.760     0.62059       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.323   0.323     -1.0300       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   0.763   0.763     0.68680E-02   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   6.130   6.130     0.32229       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0  13.798  13.798     0.13852       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.47293E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.000   0.000     0.22251       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  26.609  45.464     0.98791       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    0    2    3    7    7    6   10   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      131        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      131       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      665      52\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :           6.2337\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.002   0.002     0.13120E-01   1.000\n",
      " 2 Bond angle potential               :      14       0      0   1.326   1.326     0.41756       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      0  48.971  48.971      3.3171       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   0.548   0.548     0.31457E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.312   0.312     -1.1038       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       1       0      0   1.639   1.639     0.31691E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 136.336 136.336     0.31591       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   5.858   5.858    -0.11118       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  80.572  80.572      1.3011       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 111.471 111.471     0.94233       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  26.576  45.016      1.0784       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30  103   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    0    2    3    7    6    7    9   13\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      131      131\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180176    1152.516     1.126\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1180704    1153.031     1.126\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1229304    1200.492     1.172\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1242604    1213.480     1.185\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381856    1349.469     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1383456    1351.031     1.319\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387436    1354.918     1.323\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1389436    1356.871     1.325\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1392436    1359.801     1.328\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1396936    1364.195     1.332\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1402508    1369.637     1.338\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1405308    1372.371     1.340\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1409508    1376.473     1.344\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1415808    1382.625     1.350\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418580    1385.332     1.353\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1419980    1386.699     1.354\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431017    1397.478     1.365\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1431093    1397.552     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1431264    1397.719     1.365\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1432114    1398.549     1.366\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1433372    1399.777     1.367\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1435276    1401.637     1.369\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1448700    1414.746     1.382\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1428956    1395.465     1.363\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1438430    1404.717     1.372\n",
      "mutate_mode_> Model residue 6 changed from PHE to PRO\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       931974     910.131     0.889\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       941448     919.383     0.898\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       943376     921.266     0.900\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131      0         0        55\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       949124     926.879     0.905\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  THR  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    126    131    180       226        55\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       955220     932.832     0.911\n",
      "patch_______> segment topology patched using RTF:  16 ;  GLN  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    132    182       228        56\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1       16    127    132    182       228        56\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:      126        1\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966257     943.610     0.921\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       966333     943.685     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       966504     943.852     0.922\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       967354     944.682     0.923\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       968612     945.910     0.924\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       970516     947.770     0.926\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       983940     960.879     0.938\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      131\n",
      "read_mo_298_> Segment:        1  1:M  16:M      131\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdbPRO6.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutated_match.pdbPRO6.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       523850     511.572     0.500\n",
      "read_mo_297_> Segments, residues, atoms:        1       16      127\n",
      "read_mo_298_> Segment:        1  1:M  16:M      127\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       630318     615.545     0.601\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       638510     623.545     0.609\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       654894     639.545     0.625\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :      132     182     235      53\n",
      "              Total number of restraints before, now         :        0      602\n",
      "make_re_422_> Number of previous, current restraints         :        0      602\n",
      "make_re_423_> Number of previous, current selected restraints:        0      602\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654894     639.545     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      602      614\n",
      "make_re_423_> Number of previous, current selected restraints:      602      614\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654910     639.561     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:       18\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      614      627\n",
      "make_re_423_> Number of previous, current selected restraints:      614      609\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654910     639.561     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      627      640\n",
      "make_re_423_> Number of previous, current selected restraints:      609      622\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654910     639.561     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      640      649\n",
      "make_re_423_> Number of previous, current selected restraints:      622      631\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654910     639.561     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      649      655\n",
      "make_re_423_> Number of previous, current selected restraints:      631      637\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       654910     639.561     0.625\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match27\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      655      657\n",
      "make_re_423_> Number of previous, current selected restraints:      637      639\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       64\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       691450     675.244     0.659\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       987102     963.967     0.941\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      657      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        9\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         465.3382\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       1      1   0.436   0.436      381.52       1.000\n",
      " 2 Bond angle potential               :      15       2      2  13.827  13.827      60.714       1.000\n",
      " 3 Stereochemical cosine torsion poten:      29       0      0  36.874  36.874      6.6220       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.690   1.690     0.30690       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       9       0      1   0.577   0.577      3.3040       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.756   1.756     0.72768E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      1  36.135  36.135      5.0254       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1  26.785  26.785      2.8274       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      1  93.130  93.130      2.4774       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  28.001  28.001      2.4699       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     49   6P   6P CG  CD     46   44    2.84    1.54    1.31   35.88    1.54    1.31   35.88\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30   99   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    0    2    3    7    6    6   10   11\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:      127        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :       16\n",
      "Number of all, selected real atoms                :      127       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      657      64\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        8\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          13.5217\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       9       0      0   0.006   0.006     0.70359E-01   1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.522   3.522      3.5569       1.000\n",
      " 3 Stereochemical cosine torsion poten:      29       0      0  40.953  40.953      7.6788       1.000\n",
      " 4 Stereochemical improper torsion pot:       3       0      0   1.403   1.403     0.18666       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       8       0      0   0.500   0.500     0.69552       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.246   3.246     0.24872       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.883   1.883    -0.30045       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   3.771   3.771    -0.22209       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  57.488  57.488    -0.65602E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      0  26.462  26.462      1.6729       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    30   99   2.017\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    0    1    0    2    3    6    7    4   11   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match27_1ebp/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:       16      127      127\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1178668    1151.043     1.124\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1179196    1151.559     1.125\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1227796    1199.020     1.171\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1241096    1212.008     1.184\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1380348    1347.996     1.316\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1381948    1349.559     1.318\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1385928    1353.445     1.322\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1387928    1355.398     1.324\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1390928    1358.328     1.326\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1395428    1362.723     1.331\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1401000    1368.164     1.336\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1403800    1370.898     1.339\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1408000    1375.000     1.343\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1414300    1381.152     1.349\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1417072    1383.859     1.351\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1418472    1385.227     1.353\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429509    1396.005     1.363\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1429585    1396.079     1.363\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1429756    1396.246     1.364\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1430606    1397.076     1.364\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb, Line: 65\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1438078    1404.373     1.371\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       64\n",
      "read_mo_298_> Segment:        1  1:M  7:M       64\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1418141    1384.903     1.352\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1422520    1389.180     1.357\n",
      "mutate_mode_> Model residue 2 changed from GLU to ARG\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       917423     895.921     0.875\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       921802     900.197     0.879\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       922802     901.174     0.880\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     66     67      0         0        29\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       925386     903.697     0.883\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  PHE  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     66     67     87        98        29\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       928402     906.643     0.885\n",
      "patch_______> segment topology patched using RTF:  7 ;  PHE  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     67     68     89       100        30\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     67     68     89       100        30\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       63        4\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       939439     917.421     0.896\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       939515     917.495     0.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       939686     917.662     0.896\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       940536     918.492     0.897\n",
      "readlinef__W> File: /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb, Line: 65\n",
      "              Modeller will only read the first 80 characters of this line.\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       948008     925.789     0.904\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       64\n",
      "read_mo_298_> Segment:        1  1:M  7:M       64\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdbARG2.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       67       67\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/cpep_aligned.pdbARG2.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       487918     476.482     0.465\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       67\n",
      "read_mo_298_> Segment:        1  1:M  7:M       67\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       594386     580.455     0.567\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       68      89      93      30\n",
      "              Total number of restraints before, now         :        0      280\n",
      "make_re_422_> Number of previous, current restraints         :        0      280\n",
      "make_re_423_> Number of previous, current selected restraints:        0      280\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       594488     580.555     0.567\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "errdih__638_> Missing atoms in a dihedral, residue index, residue type:        7        2  GLU\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      280      285\n",
      "make_re_423_> Number of previous, current selected restraints:      280      285\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       594504     580.570     0.567\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:        6\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       602696     588.570     0.575\n",
      "make_re_422_> Number of previous, current restraints         :      285      291\n",
      "make_re_423_> Number of previous, current selected restraints:      285      285\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602696     588.570     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      291      298\n",
      "make_re_423_> Number of previous, current selected restraints:      285      292\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602696     588.570     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      298      305\n",
      "make_re_423_> Number of previous, current selected restraints:      292      299\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602696     588.570     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      305      307\n",
      "make_re_423_> Number of previous, current selected restraints:      299      301\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602696     588.570     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      307      309\n",
      "make_re_423_> Number of previous, current selected restraints:      301      303\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       11       60\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       621956     607.379     0.593\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       916168     894.695     0.874\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       67      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      309      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       28\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :       55977.1055\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.036   0.036      4.7345       1.000\n",
      " 2 Bond angle potential               :      17       0      0   3.712   3.712      4.0501       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  61.712  61.712      7.1224       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   5.224   5.224      1.8498       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      28       0     10   1.058   1.058      55951.       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   3.937   3.937     0.36593       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  12.966  12.966     0.29471       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0 106.198 106.198     0.68836       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   0.000   0.000     0.42810E-01   1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  90.008  90.008     0.73240       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      2  23.061  23.061      5.8959       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    16   45   1.342\n",
      "  serious non-bonded atom clash:    17   45   2.155\n",
      "  serious non-bonded atom clash:    20   44   1.976\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    0    0    2    0    1    3    1    0    1    1    7    2   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       67       11        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       67      11\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      309      60\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       15\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          22.4641\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      12       0      0   0.019   0.019      1.2089       1.000\n",
      " 2 Bond angle potential               :      17       0      0   4.607   4.607      6.1981       1.000\n",
      " 3 Stereochemical cosine torsion poten:      19       0      2  63.167  63.167      7.1724       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   3.156   3.156     0.66968       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      15       0      1   0.571   0.571      3.9481       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.517   4.517     0.48167       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 111.996 111.996      1.1032       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   9.061   9.061     0.17035       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0   7.231   7.231     0.17661       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 115.780 115.780     0.76502       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       2       0      0  13.497  13.497     0.57002       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    2    2    5    1   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutation0.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       67       67\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1107734    1081.771     1.056\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1108262    1082.287     1.057\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1156862    1129.748     1.103\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1170162    1142.736     1.116\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1309414    1278.725     1.249\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1311014    1280.287     1.250\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1314994    1284.174     1.254\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1316994    1286.127     1.256\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1319994    1289.057     1.259\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1324494    1293.451     1.263\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1330066    1298.893     1.268\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332866    1301.627     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1337066    1305.729     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343366    1311.881     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1346138    1314.588     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347538    1315.955     1.285\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1358575    1326.733     1.296\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1358651    1326.808     1.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1358822    1326.975     1.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1359672    1327.805     1.297\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1367144    1335.102     1.304\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       67\n",
      "read_mo_298_> Segment:        1  1:M  7:M       67\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1357637    1325.817     1.295\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1362118    1330.193     1.299\n",
      "mutate_mode_> Model residue 3 changed from LEU to VAL\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       908411     887.120     0.866\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       912892     891.496     0.871\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       913884     892.465     0.872\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     65     66      0         0        29\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       916456     894.977     0.874\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  PHE  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     65     66     86        98        29\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       919472     897.922     0.877\n",
      "patch_______> segment topology patched using RTF:  7 ;  PHE  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     66     67     88       100        30\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     66     67     88       100        30\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       64        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       930509     908.700     0.887\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       930585     908.774     0.887\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       930756     908.941     0.888\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       931606     909.771     0.888\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       939078     917.068     0.896\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       67\n",
      "read_mo_298_> Segment:        1  1:M  7:M       67\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbVAL3.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       66       66\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbVAL3.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       488102     476.662     0.465\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       66\n",
      "read_mo_298_> Segment:        1  1:M  7:M       66\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       594570     580.635     0.567\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       67      88      93      30\n",
      "              Total number of restraints before, now         :        0      278\n",
      "make_re_422_> Number of previous, current restraints         :        0      278\n",
      "make_re_423_> Number of previous, current selected restraints:        0      278\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       594570     580.635     0.567\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      278      283\n",
      "make_re_423_> Number of previous, current selected restraints:      278      283\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       594586     580.650     0.567\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:        6\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      283      289\n",
      "make_re_423_> Number of previous, current selected restraints:      283      283\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       594586     580.650     0.567\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       602778     588.650     0.575\n",
      "make_re_422_> Number of previous, current restraints         :      289      296\n",
      "make_re_423_> Number of previous, current selected restraints:      283      290\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602778     588.650     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      296      302\n",
      "make_re_423_> Number of previous, current selected restraints:      290      296\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602778     588.650     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      302      304\n",
      "make_re_423_> Number of previous, current selected restraints:      296      298\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       602778     588.650     0.575\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      304      306\n",
      "make_re_423_> Number of previous, current selected restraints:      298      300\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        7       48\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       621750     607.178     0.593\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       915938     894.471     0.874\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       66       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      306      48\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        6\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          28.3244\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.046   0.046      4.4294       1.000\n",
      " 2 Bond angle potential               :      13       0      0   6.307   6.307      8.9175       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      1  76.011  76.011      7.3179       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.438   2.438     0.40069       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       6       0      0   0.412   0.412     0.16179       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   1.035   1.035     0.25288E-01   1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 116.229 116.229     0.79460E-01   1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  64.620  64.620      6.9923       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    281   3V   4D C   N      28   30  158.72   54.50  110.12   14.17   54.50  110.12   14.17\n",
      "    1          4D   4D N   CA     30   31   76.47   40.90                   40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    2    2    5    1   12\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       66        7        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       66       7\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      306      48\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):        9\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          17.2517\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :       8       0      0   0.013   0.013     0.37234       1.000\n",
      " 2 Bond angle potential               :      13       0      0   3.138   3.138      2.2164       1.000\n",
      " 3 Stereochemical cosine torsion poten:      17       0      1  72.934  72.934      6.4309       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   2.542   2.542     0.50658       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       9       0      0   0.458   0.458     0.98270       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   2.797   2.797     0.18462       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 116.351 116.351     0.83952       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       1      1  50.551  79.927      5.7187       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1    281   3V   4D C   N      28   30 -173.11  -96.50   85.40    3.48   54.50  137.08   17.46\n",
      "    1          4D   4D N   CA     30   31   76.47  114.20                   40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    2    3    6    3   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutation1.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       66       66\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1107504    1081.547     1.056\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1108032    1082.062     1.057\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1156632    1129.523     1.103\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1169932    1142.512     1.116\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1309184    1278.500     1.249\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1310784    1280.062     1.250\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1314764    1283.949     1.254\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1316764    1285.902     1.256\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1319764    1288.832     1.259\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1324264    1293.227     1.263\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1329836    1298.668     1.268\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332636    1301.402     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1336836    1305.504     1.275\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1343136    1311.656     1.281\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345908    1314.363     1.284\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1347308    1315.730     1.285\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1358345    1326.509     1.295\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1358421    1326.583     1.295\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1358592    1326.750     1.296\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1359442    1327.580     1.296\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1366914    1334.877     1.304\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       66\n",
      "read_mo_298_> Segment:        1  1:M  7:M       66\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1357305    1325.493     1.294\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1361752    1329.836     1.299\n",
      "mutate_mode_> Model residue 4 changed from ASP to TRP\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       908377     887.087     0.866\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       912824     891.430     0.871\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       913976     892.555     0.872\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     71     74      0         0        35\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       917012     895.520     0.875\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  PHE  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     71     74     98       118        35\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       920028     898.465     0.877\n",
      "patch_______> segment topology patched using RTF:  7 ;  PHE  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     72     75    100       120        36\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     72     75    100       120        36\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       64        8\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       931065     909.243     0.888\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       931141     909.317     0.888\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       931312     909.484     0.888\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       932162     910.314     0.889\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       939634     917.611     0.896\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       66\n",
      "read_mo_298_> Segment:        1  1:M  7:M       66\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbTRP4.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       72       72\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbTRP4.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       488658     477.205     0.466\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       72\n",
      "read_mo_298_> Segment:        1  1:M  7:M       72\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       595126     581.178     0.568\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       603318     589.178     0.575\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       75     100     113      36\n",
      "              Total number of restraints before, now         :        0      324\n",
      "make_re_422_> Number of previous, current restraints         :        0      324\n",
      "make_re_423_> Number of previous, current selected restraints:        0      324\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603522     589.377     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      324      329\n",
      "make_re_423_> Number of previous, current selected restraints:      324      329\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603538     589.393     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:        6\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      329      335\n",
      "make_re_423_> Number of previous, current selected restraints:      329      329\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603538     589.393     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      335      342\n",
      "make_re_423_> Number of previous, current selected restraints:      329      336\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603538     589.393     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      342      348\n",
      "make_re_423_> Number of previous, current selected restraints:      336      342\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603538     589.393     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      348      350\n",
      "make_re_423_> Number of previous, current selected restraints:      342      344\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603538     589.393     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      350      352\n",
      "make_re_423_> Number of previous, current selected restraints:      344      346\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;       14       97\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       624238     609.607     0.595\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       918570     897.041     0.876\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       72      14\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      352      97\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          29.3833\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      17       0      0   0.029   0.029      4.1221       1.000\n",
      " 2 Bond angle potential               :      26       0      1   3.296   3.296      5.6801       1.000\n",
      " 3 Stereochemical cosine torsion poten:      37       0      2  56.983  56.983      9.3340       1.000\n",
      " 4 Stereochemical improper torsion pot:      10       0      0   2.221   2.221     0.97154       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      0   0.388   0.388     0.30747       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.588   4.588     0.49677       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   1.155   1.155    -0.30820       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   8.803   8.803     0.72499E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  52.752  54.850      8.7069       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    2    3    6    3   10\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       72       14        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       72      14\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      352      97\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       12\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          20.5471\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      17       0      0   0.004   0.004     0.83212E-01   1.000\n",
      " 2 Bond angle potential               :      26       0      0   2.561   2.561      3.2953       1.000\n",
      " 3 Stereochemical cosine torsion poten:      37       0      2  55.690  55.690      8.9069       1.000\n",
      " 4 Stereochemical improper torsion pot:      10       0      0   0.311   0.311     0.32479E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      12       0      0   0.391   0.391     0.60716       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   4.808   4.808     0.54560       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0   5.658   5.658    -0.16966       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   3.618   3.618    -0.10427       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      1  48.581  48.581      7.3504       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    3    2    5    3    9\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutation2.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       72       72\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1110136    1084.117     1.059\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1110664    1084.633     1.059\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1159264    1132.094     1.106\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1172564    1145.082     1.118\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch3.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/xs4.mat\n",
      "rdrrwgh_268_> Number of residue types:       21\n",
      "openf___224_> Open           $(LIB)/top_heav.lib\n",
      "read_to_681_> topology.submodel read from topology file:        3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "openf___224_> Open           $(LIB)/par.lib\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1311816    1281.070     1.251\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1313416    1282.633     1.253\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1317396    1286.520     1.256\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1319396    1288.473     1.258\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1322396    1291.402     1.261\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1326896    1295.797     1.265\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1332468    1301.238     1.271\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1335268    1303.973     1.273\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1339468    1308.074     1.277\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1345768    1314.227     1.283\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1348540    1316.934     1.286\n",
      "\n",
      "Dynamically allocated memory at  amaxparameters [B,KiB,MiB]:      1349940    1318.301     1.287\n",
      "read_pa_232_> parameters    BONDS   ANGLS  DIHEDS IMPROPS    MODE\n",
      "                              227     561     661     112       0\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1360977    1329.079     1.298\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:      1361053    1329.153     1.298\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1361224    1329.320     1.298\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:      1362074    1330.150     1.299\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:      1369546    1337.447     1.306\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       72\n",
      "read_mo_298_> Segment:        1  1:M  7:M       72\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:      1359801    1327.931     1.297\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:      1364452    1332.473     1.301\n",
      "mutate_mode_> Model residue 5 changed from LYS to GLU\n",
      "\n",
      "Dynamically allocated memory at   amaxalignment [B,KiB,MiB]:       908581     887.286     0.866\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       913232     891.828     0.871\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       914400     892.969     0.872\n",
      "\n",
      "mkilst______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     71     74      0         0        36\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       917448     895.945     0.875\n",
      "\n",
      "patch_______> segment topology patched using RTF:  1 ;  PHE  ;  NTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     71     74     99       118        36\n",
      "\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       920464     898.891     0.878\n",
      "patch_______> segment topology patched using RTF:  7 ;  PHE  ;  CTER\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     72     75    101       120        37\n",
      "\n",
      "genseg______> segment topology constructed from sequence and RTF:\n",
      "              segments residues  atoms  bonds angles dihedrals impropers:\n",
      "                     1        7     72     75    101       120        37\n",
      "transfe_506_> MODEL is an average of all templates.\n",
      "transfe_511_> Number of templates for coordinate transfer:        1\n",
      "              After transferring coordinates of the equivalent template atoms,\n",
      "              there are defined, undefined atoms in MODEL:       70        2\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       931501     909.669     0.888\n",
      "\n",
      "Dynamically allocated memory at    amaxsequence [B,KiB,MiB]:       931577     909.743     0.888\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       931748     909.910     0.889\n",
      "\n",
      "Dynamically allocated memory at amaxcoordinates [B,KiB,MiB]:       932598     910.740     0.889\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       940070     918.037     0.897\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       72\n",
      "read_mo_298_> Segment:        1  1:M  7:M       72\n",
      "  #      ALGNMT CODE\n",
      "\n",
      "  1  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "  2  /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdb\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbGLU5.tmp\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       72       72\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutated_match.pdbGLU5.tmp\n",
      "\n",
      "Dynamically allocated memory at       amaxmodel [B,KiB,MiB]:       489094     477.631     0.466\n",
      "read_mo_297_> Segments, residues, atoms:        1        7       72\n",
      "read_mo_298_> Segment:        1  1:M  7:M       72\n",
      "make_re_417_> Restraint type to be calculated:  stereo\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       595562     581.604     0.568\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       603754     589.604     0.576\n",
      "r_stere_606_> Stereochemical restraints were constructed from RTF & PRMF.\n",
      "              Added bond,angle,dihedral,improper restraints  :       75     101     113      37\n",
      "              Total number of restraints before, now         :        0      326\n",
      "make_re_422_> Number of previous, current restraints         :        0      326\n",
      "make_re_423_> Number of previous, current selected restraints:        0      326\n",
      "make_re_417_> Restraint type to be calculated:  phi-psi_binormal\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603754     589.604     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      326      331\n",
      "make_re_423_> Number of previous, current selected restraints:      326      331\n",
      "make_re_417_> Restraint type to be calculated:  omega_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603770     589.619     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/omega.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "omgdel__425_> Unselected all O C +N +CA dihedrals:        6\n",
      "              (This is to avoid clashes between STEREO\n",
      "              and OMEGA_DIHEDRAL restraints)\n",
      "make_re_422_> Number of previous, current restraints         :      331      337\n",
      "make_re_423_> Number of previous, current selected restraints:      331      331\n",
      "make_re_417_> Restraint type to be calculated:  chi1_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603770     589.619     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      337      344\n",
      "make_re_423_> Number of previous, current selected restraints:      331      338\n",
      "make_re_417_> Restraint type to be calculated:  chi2_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603770     589.619     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi2.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      344      350\n",
      "make_re_423_> Number of previous, current selected restraints:      338      344\n",
      "make_re_417_> Restraint type to be calculated:  chi3_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603770     589.619     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi3.mdt\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      350      352\n",
      "make_re_423_> Number of previous, current selected restraints:      344      346\n",
      "make_re_417_> Restraint type to be calculated:  chi4_dihedral\n",
      "\n",
      "Dynamically allocated memory at   amaxstructure [B,KiB,MiB]:       603770     589.619     0.576\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi1234.bin\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/chi4.mdt\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "getdata_643_> Protein accepted:  /home/brianda/Dev/cPEPmatch/1e03/match28\n",
      "getdata_289_> Proteins (all/accepted):        1        1\n",
      "make_re_422_> Number of previous, current restraints         :      352      353\n",
      "make_re_423_> Number of previous, current selected restraints:      346      347\n",
      "pick_re_612_> Number of MODEL atoms, selected restraints;        9       55\n",
      "\n",
      "Dynamically allocated memory at amaxhash_contac [B,KiB,MiB]:       624470     609.834     0.596\n",
      "\n",
      "Dynamically allocated memory at  amaxrestraints [B,KiB,MiB]:       918802     897.268     0.876\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       72       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      353      55\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       11\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          43.4559\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.028   0.028      2.0581       1.000\n",
      " 2 Bond angle potential               :      15       0      3   6.912   6.912      13.132       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      1  53.901  53.901      5.1210       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   3.980   3.980      1.0763       1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      11       0      1   0.501   0.501      3.5978       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   6.074   6.074     0.87088       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 135.432 135.432     0.54173       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      1 178.413 178.413      5.0258       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 179.254 179.254      1.1357       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  50.220  50.220      10.897       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1    0    0    1    0    4    3    4    3    8\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "randomi_498_> Atoms,selected atoms,random_seed,amplitude:       72        9        1        4.0000\n",
      "randomi_496_> Amplitude is > 0; randomization is done.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :        7\n",
      "Number of all, selected real atoms                :       72       9\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :      353      55\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       16\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       T       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          20.0174\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      10       0      0   0.010   0.010     0.34049       1.000\n",
      " 2 Bond angle potential               :      15       0      0   3.291   3.291      3.2519       1.000\n",
      " 3 Stereochemical cosine torsion poten:      18       0      1  53.718  53.718      5.1665       1.000\n",
      " 4 Stereochemical improper torsion pot:       4       0      0   0.419   0.419     0.25557E-01   1.000\n",
      " 5 Soft-sphere overlap restraints     :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :      16       0      0   0.292   0.292    -0.93714       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "10 Distance restraints 2 (N-O)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:       2       0      0   9.101   9.101      1.9552       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0 107.488 107.488     0.54171       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   4.241   4.241     0.45681E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  12.730  12.730     0.80773       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:       3       0      2  47.448  47.448      8.8198       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    0    0    1    0    3    1    3    4    7\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "openf___224_> Open           /home/brianda/Dev/cPEPmatch/1e03/match28_1foz/mutation3.pdb\n",
      "wrpdb___568_> Residues, atoms, selected atoms:        7       72       72\n",
      "openf___224_> Open           $(LIB)/restyp.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resgrp.lib\n",
      "rdresgr_266_> Number of residue groups:        2\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/sstruc.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1110368    1084.344     1.059\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1110896    1084.859     1.059\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/resdih.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1159496    1132.320     1.106\n",
      "rdrdih__263_> Number of dihedral angle types         :        9\n",
      "              Maximal number of dihedral angle optima:        3\n",
      "              Dihedral angle names                   :  Alph Phi Psi Omeg chi1 chi2 chi3 chi4 chi5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/radii.lib\n",
      "\n",
      "Dynamically allocated memory at   amaxlibraries [B,KiB,MiB]:      1172796    1145.309     1.118\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/af_mnchdef.lib\n",
      "rdwilmo_274_> Mainchain residue conformation classes:  APBLE\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch.lib\n",
      "rdclass_257_> Number of classes:        5\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch1.lib\n",
      "openf___224_> Open           ${MODINSTALL10v1}/modlib/mnch2.lib\n"
     ]
    }
   ],
   "source": [
    "#TODO: Adapt a 5-motif superimposition and mutation\n",
    "\n",
    "os.chdir('{}{}'.format(my_location, pdb_name))\n",
    "cwd = ('{}{}'.format(my_location, pdb_name))\n",
    "\n",
    "#Delete old match files \n",
    "dir = [os.listdir('.')]\n",
    "for f in dir:\n",
    "    for a in f:\n",
    "        if \"match\" in a:\n",
    "            try:\n",
    "                os.remove(a)\n",
    "            except OSError as e:\n",
    "                 pass\n",
    "                    \n",
    "                    \n",
    "\n",
    "match_directory_list = []\n",
    "all_mutations = dict()\n",
    "motif_end = motif_size - 1\n",
    "\n",
    "cr = 1\n",
    "c1 = 1\n",
    "real_match_count = 1\n",
    "\n",
    "with open('match_list.txt', 'w') as f:\n",
    "    f.write(\"List of matches found by cPEPmatch.\\nMotif Size: {} \\nInterface Cutoff: {} \\nFit-RMSD Threshold: {} \\n\\n\" .format(motif_size, interface_cutoff,frmsd_threshold))\n",
    "    f.write(\" Match  pdb   Dist-RMSD    CPep Residues        PP-Interface Residues   Fit-RMSD\\n\\n\")\n",
    "    \n",
    "    for match in all_match: \n",
    "        \n",
    "        no_overlap_match = []\n",
    "    \n",
    "        pdb_parser = Bio.PDB.PDBParser(QUIET = True)\n",
    "        matched_cyclopep = match[0]\n",
    "        match_dir = (\"match{}_{}\".format(cr, matched_cyclopep))\n",
    "        match_directory_list.append(match_dir)\n",
    "    \n",
    "        \n",
    "        #Selection of residues to be aligned\n",
    "    \n",
    "        #cyclo_start_id = match[2][0]\n",
    "        #cyclo_end_id = match[2][motif_end]\n",
    "        cyclo_residues_to_be_aligned = match[2]\n",
    "    \n",
    "        #ppi_start_id = match[3][0]\n",
    "        #ppi_end_id = match[3][motif_end]\n",
    "        ppi_residues_to_be_aligned = match[3]\n",
    "    \n",
    "    \n",
    "        #Create a directory for each match\n",
    "    \n",
    "        try:\n",
    "            os.mkdir(\"{}\".format(match_dir))\n",
    "            cr  =  cr + 1\n",
    "        except:\n",
    "            print(\"Match folder already exist\")\n",
    "    \n",
    "        #cpep = (cwd + ('/cyclo_library/{}-cp.pdb'.format(matched_cyclopep)))\n",
    "        cpep = ('{}cyclo_database/{}-cp.pdb'.format(my_location, matched_cyclopep))\n",
    "        match_folder = (cwd + ('/{}/cpep_match.pdb'.format(match_dir)))\n",
    "        match_path = (cwd + ('/{}/'.format(match_dir)))\n",
    "\n",
    "        try:\n",
    "            #os.popen('cp {} {}'.format(cpep, os.path.join(match_folder, filename)))\n",
    "            #copyfile(cpep, match_folder)\n",
    "            shutil.copy(cpep, match_folder, follow_symlinks=True)\n",
    "            \n",
    "        except IOError as e:\n",
    "            print(\"Unable to copy file. %s\" % e)\n",
    "            \n",
    "        \n",
    "        # Get the structures\n",
    "        cyclo_structure = pdb_parser.get_structure(\"reference\", (cwd + (\"/{}/cpep_match.pdb\".format(match_dir))))  \n",
    "        ppi_structure = pdb_parser.get_structure(\"sample\", (cwd + \"/interface.pdb\"))\n",
    "        \n",
    "        \n",
    "        cyclo_model = cyclo_structure[0]\n",
    "        cyclo_chain = cyclo_model['M']\n",
    "        ppi_model = ppi_structure[0]\n",
    "        ppi_chain = ppi_model['X']\n",
    "        \n",
    "\n",
    "        # List of atoms to align.\n",
    "        cyclo_atoms = []\n",
    "        ppi_atoms = []\n",
    "\n",
    "        for cyclo_chain in cyclo_model:\n",
    "            for cyclo_res in cyclo_chain:\n",
    "                if cyclo_res.get_id()[1] in cyclo_residues_to_be_aligned: # Check if residue number ( .get_id() ) is in the list\n",
    "                    cyclo_atoms.append(cyclo_res['CA'])\n",
    "\n",
    "        \n",
    "        for ppi_chain in ppi_model:\n",
    "            for ppi_res in ppi_chain:\n",
    "                if ppi_res.get_id()[1] in ppi_residues_to_be_aligned:\n",
    "                    ppi_atoms.append(ppi_res['CA'])\n",
    "        \n",
    "        # Superimposer:\n",
    "        super_imposer = Bio.PDB.Superimposer()\n",
    "        super_imposer.set_atoms(ppi_atoms, cyclo_atoms)\n",
    "        super_imposer.apply(cyclo_model.get_atoms())\n",
    "\n",
    "        # Add to fit_RMSD the results:\n",
    "        superimpose_rmsd = super_imposer.rms\n",
    "        lst = list(match)\n",
    "        lst = lst + [superimpose_rmsd]\n",
    "        match = tuple(lst)\n",
    "    \n",
    "        # Save the aligned version \n",
    "        io = Bio.PDB.PDBIO()\n",
    "        io.set_structure(cyclo_structure) \n",
    "        io.save(match_path + \"cpep_aligned.pdb\") \n",
    "        \n",
    "        \n",
    "        #Eliminates the matched cyclic peptides that overlap with the receptor. \n",
    "        \n",
    "        cyclo_rlist = int_list(match[2], quote='\"')\n",
    "        matched_cyclo_residues = cyclo_rlist\n",
    "        overlap = overlap_finder(match_path, (cwd + (\"/{}.pdb\".format(pdb_name))), matched_cyclo_residues)\n",
    "\n",
    "        if overlap == [[],[]]:\n",
    "            no_overlap_match = match\n",
    "        \n",
    "        \n",
    "        #WRITE OUTPUT FILE\n",
    "        if no_overlap_match != []:\n",
    "            c2 = 0\n",
    "            f.write(\"{:>6}\".format(c1))\n",
    "            for i in no_overlap_match:\n",
    "                if c2 < 1:\n",
    "                    f.write(\"{:>6} \".format(i))\n",
    "                elif c2 < 2:\n",
    "                    f.write(\"{:>9.4f}\".format(i))\n",
    "                elif c2 < 3:\n",
    "                    f.write(\"   \")\n",
    "                    for r in i:\n",
    "                        f.write(\"{:>5}\".format(r))\n",
    "                elif c2 < 4:\n",
    "                    f.write(\"   \")\n",
    "                    for r in i:\n",
    "                        f.write(\"{:>5}\".format(r))\n",
    "                else:\n",
    "                    f.write(\"   \")\n",
    "                    f.write(\"{:>9.4f}\".format(i))\n",
    "\n",
    "                c2 = c2 + 1\n",
    "\n",
    "            f.write(\"\\n\")\n",
    "            c1 = c1 + 1  \n",
    "\n",
    "    \n",
    "     #MUTATE\n",
    "            if MUTATE:\n",
    "           \n",
    "                modelname = (match_path + 'cpep_aligned.pdb') \n",
    "                motif_length = len(match[2]) - 1\n",
    "                mutation_count = 0\n",
    "\n",
    "                for matched_residue in match[2]:\n",
    "\n",
    "                    respos = str(matched_residue)\n",
    "                    residue_number_to_match = match[3][mutation_count]\n",
    "                    restyp = ppi_chain[residue_number_to_match].get_resname()     \n",
    "                    cyclic_restyp = cyclo_chain[matched_residue].get_resname()\n",
    "                    chain = 'M'\n",
    "                    disulfide = 'CYX'\n",
    "\n",
    "                    if cyclic_restyp != disulfide:\n",
    "\n",
    "                        try:\n",
    "                            log.verbose()\n",
    "\n",
    "                            # Set a different value for rand_seed to get a different final model\n",
    "                            env = Environ(rand_seed=-49837)\n",
    "\n",
    "                            env.io.hetatm = True\n",
    "                            #soft sphere potential\n",
    "                            env.edat.dynamic_sphere=False\n",
    "                            #lennard-jones potential (more accurate)\n",
    "                            env.edat.dynamic_lennard=True\n",
    "                            env.edat.contact_shell = 4.0\n",
    "                            env.edat.update_dynamic = 0.39\n",
    "\n",
    "                            # Read customized topology file with phosphoserines (or standard one)\n",
    "                            env.libs.topology.read(file='$(LIB)/top_heav.lib')\n",
    "\n",
    "                            # Read customized CHARMM parameter library with phosphoserines (or standard one)\n",
    "                            env.libs.parameters.read(file='$(LIB)/par.lib')\n",
    "\n",
    "\n",
    "                            # Read the original PDB file and copy its sequence to the alignment array:\n",
    "                            mdl1 = Model(env, file=modelname)\n",
    "                            ali = Alignment(env)\n",
    "                            ali.append_model(mdl1, atom_files=modelname, align_codes=modelname)\n",
    "\n",
    "                            #set up the mutate residue selection segment\n",
    "                            s = Selection(mdl1.chains[chain].residues[respos])\n",
    "\n",
    "                            #perform the mutate residue operation\n",
    "                            s.mutate(residue_type=restyp)\n",
    "                            #get two copies of the sequence.  A modeller trick to get things set up\n",
    "                            ali.append_model(mdl1, align_codes=modelname)\n",
    "\n",
    "                            # Generate molecular topology for mutant\n",
    "                            mdl1.clear_topology()\n",
    "                            mdl1.generate_topology(ali[-1])\n",
    "\n",
    "\n",
    "                            # Transfer all the coordinates you can from the template native structure\n",
    "                            # to the mutant (this works even if the order of atoms in the native PDB\n",
    "                            # file is not standard):\n",
    "                            #here we are generating the model by reading the template coordinates\n",
    "                            mdl1.transfer_xyz(ali)\n",
    "\n",
    "                            # Build the remaining unknown coordinates\n",
    "                            mdl1.build(initialize_xyz=False, build_method='INTERNAL_COORDINATES')\n",
    "\n",
    "                            #yes model2 is the same file as model1.  It's a modeller trick.\n",
    "                            mdl2 = Model(env, file=modelname)\n",
    "\n",
    "                            #required to do a transfer_res_numb\n",
    "                            #ali.append_model(mdl2, atom_files=modelname, align_codes=modelname)\n",
    "                            #transfers from \"model 2\" to \"model 1\"\n",
    "                            mdl1.res_num_from(mdl2,ali)\n",
    "\n",
    "                            #It is usually necessary to write the mutated sequence out and read it in\n",
    "                            #before proceeding, because not all sequence related information about MODEL\n",
    "                            #is changed by this command (e.g., internal coordinates, charges, and atom\n",
    "                            #types and radii are not updated).\n",
    "\n",
    "                            mdl1.write(file=modelname+restyp+respos+'.tmp')\n",
    "                            mdl1.read(file=modelname+restyp+respos+'.tmp')\n",
    "\n",
    "                            #set up restraints before computing energy\n",
    "                            #we do this a second time because the model has been written out and read in,\n",
    "                            #clearing the previously set restraints\n",
    "                            make_restraints(mdl1, ali)\n",
    "\n",
    "                            #a non-bonded pair has to have at least as many selected atoms\n",
    "                            mdl1.env.edat.nonbonded_sel_atoms=1\n",
    "\n",
    "                            sched = autosched.loop.make_for_model(mdl1)\n",
    "\n",
    "                            #only optimize the selected residue (in first pass, just atoms in selected\n",
    "                            #residue, in second pass, include nonbonded neighboring atoms)\n",
    "                            #set up the mutate residue selection segment\n",
    "                            s = Selection(mdl1.chains[chain].residues[respos])\n",
    "\n",
    "                            mdl1.restraints.unpick_all()\n",
    "                            mdl1.restraints.pick(s)\n",
    "\n",
    "                            try:\n",
    "                                s.energy()\n",
    "                            except RuntimeError:\n",
    "                                pass\n",
    "\n",
    "                            s.randomize_xyz(deviation=4.0)\n",
    "\n",
    "                            mdl1.env.edat.nonbonded_sel_atoms=2\n",
    "\n",
    "                            try:\n",
    "                                optimize(s, sched)\n",
    "                            except RuntimeError:\n",
    "                                pass\n",
    "\n",
    "\n",
    "                            #feels environment (energy computed on pairs that have at least one member\n",
    "                            #in the selected)\n",
    "                            mdl1.env.edat.nonbonded_sel_atoms=1\n",
    "\n",
    "                            try:\n",
    "                                optimize(s, sched)\n",
    "                            except RuntimeError:\n",
    "                                pass\n",
    "\n",
    "\n",
    "                            try:\n",
    "                                s.energy()\n",
    "                            except RuntimeError:\n",
    "                                pass\n",
    "\n",
    "\n",
    "                            #Output file\n",
    "                            #mdl1.write(file=modelname+restyp+respos+'.pdb')\n",
    "                            mdl1.write(match_path + ('mutation{}.pdb'.format(mutation_count)))\n",
    "\n",
    "                            #delete the temporary file\n",
    "                            os.remove(modelname+restyp+respos+'.tmp')\n",
    "\n",
    "                            shutil.copyfile((match_path + ('mutation{}.pdb'.format(mutation_count))), (match_path + ('mutated_match.pdb')))\n",
    "                            modelname = (match_path + ('mutated_match.pdb'))\n",
    "\n",
    "                            mutation_count = mutation_count + 1\n",
    "\n",
    "\n",
    "                        except RuntimeError as e:\n",
    "                            print('Non-standard amino acid found in the cyclic peptide sequence {}.'\n",
    "                                  ' I will not be mutated'.format(matched_cyclopep))\n",
    "                            pass\n",
    "\n",
    "                    else:\n",
    "                        mutation_count = mutation_count + 1\n",
    "            \n",
    "            \n",
    "                shutil.copyfile((match_path + ('mutated_match.pdb')), \n",
    "                                ('match{}_{}.pdb'.format(real_match_count, matched_cyclopep)))\n",
    "                real_match_count = real_match_count + 1\n",
    "            \n",
    "        \n",
    "                for fname in os.listdir(match_path):\n",
    "                    if fname.startswith(\"mutation\"):\n",
    "                        os.remove(os.path.join(match_path, fname))\n",
    "            \n",
    "                shutil.rmtree(match_dir)    \n",
    "            \n",
    "            \n",
    "        else:\n",
    "            shutil.rmtree(match_dir)\n",
    "\n",
    "\n",
    "                \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
